<!DOCTYPE html>
<html>
<head>
            <meta charset="utf-8">
            <meta name="viewport" content="width=device-width, initial-scale=1.0">
            <!-- Bootstrap -->
            <link
                    rel="stylesheet"
                    href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
                    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
                    crossorigin="anonymous"
            />
            <link
                    rel="stylesheet"
                    href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"
            />
            <link
                    rel="stylesheet"
                    href="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.css"
            />
            <!-- DataTables, see: https://datatables.net/examples/styling/bootstrap4.html-->
            <link
                    rel="stylesheet"
                    href="https://cdn.datatables.net/1.10.19/css/dataTables.bootstrap4.min.css"
            />
            <style type="text/css">
                h2 {
                    margin-top: 35px;
                }

                html, body {
                    height: 100%;
                }

                body {
                    display: flex;
                    flex-flow: column;
                }

                .footer {
                    margin-top: auto;
                    padding-top: 1em;
                    background-color: #f5f5f5;
                }

                nav[data-toggle="toc"] {
                    top: 42px;
                }
            </style>

        <script src="https://kit.fontawesome.com/4c86883252.js" crossorigin="anonymous"></script>
        <title>WHITEHURST_PACLITAXEL_SENSITIVITY</title>
</head>

<body data-spy="scroll" data-target="#toc">
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>


    <div class="container" style="margin-top: 50px; margin-bottom: 50px">
        <div class="card">
        <div class="card-header">
            WHITEHURST_PACLITAXEL_SENSITIVITY
        </div>
        <div class="card-body">
            
                <p>Abundant evidence suggests that a unifying principle governing the molecular pathology of cancer is the co-dependent aberrant regulation of core machinery driving proliferation and suppressing apoptosis. Anomalous proteins engaged in support of this tumorigenic regulatory environment most probably represent optimal intervention targets in a heterogeneous population of cancer cells. The advent of RNA-mediated interference (RNAi)-based functional genomics provides the opportunity to derive unbiased comprehensive collections of validated gene targets supporting critical biological systems outside the framework of preconceived notions of mechanistic relationships. We have combined a high-throughput cell-based one-well/one-gene screening platform with a genome-wide synthetic library of chemically synthesized small interfering RNAs for systematic interrogation of the molecular underpinnings of cancer cell chemoresponsiveness. NCI-H1155, a human non-small-cell lung cancer line, was employed in a paclitaxel-dependent synthetic lethal screen designed to identify gene targets that specifically reduce cell viability in the presence of otherwise sublethal concentrations of paclitaxel. Using a stringent objective statistical algorithm to reduce false discovery rates below 5%, we isolated a panel of 87 genes that represent major focal points of the autonomous response of cancer cells to the abrogation of microtubule dynamics. Here we show that several of these targets sensitize lung cancer cells to paclitaxel concentrations 1,000-fold lower than otherwise required for a significant response, and we identify mechanistic relationships between cancer-associated aberrant gene expression programmes and the basic cellular machinery required for robust mitotic progression.</p>
            
            <dl>
                <dt>Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M10597">M10597</a></dd>
                
            </dl>


            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M10597&#39;, name=&#39;WHITEHURST_PACLITAXEL_SENSITIVITY&#39;), definition=&#39;Abundant evidence suggests that a unifying principle governing the molecular pathology of cancer is the co-dependent aberrant regulation of core machinery driving proliferation and suppressing apoptosis. Anomalous proteins engaged in support of this tumorigenic regulatory environment most probably represent optimal intervention targets in a heterogeneous population of cancer cells. The advent of RNA-mediated interference (RNAi)-based functional genomics provides the opportunity to derive unbiased comprehensive collections of validated gene targets supporting critical biological systems outside the framework of preconceived notions of mechanistic relationships. We have combined a high-throughput cell-based one-well/one-gene screening platform with a genome-wide synthetic library of chemically synthesized small interfering RNAs for systematic interrogation of the molecular underpinnings of cancer cell chemoresponsiveness. NCI-H1155, a human non-small-cell lung cancer line, was employed in a paclitaxel-dependent synthetic lethal screen designed to identify gene targets that specifically reduce cell viability in the presence of otherwise sublethal concentrations of paclitaxel. Using a stringent objective statistical algorithm to reduce false discovery rates below 5%, we isolated a panel of 87 genes that represent major focal points of the autonomous response of cancer cells to the abrogation of microtubule dynamics. Here we show that several of these targets sensitize lung cancer cells to paclitaxel concentrations 1,000-fold lower than otherwise required for a significant response, and we identify mechanistic relationships between cancer-associated aberrant gene expression programmes and the basic cellular machinery required for robust mitotic progression.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;17429401&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;145581&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;84966&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;143872&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5689&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;121512&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;84893&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;143098&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;90665&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;57178&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5687&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;84752&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;84519&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9367&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;57568&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;79690&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;196385&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10844&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;27158&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51807&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;144132&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5082&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;84628&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;89932&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;90161&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;114899&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;146310&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5709&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;80760&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;158521&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5707&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;161835&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;347088&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;143471&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5702&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5688&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;147323&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;143888&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;57586&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Jessica Robertson&#39;], &#39;exact_source&#39;: [&#39;Table 1&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div>
    </div>


<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>


    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
            integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
            crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"
            integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
            crossorigin="anonymous"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
            integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
            crossorigin="anonymous"></script>
    <script src="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.js"></script>

    <!-- DataTables, see: https://datatables.net/examples/styling/bootstrap4.html-->
    <script src="https://cdn.datatables.net/1.10.19/js/jquery.dataTables.min.js"></script>
    <script src="https://cdn.datatables.net/1.10.19/js/dataTables.bootstrap4.min.js"></script>

    <script src="https://d3js.org/d3.v5.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.js"></script>

    <script>
        $(function () {
            $('[data-toggle="tooltip"]').tooltip()
        })
    </script>
</body>
</html>