<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>GAZIN_EPIGENETIC_SILENCING_BY_KRAS</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            GAZIN_EPIGENETIC_SILENCING_BY_KRAS
        </div>
        <div class="card-body">
                <p>The conversion of a normal cell to a cancer cell occurs in several steps and typically involves the activation of oncogenes and the inactivation of tumour suppressor and pro-apoptotic genes. In many instances, inactivation of genes critical for cancer development occurs by epigenetic silencing, often involving hypermethylation of CpG-rich promoter regions. It remains to be determined whether silencing occurs by random acquisition of epigenetic marks that confer a selective growth advantage or through a specific pathway initiated by an oncogene. Here we perform a genome-wide RNA interference (RNAi) screen in K-ras-transformed NIH 3T3 cells and identify 28 genes required for Ras-mediated epigenetic silencing of the pro-apoptotic Fas gene. At least nine of these RESEs (Ras epigenetic silencing effectors), including the DNA methyltransferase DNMT1, are directly associated with specific regions of the Fas promoter in K-ras-transformed NIH 3T3 cells but not in untransformed NIH 3T3 cells. RNAi-mediated knockdown of any of the 28 RESEs results in failure to recruit DNMT1 to the Fas promoter, loss of Fas promoter hypermethylation, and derepression of Fas expression. Analysis of five other epigenetically repressed genes indicates that Ras directs the silencing of multiple unrelated genes through a largely common pathway. Last, we show that nine RESEs are required for anchorage-independent growth and tumorigenicity of K-ras-transformed NIH 3T3 cells; these nine genes have not previously been implicated in transformation by Ras. Our results show that Ras-mediated epigenetic silencing occurs through a specific, complex, pathway involving components that are required for maintenance of a fully transformed phenotype.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M1770">M1770</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:10090">NCBITaxon:10090</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:2185">ncbigene:2185</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9734">ncbigene:9734</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:283248">ncbigene:283248</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2146">ncbigene:2146</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4293">ncbigene:4293</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10361">ncbigene:10361</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:23741">ncbigene:23741</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:170591">ncbigene:170591</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:84444">ncbigene:84444</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:100532731">ncbigene:100532731</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:117608">ncbigene:117608</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9866">ncbigene:9866</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:150572">ncbigene:150572</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:25842">ncbigene:25842</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1869">ncbigene:1869</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5170">ncbigene:5170</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5594">ncbigene:5594</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4591">ncbigene:4591</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:55257">ncbigene:55257</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:51548">ncbigene:51548</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8726">ncbigene:8726</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1786">ncbigene:1786</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:29063">ncbigene:29063</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:11176">ncbigene:11176</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10664">ncbigene:10664</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M1770&#39;, name=&#39;GAZIN_EPIGENETIC_SILENCING_BY_KRAS&#39;), definition=&#39;The conversion of a normal cell to a cancer cell occurs in several steps and typically involves the activation of oncogenes and the inactivation of tumour suppressor and pro-apoptotic genes. In many instances, inactivation of genes critical for cancer development occurs by epigenetic silencing, often involving hypermethylation of CpG-rich promoter regions. It remains to be determined whether silencing occurs by random acquisition of epigenetic marks that confer a selective growth advantage or through a specific pathway initiated by an oncogene. Here we perform a genome-wide RNA interference (RNAi) screen in K-ras-transformed NIH 3T3 cells and identify 28 genes required for Ras-mediated epigenetic silencing of the pro-apoptotic Fas gene. At least nine of these RESEs (Ras epigenetic silencing effectors), including the DNA methyltransferase DNMT1, are directly associated with specific regions of the Fas promoter in K-ras-transformed NIH 3T3 cells but not in untransformed NIH 3T3 cells. RNAi-mediated knockdown of any of the 28 RESEs results in failure to recruit DNMT1 to the Fas promoter, loss of Fas promoter hypermethylation, and derepression of Fas expression. Analysis of five other epigenetically repressed genes indicates that Ras directs the silencing of multiple unrelated genes through a largely common pathway. Last, we show that nine RESEs are required for anchorage-independent growth and tumorigenicity of K-ras-transformed NIH 3T3 cells; these nine genes have not previously been implicated in transformation by Ras. Our results show that Ras-mediated epigenetic silencing occurs through a specific, complex, pathway involving components that are required for maintenance of a fully transformed phenotype.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;17960246&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;10090&#39;, name=&#39;Mus musculus&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2185&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9734&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;283248&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2146&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4293&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10361&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23741&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;170591&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;84444&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;100532731&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;117608&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9866&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;150572&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;25842&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1869&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5170&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5594&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4591&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55257&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51548&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8726&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1786&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;29063&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;11176&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10664&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Jessica Robertson&#39;], &#39;exact_source&#39;: [&#39;Table 1&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>