<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>LI_WILMS_TUMOR</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            LI_WILMS_TUMOR
        </div>
        <div class="card-body">
                <p>Wilms&#39; tumor (WT) has been considered a prototype for arrested cellular differentiation in cancer, but previous studies have relied on selected markers. We have now performed an unbiased survey of gene expression in WTs using oligonucleotide microarrays. Statistical criteria identified 357 genes as differentially expressed between WTs and fetal kidneys. This set contained 124 matches to genes on a microarray used by Stuart and colleagues (Stuart RO, Bush KT, Nigam SK: Changes in global gene expression patterns during development and maturation of the rat kidney. Proc Natl Acad Sci USA 2001, 98:5649-5654) to establish genes with stage-specific expression in the developing rat kidney. Mapping between the two data sets showed that WTs systematically overexpressed genes corresponding to the earliest stage of metanephric development, and underexpressed genes corresponding to later stages. Automated clustering identified a smaller group of 27 genes that were highly expressed in WTs compared to fetal kidney and heterologous tumor and normal tissues. This signature set was enriched in genes encoding transcription factors. Four of these, PAX2, EYA1, HBF2, and HOXA11, are essential for cell survival and proliferation in early metanephric development, whereas others, including SIX1, MOX1, and SALL2, are predicted to act at this stage. SIX1 and SALL2 proteins were expressed in the condensing mesenchyme in normal human fetal kidneys, but were absent (SIX1) or reduced (SALL2) in cells at other developmental stages. These data imply that the blastema in WTs has progressed to the committed stage in the mesenchymal-epithelial transition, where it is partially arrested in differentiation. The WT-signature set also contained the Wnt receptor FZD7, the tumor antigen PRAME, the imprinted gene NNAT and the metastasis-associated transcription factor E1AF.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M10773">M10773</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:4222">ncbigene:4222</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3625">ncbigene:3625</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10149">ncbigene:10149</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:23532">ncbigene:23532</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3207">ncbigene:3207</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:116372">ncbigene:116372</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10810">ncbigene:10810</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6297">ncbigene:6297</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3037">ncbigene:3037</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3205">ncbigene:3205</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8534">ncbigene:8534</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2118">ncbigene:2118</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4826">ncbigene:4826</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1280">ncbigene:1280</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1305">ncbigene:1305</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:341">ncbigene:341</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:91851">ncbigene:91851</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8091">ncbigene:8091</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4330">ncbigene:4330</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2737">ncbigene:2737</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8324">ncbigene:8324</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2138">ncbigene:2138</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6495">ncbigene:6495</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1620">ncbigene:1620</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2290">ncbigene:2290</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5076">ncbigene:5076</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1382">ncbigene:1382</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M10773&#39;, name=&#39;LI_WILMS_TUMOR&#39;), definition=&#34;Wilms&#39; tumor (WT) has been considered a prototype for arrested cellular differentiation in cancer, but previous studies have relied on selected markers. We have now performed an unbiased survey of gene expression in WTs using oligonucleotide microarrays. Statistical criteria identified 357 genes as differentially expressed between WTs and fetal kidneys. This set contained 124 matches to genes on a microarray used by Stuart and colleagues (Stuart RO, Bush KT, Nigam SK: Changes in global gene expression patterns during development and maturation of the rat kidney. Proc Natl Acad Sci USA 2001, 98:5649-5654) to establish genes with stage-specific expression in the developing rat kidney. Mapping between the two data sets showed that WTs systematically overexpressed genes corresponding to the earliest stage of metanephric development, and underexpressed genes corresponding to later stages. Automated clustering identified a smaller group of 27 genes that were highly expressed in WTs compared to fetal kidney and heterologous tumor and normal tissues. This signature set was enriched in genes encoding transcription factors. Four of these, PAX2, EYA1, HBF2, and HOXA11, are essential for cell survival and proliferation in early metanephric development, whereas others, including SIX1, MOX1, and SALL2, are predicted to act at this stage. SIX1 and SALL2 proteins were expressed in the condensing mesenchyme in normal human fetal kidneys, but were absent (SIX1) or reduced (SALL2) in cells at other developmental stages. These data imply that the blastema in WTs has progressed to the committed stage in the mesenchymal-epithelial transition, where it is partially arrested in differentiation. The WT-signature set also contained the Wnt receptor FZD7, the tumor antigen PRAME, the imprinted gene NNAT and the metastasis-associated transcription factor E1AF.&#34;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;12057921&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4222&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3625&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10149&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23532&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3207&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;116372&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10810&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6297&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3037&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3205&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8534&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2118&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4826&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1280&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1305&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;341&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;91851&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8091&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4330&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2737&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8324&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2138&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6495&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1620&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2290&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5076&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1382&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Arthur Liberzon&#39;], &#39;exact_source&#39;: [&#39;Table 1: set C&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>