<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>MARSHALL_VIRAL_INFECTION_RESPONSE_DN</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            MARSHALL_VIRAL_INFECTION_RESPONSE_DN
        </div>
        <div class="card-body">
                <p>The restriction of influenza A virus replication to mouse respiratory epithelium means that this host response is anatomically compartmentalized, on the one hand, to sites of T cell stimulation and proliferation in the secondary lymphoid tissue and, on the other hand, to the site of effector T cell function and pathology in the pneumonic lung. Thus, it is hardly surprising that virus-specific CD8(+) T cells recovered by bronchoalveolar lavage (BAL) from the infected respiratory tract seem more activated in terms of both cytolytic activity and cytokine production than those cells isolated from the spleen. The present analysis uses Affymetrix microarray technology to compare profiles of gene expression in these two lineage-related, yet anatomically separate, lymphocyte populations. Ninety differentially expressed genes were identified for influenza-specific CD8(+)D(b)NP(366)(+) T cells obtained directly ex vivo by BAL or spleen disruption, with nine genes being further analyzed by quantitative, real-time PCR at the population level. Integrin alphaE, for example, was shown by Affymetrix and real-time mRNA analyses and then by single-cell PCR and protein staining to be present at a much higher prevalence on the BAL CD8(+)D(b)NP(366)(+) set. The unpredicted finding, however, was that mRNA expression for 75% of the 90 genes was lower in T cells from the BAL than from the spleen. Apparently, the localization of virus-specific CD8(+) T cells to the site of virus-induced pathology is associated with a narrowing, or focusing, of gene expression that favors enhanced effector function in the damaged, high-antigen load environment of the pneumonic lung.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M1882">M1882</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:10090">NCBITaxon:10090</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:4792">ncbigene:4792</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10219">ncbigene:10219</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3015">ncbigene:3015</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10365">ncbigene:10365</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1164">ncbigene:1164</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8698">ncbigene:8698</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:924">ncbigene:924</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10140">ncbigene:10140</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3001">ncbigene:3001</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6446">ncbigene:6446</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6368">ncbigene:6368</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3002">ncbigene:3002</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:51274">ncbigene:51274</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6777">ncbigene:6777</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3695">ncbigene:3695</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7050">ncbigene:7050</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9124">ncbigene:9124</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6932">ncbigene:6932</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4793">ncbigene:4793</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9493">ncbigene:9493</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4062">ncbigene:4062</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:22821">ncbigene:22821</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4602">ncbigene:4602</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7294">ncbigene:7294</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5292">ncbigene:5292</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5971">ncbigene:5971</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5111">ncbigene:5111</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2553">ncbigene:2553</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1840">ncbigene:1840</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8807">ncbigene:8807</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M1882&#39;, name=&#39;MARSHALL_VIRAL_INFECTION_RESPONSE_DN&#39;), definition=&#39;The restriction of influenza A virus replication to mouse respiratory epithelium means that this host response is anatomically compartmentalized, on the one hand, to sites of T cell stimulation and proliferation in the secondary lymphoid tissue and, on the other hand, to the site of effector T cell function and pathology in the pneumonic lung. Thus, it is hardly surprising that virus-specific CD8(+) T cells recovered by bronchoalveolar lavage (BAL) from the infected respiratory tract seem more activated in terms of both cytolytic activity and cytokine production than those cells isolated from the spleen. The present analysis uses Affymetrix microarray technology to compare profiles of gene expression in these two lineage-related, yet anatomically separate, lymphocyte populations. Ninety differentially expressed genes were identified for influenza-specific CD8(+)D(b)NP(366)(+) T cells obtained directly ex vivo by BAL or spleen disruption, with nine genes being further analyzed by quantitative, real-time PCR at the population level. Integrin alphaE, for example, was shown by Affymetrix and real-time mRNA analyses and then by single-cell PCR and protein staining to be present at a much higher prevalence on the BAL CD8(+)D(b)NP(366)(+) set. The unpredicted finding, however, was that mRNA expression for 75% of the 90 genes was lower in T cells from the BAL than from the spleen. Apparently, the localization of virus-specific CD8(+) T cells to the site of virus-induced pathology is associated with a narrowing, or focusing, of gene expression that favors enhanced effector function in the damaged, high-antigen load environment of the pneumonic lung.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;15831586&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;10090&#39;, name=&#39;Mus musculus&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4792&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10219&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3015&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10365&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1164&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8698&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;924&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10140&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3001&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6446&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6368&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3002&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51274&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6777&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3695&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7050&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9124&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6932&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4793&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9493&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4062&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;22821&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4602&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7294&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5292&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5971&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5111&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2553&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1840&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8807&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Arthur Liberzon&#39;], &#39;exact_source&#39;: [&#39;Table 1&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>