<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>HOLLERN_MICROACINAR_BREAST_TUMOR_DN</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            HOLLERN_MICROACINAR_BREAST_TUMOR_DN
        </div>
        <div class="card-body">
                <p>Human breast cancer has been characterized by extensive transcriptional heterogeneity, with dominant patterns reflected in the intrinsic subtypes. Mouse models of breast cancer also have heterogeneous transcriptomes and we noted that specific histological subtypes were associated with particular subsets. We hypothesized that unique sets of genes define each tumor histological type across mouse models of breast cancer. Using mouse models that contained both gene expression data and expert pathologist classification of tumor histology on a sample by sample basis, we predicted and validated gene expression signatures for Papillary, EMT, Microacinar and other histological subtypes. These signatures predict known histological events across murine breast cancer models and identify counterparts of mouse mammary tumor types in subtypes of human breast cancer. Importantly, the EMT, Adenomyoepithelial, and Solid signatures were predictive of clinical events in human breast cancer. In addition, a pan-cancer comparison revealed that the histological signatures were active in a variety of human cancers such as lung, oral, and esophageal squamous tumors. Finally, the differentiation status and transcriptional activity implicit within these signatures was identified. These data reveal that within tumor histology groups are unique gene expression profiles of differentiation and pathway activity that stretch well beyond the transgenic initiating events and that have clear applicability to human cancers. As a result, our work provides a predictive resource and insights into possible mechanisms that govern tumor heterogeneity.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M717">M717</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:10090">NCBITaxon:10090</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:9536">ncbigene:9536</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:26872">ncbigene:26872</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7127">ncbigene:7127</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:347733">ncbigene:347733</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:100131755">ncbigene:100131755</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3248">ncbigene:3248</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:81029">ncbigene:81029</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:11096">ncbigene:11096</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3371">ncbigene:3371</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:55332">ncbigene:55332</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:11346">ncbigene:11346</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5802">ncbigene:5802</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:57007">ncbigene:57007</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6480">ncbigene:6480</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6566">ncbigene:6566</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M717&#39;, name=&#39;HOLLERN_MICROACINAR_BREAST_TUMOR_DN&#39;), definition=&#39;Human breast cancer has been characterized by extensive transcriptional heterogeneity, with dominant patterns reflected in the intrinsic subtypes. Mouse models of breast cancer also have heterogeneous transcriptomes and we noted that specific histological subtypes were associated with particular subsets. We hypothesized that unique sets of genes define each tumor histological type across mouse models of breast cancer. Using mouse models that contained both gene expression data and expert pathologist classification of tumor histology on a sample by sample basis, we predicted and validated gene expression signatures for Papillary, EMT, Microacinar and other histological subtypes. These signatures predict known histological events across murine breast cancer models and identify counterparts of mouse mammary tumor types in subtypes of human breast cancer. Importantly, the EMT, Adenomyoepithelial, and Solid signatures were predictive of clinical events in human breast cancer. In addition, a pan-cancer comparison revealed that the histological signatures were active in a variety of human cancers such as lung, oral, and esophageal squamous tumors. Finally, the differentiation status and transcriptional activity implicit within these signatures was identified. These data reveal that within tumor histology groups are unique gene expression profiles of differentiation and pathway activity that stretch well beyond the transgenic initiating events and that have clear applicability to human cancers. As a result, our work provides a predictive resource and insights into possible mechanisms that govern tumor heterogeneity.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;29346386&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;10090&#39;, name=&#39;Mus musculus&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9536&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;26872&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7127&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;347733&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;100131755&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3248&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;81029&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;11096&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3371&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55332&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;11346&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5802&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;57007&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6480&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6566&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Daniel Hollern&#39;], &#39;exact_source&#39;: [&#39;File S1: Downregulated_In_Microacinar&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>