<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>NGO_MALIGNANT_GLIOMA_1P_LOH</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            NGO_MALIGNANT_GLIOMA_1P_LOH
        </div>
        <div class="card-body">
                <p>BACKGROUND: Malignant gliomas are generally resistant to all conventional therapies. Notable exceptions are anaplastic oligodendrogliomas with loss of heterozygosity on chromosome 1p (1p+/-). Patients with 1p+/- anaplastic oligodendroglioma frequently respond to procarbazine, 1-(2-chloroethyl)-3-cyclohexyl-l-nitrosourea, and vincristine. Because the underlying biologic basis for this clinical finding is unclear, we evaluated differentially expressed 1p-encoded proteins in 1p+/- and 1p+/+ malignant glioma cell lines and then examined whether their expression was associated with outcome of patients with anaplastic oligodendroglioma. METHODS: We used a comparative proteomic screen of A172 (1p+/-) and U251 (1p+/+) malignant glioma cell lines to identify differentially expressed 1p-encoded proteins, including stathmin, a microtubule-associated protein. 1p+/- and 1p+/+ anaplastic oligodendroglioma specimens from 24 patients were assessed for stathmin expression by immunohistochemistry. The relationship between stathmin expression and clinical outcome was assessed with Kaplan-Meier analyses. RNA inhibition and cDNA transfection experiments tested effects of stathmin under- and overexpression, respectively, on the in vitro and in vivo resistance of malignant glioma cells to treatment with nitrosourea. For in vivo resistance studies, 36 mice with intracranial and 16 mice with subcutaneous xenograft tumor implants were used (one tumor per mouse). Flow cytometry was used for cell cycle analysis. Immunoblotting was used to assess protein expression. All statistical tests were two-sided. RESULTS: Decreased stathmin expression in tumors was statistically significantly associated with loss of heterozygosity in 1p (P&lt;.001) and increased recurrence-free survival (P&lt;.001). The median recurrence-free survival times for patients with tumors expressing low, intermediate, or high stathmin levels were 45 months (95% confidence interval [CI] = 0 to 90 months), 17 months (95% CI = 10.6 to 23.4 months), and 6 months (95% CI = 1.7 to 10.3 months), respectively. Expression of stathmin was inversely associated with overall survival of nitrosourea-treated mice carrying xenograft tumors. Median survival of mice with stathmin+/- tumors was 95 days (95% CI = 68.7 to 121.3 days) and that of mice with stathmin+/+ tumors was 64 days (95% CI = 58.2 to 69.8 days) (difference = 31 days, 95% CI = 4.1 to 57.9 days; P&lt;.001, log-rank test). Nitrosoureas induced mitotic arrest in malignant glioma cells, and this effect was greater in cells with decreased stathmin expression. CONCLUSIONS: Loss of heterozygosity for the stathmin gene may be associated with improved outcomes of patients with 1p+/- anaplastic oligodendroglioma tumors.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M19965">M19965</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:7170">ncbigene:7170</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3921">ncbigene:3921</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3309">ncbigene:3309</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:302">ncbigene:302</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7171">ncbigene:7171</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7431">ncbigene:7431</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2023">ncbigene:2023</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3190">ncbigene:3190</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1892">ncbigene:1892</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4735">ncbigene:4735</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5230">ncbigene:5230</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2597">ncbigene:2597</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3925">ncbigene:3925</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5034">ncbigene:5034</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:11315">ncbigene:11315</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M19965&#39;, name=&#39;NGO_MALIGNANT_GLIOMA_1P_LOH&#39;), definition=&#39;BACKGROUND: Malignant gliomas are generally resistant to all conventional therapies. Notable exceptions are anaplastic oligodendrogliomas with loss of heterozygosity on chromosome 1p (1p+/-). Patients with 1p+/- anaplastic oligodendroglioma frequently respond to procarbazine, 1-(2-chloroethyl)-3-cyclohexyl-l-nitrosourea, and vincristine. Because the underlying biologic basis for this clinical finding is unclear, we evaluated differentially expressed 1p-encoded proteins in 1p+/- and 1p+/+ malignant glioma cell lines and then examined whether their expression was associated with outcome of patients with anaplastic oligodendroglioma. METHODS: We used a comparative proteomic screen of A172 (1p+/-) and U251 (1p+/+) malignant glioma cell lines to identify differentially expressed 1p-encoded proteins, including stathmin, a microtubule-associated protein. 1p+/- and 1p+/+ anaplastic oligodendroglioma specimens from 24 patients were assessed for stathmin expression by immunohistochemistry. The relationship between stathmin expression and clinical outcome was assessed with Kaplan-Meier analyses. RNA inhibition and cDNA transfection experiments tested effects of stathmin under- and overexpression, respectively, on the in vitro and in vivo resistance of malignant glioma cells to treatment with nitrosourea. For in vivo resistance studies, 36 mice with intracranial and 16 mice with subcutaneous xenograft tumor implants were used (one tumor per mouse). Flow cytometry was used for cell cycle analysis. Immunoblotting was used to assess protein expression. All statistical tests were two-sided. RESULTS: Decreased stathmin expression in tumors was statistically significantly associated with loss of heterozygosity in 1p (P&lt;.001) and increased recurrence-free survival (P&lt;.001). The median recurrence-free survival times for patients with tumors expressing low, intermediate, or high stathmin levels were 45 months (95% confidence interval [CI] = 0 to 90 months), 17 months (95% CI = 10.6 to 23.4 months), and 6 months (95% CI = 1.7 to 10.3 months), respectively. Expression of stathmin was inversely associated with overall survival of nitrosourea-treated mice carrying xenograft tumors. Median survival of mice with stathmin+/- tumors was 95 days (95% CI = 68.7 to 121.3 days) and that of mice with stathmin+/+ tumors was 64 days (95% CI = 58.2 to 69.8 days) (difference = 31 days, 95% CI = 4.1 to 57.9 days; P&lt;.001, log-rank test). Nitrosoureas induced mitotic arrest in malignant glioma cells, and this effect was greater in cells with decreased stathmin expression. CONCLUSIONS: Loss of heterozygosity for the stathmin gene may be associated with improved outcomes of patients with 1p+/- anaplastic oligodendroglioma tumors.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;17440165&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7170&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3921&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3309&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;302&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7171&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7431&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2023&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3190&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1892&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4735&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5230&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2597&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3925&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5034&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;11315&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Jessica Robertson&#39;], &#39;exact_source&#39;: [&#39;Table 1&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>