<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>MASRI_RESISTANCE_TO_TAMOXIFEN_AND_AROMATASE_INHIBITORS_UP</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            MASRI_RESISTANCE_TO_TAMOXIFEN_AND_AROMATASE_INHIBITORS_UP
        </div>
        <div class="card-body">
                <p>Acquired resistance to either tamoxifen or aromatase inhibitors (AI) develops after prolonged treatment in a majority of hormone-responsive breast cancers. In an attempt to further elucidate mechanisms of acquired resistance to AIs, MCF-7aro cells resistant to letrozole (T+LET R), anastrozole (T+ANA R), and exemestane (T+EXE R), as well as long-term estrogen deprived (LTEDaro) and tamoxifen-resistant (T+TAM R) lines were generated. This is the first complete panel of endocrine therapy-resistant cell lines, which were generated as multiple independent biological replicates for unbiased genome-wide analysis using affymetrix microarrays. Although similarities are apparent, microarray results clearly show gene signatures unique to AI-resistance were inherently different from LTEDaro and T+TAM R gene expression profiles. Based on hierarchical clustering, unique estrogen-responsive gene signatures vary depending on cell line, with some genes up-regulated in all lines versus other genes up-regulated only in the AI-resistant lines. Characterization of these resistant lines showed that LTEDaro, T+LET R, and T+ANA R cells contained a constitutively active estrogen receptor (ER)alpha that does not require estrogen for activation. This ligand-independent activation of ER was not observed in the parental cells, as well as T+EXE R and T+TAM R cells. Further characterization of these resistant lines was performed using cell cycle analysis, immunofluorescence experiments to visualize ER subcellular localization, as well as cross-resistance studies to determine second-line inhibitor response. Using this well-defined model system, our studies provide important information regarding differences in resistance mechanisms to AIs, TAM, and LTEDaro, which are critical in overcoming resistance when treating hormone-responsive breast cancers.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M9611">M9611</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:9687">ncbigene:9687</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7031">ncbigene:7031</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4603">ncbigene:4603</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4886">ncbigene:4886</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5241">ncbigene:5241</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:54898">ncbigene:54898</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:26033">ncbigene:26033</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2273">ncbigene:2273</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:11012">ncbigene:11012</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:23213">ncbigene:23213</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:196475">ncbigene:196475</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:374">ncbigene:374</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10551">ncbigene:10551</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4256">ncbigene:4256</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5174">ncbigene:5174</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2697">ncbigene:2697</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:25840">ncbigene:25840</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:56603">ncbigene:56603</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1756">ncbigene:1756</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:26154">ncbigene:26154</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M9611&#39;, name=&#39;MASRI_RESISTANCE_TO_TAMOXIFEN_AND_AROMATASE_INHIBITORS_UP&#39;), definition=&#39;Acquired resistance to either tamoxifen or aromatase inhibitors (AI) develops after prolonged treatment in a majority of hormone-responsive breast cancers. In an attempt to further elucidate mechanisms of acquired resistance to AIs, MCF-7aro cells resistant to letrozole (T+LET R), anastrozole (T+ANA R), and exemestane (T+EXE R), as well as long-term estrogen deprived (LTEDaro) and tamoxifen-resistant (T+TAM R) lines were generated. This is the first complete panel of endocrine therapy-resistant cell lines, which were generated as multiple independent biological replicates for unbiased genome-wide analysis using affymetrix microarrays. Although similarities are apparent, microarray results clearly show gene signatures unique to AI-resistance were inherently different from LTEDaro and T+TAM R gene expression profiles. Based on hierarchical clustering, unique estrogen-responsive gene signatures vary depending on cell line, with some genes up-regulated in all lines versus other genes up-regulated only in the AI-resistant lines. Characterization of these resistant lines showed that LTEDaro, T+LET R, and T+ANA R cells contained a constitutively active estrogen receptor (ER)alpha that does not require estrogen for activation. This ligand-independent activation of ER was not observed in the parental cells, as well as T+EXE R and T+TAM R cells. Further characterization of these resistant lines was performed using cell cycle analysis, immunofluorescence experiments to visualize ER subcellular localization, as well as cross-resistance studies to determine second-line inhibitor response. Using this well-defined model system, our studies provide important information regarding differences in resistance mechanisms to AIs, TAM, and LTEDaro, which are critical in overcoming resistance when treating hormone-responsive breast cancers.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;18559539&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9687&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7031&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4603&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4886&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5241&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;54898&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;26033&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2273&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;11012&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23213&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;196475&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;374&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10551&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4256&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5174&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2697&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;25840&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;56603&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1756&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;26154&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Leona Saunders&#39;], &#39;exact_source&#39;: [&#39;Table 3S&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>