<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>HO_LIVER_CANCER_VASCULAR_INVASION</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            HO_LIVER_CANCER_VASCULAR_INVASION
        </div>
        <div class="card-body">
                <p>BACKGROUND: Recurrence after hepatocellular carcinoma (HCC) resection is the major obstacle to improved survival. The presence of vascular invasion (VI) in pathology specimens is a well-known unfavorable prognostic factor for HCC recurrence. Though some VI-related genes have been reported, their association with recurrence-free survival is not known. We hypothesized that a gene expression profile for VI can predict the recurrence of HCC after liver resection. METHODS: Eighteen patients receiving complete HCC resection were included as a training group. Genome-wide gene expression profile was obtained for each tumor using a microarray technique. Datasets were subjected to clustering analysis supervised by the presence or absence of VI to obtain 14 discriminative genes. We then applied those genes to execute pattern recognition using the k-Nearest Neighbor (KNN) classification method, and the best model for this VI gene signature to predict recurrence-free survival in the training group was obtained. The resulting model was then tested in an independent test group of 35 patients. RESULTS: A 14-gene profile was extracted which could accurately separate ten patients with VI and eight patients without VI in the training group. In the test group, significant difference in disease-free survival was found between patients predicted to have and not to have recurrence (P = .02823). In patients with stage_I disease, this model can also predict outcomes (P = .000205). CONCLUSIONS: Using the 14-gene expression profile extracted from microarrays based on the presence of VI can effectively predict recurrence after HCC resection. This approach might facilitate personalized medicine for HCC patients after surgical resection.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M8773">M8773</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:1441">ncbigene:1441</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4968">ncbigene:4968</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:57592">ncbigene:57592</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:54734">ncbigene:54734</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:81603">ncbigene:81603</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:760">ncbigene:760</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4108">ncbigene:4108</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6261">ncbigene:6261</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6875">ncbigene:6875</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:26015">ncbigene:26015</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:272">ncbigene:272</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9690">ncbigene:9690</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5269">ncbigene:5269</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M8773&#39;, name=&#39;HO_LIVER_CANCER_VASCULAR_INVASION&#39;), definition=&#39;BACKGROUND: Recurrence after hepatocellular carcinoma (HCC) resection is the major obstacle to improved survival. The presence of vascular invasion (VI) in pathology specimens is a well-known unfavorable prognostic factor for HCC recurrence. Though some VI-related genes have been reported, their association with recurrence-free survival is not known. We hypothesized that a gene expression profile for VI can predict the recurrence of HCC after liver resection. METHODS: Eighteen patients receiving complete HCC resection were included as a training group. Genome-wide gene expression profile was obtained for each tumor using a microarray technique. Datasets were subjected to clustering analysis supervised by the presence or absence of VI to obtain 14 discriminative genes. We then applied those genes to execute pattern recognition using the k-Nearest Neighbor (KNN) classification method, and the best model for this VI gene signature to predict recurrence-free survival in the training group was obtained. The resulting model was then tested in an independent test group of 35 patients. RESULTS: A 14-gene profile was extracted which could accurately separate ten patients with VI and eight patients without VI in the training group. In the test group, significant difference in disease-free survival was found between patients predicted to have and not to have recurrence (P = .02823). In patients with stage_I disease, this model can also predict outcomes (P = .000205). CONCLUSIONS: Using the 14-gene expression profile extracted from microarrays based on the presence of VI can effectively predict recurrence after HCC resection. This approach might facilitate personalized medicine for HCC patients after surgical resection.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;17009164&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1441&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4968&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;57592&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;54734&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;81603&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;760&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4108&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6261&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6875&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;26015&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;272&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9690&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5269&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Yujin Hoshida&#39;], &#39;exact_source&#39;: [&#39;Fig. 2&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>