<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>MEINHOLD_OVARIAN_CANCER_LOW_GRADE_DN</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            MEINHOLD_OVARIAN_CANCER_LOW_GRADE_DN
        </div>
        <div class="card-body">
                <p>Profiles of gene transcription have begun to delineate the molecular basis of ovarian cancer, including distinctions between carcinomas of differing histology, tumor progression and patient outcome. However, the similarities and differences among the most commonly diagnosed noninvasive borderline (low malignant potential, LMP) lesions and invasive serous carcinomas of varying grade (G1, G2 and G3) have not yet been explored. Here, we used oligonucleotide arrays to profile the expression of 12,500 genes in a series of 57 predominantly stage III serous ovarian adenocarcinomas from 52 patients, eight with borderline tumors and 44 with adenocarcinomas of varying grade. Unsupervised and supervised analyses showed that LMP lesions were distinct from high-grade serous adenocarcinomas, as might be expected; however, well-differentiated (G1) invasive adenocarcinomas showed a strikingly similar profile to LMP tumors as compared to cancers with moderate (G2) or poor (G3) cellular differentiation, which were also highly similar. Comparative genomic hybridization of an independent cohort of five LMP and 63 invasive carcinomas of varying grade demonstrated LMP and G1 were again similar, exhibiting significantly less chromosomal aberration than G2/G3 carcinomas. A majority of LMP and G1 tumors were characterized by high levels of p21/WAF1, with concomitant expression of cell growth suppressors, gadd34 and BTG-2. In contrast, G2/G3 cancers were characterized by the expression of genes associated with the cell cycle and by STAT-1-, STAT-3/JAK-1/2-induced gene expression. The distinction between the LMP-G1 and G2-G3 groups of tumors was highly correlated to patient outcome (chi(2) for equivalence of death rates=7.681189; P=0.0056, log-rank test). Our results are consistent with the recent demonstration of a poor differentiation molecular &#39;meta-signature&#39; in human cancer, and underscore a number of cell-cycle- and STAT-associated targets that may prove useful as points of therapeutic intervention for those patients with aggressive disease.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M867">M867</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:1072">ncbigene:1072</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6009">ncbigene:6009</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5096">ncbigene:5096</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9133">ncbigene:9133</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8314">ncbigene:8314</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6772">ncbigene:6772</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4690">ncbigene:4690</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10376">ncbigene:10376</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9928">ncbigene:9928</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8073">ncbigene:8073</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:65005">ncbigene:65005</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10899">ncbigene:10899</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4175">ncbigene:4175</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2194">ncbigene:2194</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6628">ncbigene:6628</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:56910">ncbigene:56910</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7405">ncbigene:7405</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5885">ncbigene:5885</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:22974">ncbigene:22974</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M867&#39;, name=&#39;MEINHOLD_OVARIAN_CANCER_LOW_GRADE_DN&#39;), definition=&#34;Profiles of gene transcription have begun to delineate the molecular basis of ovarian cancer, including distinctions between carcinomas of differing histology, tumor progression and patient outcome. However, the similarities and differences among the most commonly diagnosed noninvasive borderline (low malignant potential, LMP) lesions and invasive serous carcinomas of varying grade (G1, G2 and G3) have not yet been explored. Here, we used oligonucleotide arrays to profile the expression of 12,500 genes in a series of 57 predominantly stage III serous ovarian adenocarcinomas from 52 patients, eight with borderline tumors and 44 with adenocarcinomas of varying grade. Unsupervised and supervised analyses showed that LMP lesions were distinct from high-grade serous adenocarcinomas, as might be expected; however, well-differentiated (G1) invasive adenocarcinomas showed a strikingly similar profile to LMP tumors as compared to cancers with moderate (G2) or poor (G3) cellular differentiation, which were also highly similar. Comparative genomic hybridization of an independent cohort of five LMP and 63 invasive carcinomas of varying grade demonstrated LMP and G1 were again similar, exhibiting significantly less chromosomal aberration than G2/G3 carcinomas. A majority of LMP and G1 tumors were characterized by high levels of p21/WAF1, with concomitant expression of cell growth suppressors, gadd34 and BTG-2. In contrast, G2/G3 cancers were characterized by the expression of genes associated with the cell cycle and by STAT-1-, STAT-3/JAK-1/2-induced gene expression. The distinction between the LMP-G1 and G2-G3 groups of tumors was highly correlated to patient outcome (chi(2) for equivalence of death rates=7.681189; P=0.0056, log-rank test). Our results are consistent with the recent demonstration of a poor differentiation molecular &#39;meta-signature&#39; in human cancer, and underscore a number of cell-cycle- and STAT-associated targets that may prove useful as points of therapeutic intervention for those patients with aggressive disease.&#34;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;15558012&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1072&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6009&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5096&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9133&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8314&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6772&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4690&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10376&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9928&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8073&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;65005&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10899&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4175&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2194&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6628&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;56910&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7405&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5885&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;22974&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Arthur Liberzon&#39;], &#39;exact_source&#39;: [&#39;Table 2: Higher relative expression in G2-G3&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>