<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>GRATIAS_RETINOBLASTOMA_16Q24</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            GRATIAS_RETINOBLASTOMA_16Q24
        </div>
        <div class="card-body">
                <p>In addition to RB1 gene mutations, retinoblastomas frequently show gains of 1q and 6p and losses of 16q. To identify suppressor genes on 16q, we analyzed 22 short tandem repeat loci in 58 patients with known RB1 mutations. A subset of tumors was also investigated by conventional and matrix comparative genomic hybridization. In 40 of 58 (69%) tumors, we found no loss of heterozygosity (LOH) at any 16q marker. LOH was detected in 18 of 58 (31%) tumors, including five with allelic imbalance at some markers. In one tumor LOH was only observed at 16q24. As the parental origin of allele loss was unbiased, an imprinted locus is unlikely to be involved. Analysis of gene expression by microarray hybridization and quantitative RT real-time PCR did not identify a candidate suppressor in 16q24. Cadherin 13 (CDH13), CBFA2T3, and WFDC1, which are candidate suppressors in other tumor entities with 16q24 loss, did not show loss of expression. In addition, mutation and methylation analysis showed no somatic alteration of CDH13. Results in all tumors with chromosome 16 alterations define a single minimal deleted region of 5.7 Mb in the telomeric part of 16q24 with the centromeric boundary defined by retention of heterozygosity for a single nucleotide variant in exon 10 of CDH13 (Mb 82.7). Interestingly, clinical presentation of tumors with and without 16q alterations was distinct. Specifically, almost all retinoblastomas with 16q24 loss showed diffuse intraocular seeding. This suggests that genetic alterations in the minimal deleted region are associated with impaired cell-to-cell adhesion.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M8458">M8458</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:79791">ncbigene:79791</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2588">ncbigene:2588</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1327">ncbigene:1327</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8720">ncbigene:8720</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:57707">ncbigene:57707</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:51693">ncbigene:51693</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9100">ncbigene:9100</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:29123">ncbigene:29123</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:22980">ncbigene:22980</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5119">ncbigene:5119</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:81631">ncbigene:81631</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:23174">ncbigene:23174</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:353">ncbigene:353</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8140">ncbigene:8140</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9764">ncbigene:9764</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10328">ncbigene:10328</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:55625">ncbigene:55625</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6687">ncbigene:6687</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M8458&#39;, name=&#39;GRATIAS_RETINOBLASTOMA_16Q24&#39;), definition=&#39;In addition to RB1 gene mutations, retinoblastomas frequently show gains of 1q and 6p and losses of 16q. To identify suppressor genes on 16q, we analyzed 22 short tandem repeat loci in 58 patients with known RB1 mutations. A subset of tumors was also investigated by conventional and matrix comparative genomic hybridization. In 40 of 58 (69%) tumors, we found no loss of heterozygosity (LOH) at any 16q marker. LOH was detected in 18 of 58 (31%) tumors, including five with allelic imbalance at some markers. In one tumor LOH was only observed at 16q24. As the parental origin of allele loss was unbiased, an imprinted locus is unlikely to be involved. Analysis of gene expression by microarray hybridization and quantitative RT real-time PCR did not identify a candidate suppressor in 16q24. Cadherin 13 (CDH13), CBFA2T3, and WFDC1, which are candidate suppressors in other tumor entities with 16q24 loss, did not show loss of expression. In addition, mutation and methylation analysis showed no somatic alteration of CDH13. Results in all tumors with chromosome 16 alterations define a single minimal deleted region of 5.7 Mb in the telomeric part of 16q24 with the centromeric boundary defined by retention of heterozygosity for a single nucleotide variant in exon 10 of CDH13 (Mb 82.7). Interestingly, clinical presentation of tumors with and without 16q alterations was distinct. Specifically, almost all retinoblastomas with 16q24 loss showed diffuse intraocular seeding. This suggests that genetic alterations in the minimal deleted region are associated with impaired cell-to-cell adhesion.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;17210724&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;79791&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2588&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1327&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8720&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;57707&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51693&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9100&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;29123&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;22980&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5119&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;81631&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23174&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;353&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8140&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9764&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10328&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55625&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6687&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Jessica Robertson&#39;], &#39;exact_source&#39;: [&#39;Table 1&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>