<!DOCTYPE html>
<html>
<head>
            <meta charset="utf-8">
            <meta name="viewport" content="width=device-width, initial-scale=1.0">
            <!-- Bootstrap -->
            <link
                    rel="stylesheet"
                    href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
                    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
                    crossorigin="anonymous"
            />
            <link
                    rel="stylesheet"
                    href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"
            />
            <link
                    rel="stylesheet"
                    href="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.css"
            />
            <!-- DataTables, see: https://datatables.net/examples/styling/bootstrap4.html-->
            <link
                    rel="stylesheet"
                    href="https://cdn.datatables.net/1.10.19/css/dataTables.bootstrap4.min.css"
            />
            <style type="text/css">
                h2 {
                    margin-top: 35px;
                }

                html, body {
                    height: 100%;
                }

                body {
                    display: flex;
                    flex-flow: column;
                }

                .footer {
                    margin-top: auto;
                    padding-top: 1em;
                    background-color: #f5f5f5;
                }

                nav[data-toggle="toc"] {
                    top: 42px;
                }
            </style>

        <script src="https://kit.fontawesome.com/4c86883252.js" crossorigin="anonymous"></script>
        <title>YAMASHITA_LIVER_CANCER_WITH_EPCAM_DN</title>
</head>

<body data-spy="scroll" data-target="#toc">
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>


    <div class="container" style="margin-top: 50px; margin-bottom: 50px">
        <div class="card">
        <div class="card-header">
            YAMASHITA_LIVER_CANCER_WITH_EPCAM_DN
        </div>
        <div class="card-body">
            
                <p>The heterogeneous nature of hepatocellular carcinoma (HCC) and the lack of appropriate biomarkers have hampered patient prognosis and treatment stratification. Recently, we have identified that a hepatic stem cell marker, epithelial cell adhesion molecule (EpCAM), may serve as an early biomarker of HCC because its expression is highly elevated in premalignant hepatic tissues and in a subset of HCC. In this study, we aimed to identify novel HCC subtypes that resemble certain stages of liver lineages by searching for EpCAM-coexpressed genes. A unique signature of EpCAM-positive HCCs was identified by cDNA microarray analysis of 40 HCC cases and validated by oligonucleotide microarray analysis of 238 independent HCC cases, which was further confirmed by immunohistochemical analysis of an additional 101 HCC cases. EpCAM-positive HCC displayed a distinct molecular signature with features of hepatic progenitor cells including the presence of known stem/progenitor markers such as cytokeratin 19, c-Kit, EpCAM, and activated Wnt-beta-catenin signaling, whereas EpCAM-negative HCC displayed genes with features of mature hepatocytes. Moreover, EpCAM-positive and EpCAM-negative HCC could be further subclassified into four groups with prognostic implication by determining the level of alpha-fetoprotein (AFP). These four subtypes displayed distinct gene expression patterns with features resembling certain stages of hepatic lineages. Taken together, we proposed an easy classification system defined by EpCAM and AFP to reveal HCC subtypes similar to hepatic cell maturation lineages, which may enable prognostic stratification and assessment of HCC patients with adjuvant therapy and provide new insights into the potential cellular origin of HCC and its activated molecular pathways.</p>
            
            <dl>
                <dt>Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M9914">M9914</a></dd>
                
            </dl>


            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M9914&#39;, name=&#39;YAMASHITA_LIVER_CANCER_WITH_EPCAM_DN&#39;), definition=&#39;The heterogeneous nature of hepatocellular carcinoma (HCC) and the lack of appropriate biomarkers have hampered patient prognosis and treatment stratification. Recently, we have identified that a hepatic stem cell marker, epithelial cell adhesion molecule (EpCAM), may serve as an early biomarker of HCC because its expression is highly elevated in premalignant hepatic tissues and in a subset of HCC. In this study, we aimed to identify novel HCC subtypes that resemble certain stages of liver lineages by searching for EpCAM-coexpressed genes. A unique signature of EpCAM-positive HCCs was identified by cDNA microarray analysis of 40 HCC cases and validated by oligonucleotide microarray analysis of 238 independent HCC cases, which was further confirmed by immunohistochemical analysis of an additional 101 HCC cases. EpCAM-positive HCC displayed a distinct molecular signature with features of hepatic progenitor cells including the presence of known stem/progenitor markers such as cytokeratin 19, c-Kit, EpCAM, and activated Wnt-beta-catenin signaling, whereas EpCAM-negative HCC displayed genes with features of mature hepatocytes. Moreover, EpCAM-positive and EpCAM-negative HCC could be further subclassified into four groups with prognostic implication by determining the level of alpha-fetoprotein (AFP). These four subtypes displayed distinct gene expression patterns with features resembling certain stages of hepatic lineages. Taken together, we proposed an easy classification system defined by EpCAM and AFP to reveal HCC subtypes similar to hepatic cell maturation lineages, which may enable prognostic stratification and assessment of HCC patients with adjuvant therapy and provide new insights into the potential cellular origin of HCC and its activated molecular pathways.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;18316609&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7364&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;462&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51540&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2940&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6495&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1066&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;123876&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10840&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;341&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3158&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8630&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10196&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10720&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3273&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2939&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55268&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;720&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Yujin Hoshida&#39;], &#39;exact_source&#39;: [&#39;Fig.1A&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div>
    </div>


<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>


    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
            integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
            crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"
            integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
            crossorigin="anonymous"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
            integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
            crossorigin="anonymous"></script>
    <script src="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.js"></script>

    <!-- DataTables, see: https://datatables.net/examples/styling/bootstrap4.html-->
    <script src="https://cdn.datatables.net/1.10.19/js/jquery.dataTables.min.js"></script>
    <script src="https://cdn.datatables.net/1.10.19/js/dataTables.bootstrap4.min.js"></script>

    <script src="https://d3js.org/d3.v5.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.js"></script>

    <script>
        $(function () {
            $('[data-toggle="tooltip"]').tooltip()
        })
    </script>
</body>
</html>