<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>MUNSHI_MULTIPLE_MYELOMA_DN</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            MUNSHI_MULTIPLE_MYELOMA_DN
        </div>
        <div class="card-body">
                <p>Genetic heterogeneity between individuals confounds the comparison of gene profiling of multiple myeloma (MM) cells versus normal plasma cells (PCs). To overcome this barrier, we compared the gene expression profile of CD138+ MM cells from a patient bone marrow (BM) sample with CD138+ PCs from a genetically identical twin BM sample using microarray profiling. Two hundred and ninety-six genes were up-regulated and 103 genes were down-regulated at least 2-fold in MM cells versus normal twin PCs. Highly expressed genes in MM cells included cell survival pathway genes such as mcl-1, dad-1, caspase 8, and FADD-like apoptosis regulator (FLIP); oncogenes/transcriptional factors such as Jun-D, Xbp-1, calmodulin, Calnexin, and FGFR-3; stress response and ubiquitin/proteasome pathway-related genes and various ribosomal genes reflecting increased metabolic and translational activity. Genes that were down-regulated in MM cells versus healthy twin PCs included RAD51, killer cell immunoglobulin-like receptor protein, and apoptotic protease activating factor. Microarray results were further confirmed by Western blot analyses, immunohistochemistry, fluorescent in situ hybridization (FISH), and functional assays of telomerase activity and bone marrow angiogenesis. This molecular profiling provides potential insights into mechanisms of malignant transformation in MM. For example, FGFR3, xbp-1, and both mcl-1 and dad-1 may mediate transformation, differentiation, and survival, respectively, and may have clinical implications. By identifying genes uniquely altered in MM cells compared with normal PCs in an identical genotypic background, the current study provides the framework to identify novel therapeutic targets.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M3900">M3900</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:10791">ncbigene:10791</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5888">ncbigene:5888</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3805">ncbigene:3805</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6597">ncbigene:6597</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3297">ncbigene:3297</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1736">ncbigene:1736</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7345">ncbigene:7345</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6498">ncbigene:6498</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10641">ncbigene:10641</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4848">ncbigene:4848</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M3900&#39;, name=&#39;MUNSHI_MULTIPLE_MYELOMA_DN&#39;), definition=&#39;Genetic heterogeneity between individuals confounds the comparison of gene profiling of multiple myeloma (MM) cells versus normal plasma cells (PCs). To overcome this barrier, we compared the gene expression profile of CD138+ MM cells from a patient bone marrow (BM) sample with CD138+ PCs from a genetically identical twin BM sample using microarray profiling. Two hundred and ninety-six genes were up-regulated and 103 genes were down-regulated at least 2-fold in MM cells versus normal twin PCs. Highly expressed genes in MM cells included cell survival pathway genes such as mcl-1, dad-1, caspase 8, and FADD-like apoptosis regulator (FLIP); oncogenes/transcriptional factors such as Jun-D, Xbp-1, calmodulin, Calnexin, and FGFR-3; stress response and ubiquitin/proteasome pathway-related genes and various ribosomal genes reflecting increased metabolic and translational activity. Genes that were down-regulated in MM cells versus healthy twin PCs included RAD51, killer cell immunoglobulin-like receptor protein, and apoptotic protease activating factor. Microarray results were further confirmed by Western blot analyses, immunohistochemistry, fluorescent in situ hybridization (FISH), and functional assays of telomerase activity and bone marrow angiogenesis. This molecular profiling provides potential insights into mechanisms of malignant transformation in MM. For example, FGFR3, xbp-1, and both mcl-1 and dad-1 may mediate transformation, differentiation, and survival, respectively, and may have clinical implications. By identifying genes uniquely altered in MM cells compared with normal PCs in an identical genotypic background, the current study provides the framework to identify novel therapeutic targets.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;12969976&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10791&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5888&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3805&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6597&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3297&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1736&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7345&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6498&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10641&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4848&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Kevin Vogelsang&#39;], &#39;exact_source&#39;: [&#39;Table 1&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>