<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>CASORELLI_APL_SECONDARY_VS_DE_NOVO_DN</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            CASORELLI_APL_SECONDARY_VS_DE_NOVO_DN
        </div>
        <div class="card-body">
                <p>Acute promyelocytic leukemia (APL) is a clonal expansion of hematopoietic precursors blocked at the promyelocytic stage. Gene expression profiles of APL cells obtained from 16 patients were compared to eight samples of CD34+-derived normal promyelocytes. Malignant promyelocytes showed widespread changes in transcription in comparison to their normal counterpart and 1020 differentially expressed genes were identified. Discriminating genes include transcriptional regulators (FOS, JUN and HOX genes) and genes involved in cell cycle and DNA repair. The strong upregulation in APL of some transcripts (FLT3, CD33, CD44 and HGF) was also confirmed at protein level. Interestingly, a trend toward a transcriptional repression of genes involved in different DNA repair pathways was found in APL and confirmed by real-time polymerase chain reactor (PCR) in a new set of nine APLs. Our results suggest that both inefficient base excision repair and recombinational repair might play a role in APLs development. To investigate the expression pathways underlying the development of APL occurring as a second malignancy (sAPL), we included in our study eight cases of sAPL. Although both secondary and de novo APL were characterized by a strong homogeneity in expression profiling, we identified a small set of differentially expressed genes that discriminate sAPL from de novo cases.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M19979">M19979</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:55544">ncbigene:55544</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:83700">ncbigene:83700</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:85001">ncbigene:85001</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8780">ncbigene:8780</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M19979&#39;, name=&#39;CASORELLI_APL_SECONDARY_VS_DE_NOVO_DN&#39;), definition=&#39;Acute promyelocytic leukemia (APL) is a clonal expansion of hematopoietic precursors blocked at the promyelocytic stage. Gene expression profiles of APL cells obtained from 16 patients were compared to eight samples of CD34+-derived normal promyelocytes. Malignant promyelocytes showed widespread changes in transcription in comparison to their normal counterpart and 1020 differentially expressed genes were identified. Discriminating genes include transcriptional regulators (FOS, JUN and HOX genes) and genes involved in cell cycle and DNA repair. The strong upregulation in APL of some transcripts (FLT3, CD33, CD44 and HGF) was also confirmed at protein level. Interestingly, a trend toward a transcriptional repression of genes involved in different DNA repair pathways was found in APL and confirmed by real-time polymerase chain reactor (PCR) in a new set of nine APLs. Our results suggest that both inefficient base excision repair and recombinational repair might play a role in APLs development. To investigate the expression pathways underlying the development of APL occurring as a second malignancy (sAPL), we included in our study eight cases of sAPL. Although both secondary and de novo APL were characterized by a strong homogeneity in expression profiling, we identified a small set of differentially expressed genes that discriminate sAPL from de novo cases.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;16990782&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55544&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;83700&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;85001&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8780&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;ARCHIVED&#39;], &#39;sub_category_code&#39;: [&#39;C2_CGP&#39;], &#39;contributor&#39;: [&#39;Leona Saunders&#39;], &#39;exact_source&#39;: [&#39;Table 2: downregulated in sAPL&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=True)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>