<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>SATO_SILENCED_BY_DEACETYLATION_IN_PANCREATIC_CANCER</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            SATO_SILENCED_BY_DEACETYLATION_IN_PANCREATIC_CANCER
        </div>
        <div class="card-body">
                <p>To identify potential targets for aberrant methylation in pancreatic cancer, we analyzed global changes in gene expression profiles of four pancreatic cancer cell lines after treatment with the demethylating agent 5-aza-2&#39;-deoxycytidine (5Aza-dC) and/or the histone deacetylase inhibitor trichostatin A. A substantial number of genes were induced 5-fold or greater by 5Aza-dC alone (631 transcripts), trichostatin A alone (1196 transcripts), and by treatment with both agents (857 transcripts). Four hundred and seventy-five genes were markedly (&gt;5-fold) induced after 5Aza-dC treatment in pancreatic cancer cell lines but not in a nonneoplastic pancreatic epithelial cell line. The methylation status of 11 of these 475 genes was examined in a panel of 42 pancreatic cancers, and all 11 of these genes were aberrantly methylated in pancreatic cancer but rarely, if any, methylated in 10 normal pancreatic ductal epithelia. These genes include UCHL1 (methylated in 100% of 42 pancreatic cancers), NPTX2 (98%), SARP2 (95%), CLDN5 (93%), reprimo (86%), LHX1 (76%), WNT7A (71%), FOXE1 (69%), TJP2 (64%), CDH3 (19%), and ST14 (10%). Three of these 11 genes (NPTX2, SARP2, and CLDN5) were selected for further analysis in a larger panel of specimens, and aberrant methylation of at least one of these three genes was detectable in 100% of 43 primary pancreatic cancers and in 18 of 24 (75%) pancreatic juice samples obtained from patients with pancreatic cancer. Thus, a substantial number of genes are induced by 5Aza-dC treatment of pancreatic cancer cells, and many of them may represent novel targets for aberrant methylation in pancreatic carcinoma.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M11213">M11213</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:81557">ncbigene:81557</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1028">ncbigene:1028</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:55743">ncbigene:55743</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:133">ncbigene:133</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6756">ncbigene:6756</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:645073">ncbigene:645073</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7980">ncbigene:7980</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1112">ncbigene:1112</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:24146">ncbigene:24146</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4152">ncbigene:4152</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1490">ncbigene:1490</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10641">ncbigene:10641</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1911">ncbigene:1911</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:638">ncbigene:638</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9518">ncbigene:9518</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3398">ncbigene:3398</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3006">ncbigene:3006</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5274">ncbigene:5274</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3576">ncbigene:3576</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:80328">ncbigene:80328</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3621">ncbigene:3621</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10669">ncbigene:10669</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6781">ncbigene:6781</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:374">ncbigene:374</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:43849">ncbigene:43849</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3039">ncbigene:3039</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4804">ncbigene:4804</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3162">ncbigene:3162</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:467">ncbigene:467</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7351">ncbigene:7351</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2543">ncbigene:2543</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:726">ncbigene:726</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5187">ncbigene:5187</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5243">ncbigene:5243</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4312">ncbigene:4312</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:284">ncbigene:284</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:54361">ncbigene:54361</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4854">ncbigene:4854</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8913">ncbigene:8913</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10855">ncbigene:10855</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1961">ncbigene:1961</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1006">ncbigene:1006</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4616">ncbigene:4616</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2069">ncbigene:2069</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10202">ncbigene:10202</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:30014">ncbigene:30014</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:100008586">ncbigene:100008586</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M11213&#39;, name=&#39;SATO_SILENCED_BY_DEACETYLATION_IN_PANCREATIC_CANCER&#39;), definition=&#34;To identify potential targets for aberrant methylation in pancreatic cancer, we analyzed global changes in gene expression profiles of four pancreatic cancer cell lines after treatment with the demethylating agent 5-aza-2&#39;-deoxycytidine (5Aza-dC) and/or the histone deacetylase inhibitor trichostatin A. A substantial number of genes were induced 5-fold or greater by 5Aza-dC alone (631 transcripts), trichostatin A alone (1196 transcripts), and by treatment with both agents (857 transcripts). Four hundred and seventy-five genes were markedly (&gt;5-fold) induced after 5Aza-dC treatment in pancreatic cancer cell lines but not in a nonneoplastic pancreatic epithelial cell line. The methylation status of 11 of these 475 genes was examined in a panel of 42 pancreatic cancers, and all 11 of these genes were aberrantly methylated in pancreatic cancer but rarely, if any, methylated in 10 normal pancreatic ductal epithelia. These genes include UCHL1 (methylated in 100% of 42 pancreatic cancers), NPTX2 (98%), SARP2 (95%), CLDN5 (93%), reprimo (86%), LHX1 (76%), WNT7A (71%), FOXE1 (69%), TJP2 (64%), CDH3 (19%), and ST14 (10%). Three of these 11 genes (NPTX2, SARP2, and CLDN5) were selected for further analysis in a larger panel of specimens, and aberrant methylation of at least one of these three genes was detectable in 100% of 43 primary pancreatic cancers and in 18 of 24 (75%) pancreatic juice samples obtained from patients with pancreatic cancer. Thus, a substantial number of genes are induced by 5Aza-dC treatment of pancreatic cancer cells, and many of them may represent novel targets for aberrant methylation in pancreatic carcinoma.&#34;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;12839967&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;81557&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1028&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55743&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;133&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6756&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;645073&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7980&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1112&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;24146&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4152&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1490&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10641&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1911&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;638&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9518&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3398&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3006&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5274&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3576&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;80328&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3621&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10669&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6781&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;374&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;43849&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3039&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4804&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3162&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;467&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7351&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2543&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;726&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5187&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5243&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4312&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;284&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;54361&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4854&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8913&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10855&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1961&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1006&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4616&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2069&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10202&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;30014&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;100008586&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;John Newman&#39;], &#39;exact_source&#39;: [&#39;http://pathology2.jhu.edu/pancreas/TSA.pdf&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>