<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>DESERT_EXTRACELLULAR_MATRIX_HEPATOCELLULAR_CARCINOMA_SUBCLASS_UP</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            DESERT_EXTRACELLULAR_MATRIX_HEPATOCELLULAR_CARCINOMA_SUBCLASS_UP
        </div>
        <div class="card-body">
                <p>Hepatocellular carcinomas (HCCs) exhibit a diversity of molecular phenotypes, raising major challenges in clinical management. HCCs detected by surveillance programs at an early stage are candidates for potentially curative therapies (local ablation, resection or transplantation). In the long term, transplantation provides the lowest recurrence rates. Treatment allocation is based on tumor number, size, vascular invasion, performance status, functional liver reserve and on the prediction of early (&lt; 2 years) recurrence, which reflects the intrinsic aggressiveness of the tumor. Well-differentiated, potentially low-aggressiveness tumors form the heterogeneous molecular class of non-proliferative HCCs, characterized by an approximate 50% beta-catenin (CTNNB1) mutation rate. To define the clinical, pathological, molecular features and the outcome of non-proliferative HCCs, we constructed an 1133-HCC transcriptomic metadata set and validated findings in a publically available 210-HCC RNAseq set. We show that non-proliferative HCCs preserve the zonation program that distributes metabolic functions along the porto-central axis in normal liver. More precisely, we identified two well-differentiated, non-proliferation subclasses, namely Periportal-type (wild-type CTNNB1) and Perivenous-type (mutant CTNNB1), which expressed negatively correlated gene networks. The new Periportal-type subclass represented 29% of all HCCs; expressed an HNF4A-driven gene network, which was down-regulated in mouse Hnf4a-KO mice; were early-stage tumors by BCLC, CLIP and TNM staging systems; had no macrovascular invasion and showed the lowest metastasis-specific gene expression levels and TP53 mutation rates. Also, we identified an 8-gene Periportal-type HCC signature, which was independently associated with the highest 2-year recurrence-free survival by multivariate analyses in two independent cohorts of 247 and 210 patients. Conclusion: Well-differentiated HCCs display mutually exclusive periportal or perivenous zonation programs. Among all HCCs, Periportal-type tumors have the lowest intrinsic potential for early recurrence after curative resection.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M34033">M34033</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:2182">ncbigene:2182</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:288">ncbigene:288</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:51066">ncbigene:51066</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6363">ncbigene:6363</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1363">ncbigene:1363</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1634">ncbigene:1634</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:51635">ncbigene:51635</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2150">ncbigene:2150</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2200">ncbigene:2200</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2326">ncbigene:2326</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2744">ncbigene:2744</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3309">ncbigene:3309</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4060">ncbigene:4060</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:56667">ncbigene:56667</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10154">ncbigene:10154</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5480">ncbigene:5480</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5819">ncbigene:5819</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:51097">ncbigene:51097</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8406">ncbigene:8406</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7070">ncbigene:7070</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8876">ncbigene:8876</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:23286">ncbigene:23286</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M34033&#39;, name=&#39;DESERT_EXTRACELLULAR_MATRIX_HEPATOCELLULAR_CARCINOMA_SUBCLASS_UP&#39;), definition=&#39;Hepatocellular carcinomas (HCCs) exhibit a diversity of molecular phenotypes, raising major challenges in clinical management. HCCs detected by surveillance programs at an early stage are candidates for potentially curative therapies (local ablation, resection or transplantation). In the long term, transplantation provides the lowest recurrence rates. Treatment allocation is based on tumor number, size, vascular invasion, performance status, functional liver reserve and on the prediction of early (&lt; 2 years) recurrence, which reflects the intrinsic aggressiveness of the tumor. Well-differentiated, potentially low-aggressiveness tumors form the heterogeneous molecular class of non-proliferative HCCs, characterized by an approximate 50% beta-catenin (CTNNB1) mutation rate. To define the clinical, pathological, molecular features and the outcome of non-proliferative HCCs, we constructed an 1133-HCC transcriptomic metadata set and validated findings in a publically available 210-HCC RNAseq set. We show that non-proliferative HCCs preserve the zonation program that distributes metabolic functions along the porto-central axis in normal liver. More precisely, we identified two well-differentiated, non-proliferation subclasses, namely Periportal-type (wild-type CTNNB1) and Perivenous-type (mutant CTNNB1), which expressed negatively correlated gene networks. The new Periportal-type subclass represented 29% of all HCCs; expressed an HNF4A-driven gene network, which was down-regulated in mouse Hnf4a-KO mice; were early-stage tumors by BCLC, CLIP and TNM staging systems; had no macrovascular invasion and showed the lowest metastasis-specific gene expression levels and TP53 mutation rates. Also, we identified an 8-gene Periportal-type HCC signature, which was independently associated with the highest 2-year recurrence-free survival by multivariate analyses in two independent cohorts of 247 and 210 patients. Conclusion: Well-differentiated HCCs display mutually exclusive periportal or perivenous zonation programs. Among all HCCs, Periportal-type tumors have the lowest intrinsic potential for early recurrence after curative resection.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;28498607&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2182&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;288&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51066&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6363&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1363&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1634&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51635&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2150&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2200&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2326&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2744&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3309&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4060&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;56667&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10154&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5480&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5819&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51097&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8406&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7070&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8876&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23286&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Orlando Musso&#39;], &#39;exact_source&#39;: [&#39;Fig. 1B &amp; Supplemental Table 2.&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>