<!DOCTYPE html>
<html>
<head>
            <meta charset="utf-8">
            <meta name="viewport" content="width=device-width, initial-scale=1.0">
            <!-- Bootstrap -->
            <link
                    rel="stylesheet"
                    href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
                    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
                    crossorigin="anonymous"
            />
            <link
                    rel="stylesheet"
                    href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"
            />
            <link
                    rel="stylesheet"
                    href="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.css"
            />
            <!-- DataTables, see: https://datatables.net/examples/styling/bootstrap4.html-->
            <link
                    rel="stylesheet"
                    href="https://cdn.datatables.net/1.10.19/css/dataTables.bootstrap4.min.css"
            />
            <style type="text/css">
                h2 {
                    margin-top: 35px;
                }

                html, body {
                    height: 100%;
                }

                body {
                    display: flex;
                    flex-flow: column;
                }

                .footer {
                    margin-top: auto;
                    padding-top: 1em;
                    background-color: #f5f5f5;
                }

                nav[data-toggle="toc"] {
                    top: 42px;
                }
            </style>

        <script src="https://kit.fontawesome.com/4c86883252.js" crossorigin="anonymous"></script>
        <title>SETLUR_PROSTATE_CANCER_TMPRSS2_ERG_FUSION_UP</title>
</head>

<body data-spy="scroll" data-target="#toc">
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>


    <div class="container" style="margin-top: 50px; margin-bottom: 50px">
        <div class="card">
        <div class="card-header">
            SETLUR_PROSTATE_CANCER_TMPRSS2_ERG_FUSION_UP
        </div>
        <div class="card-body">
            
                <p>BACKGROUND: The majority of prostate cancers harbor gene fusions of the 5&#39;-untranslated region of the androgen-regulated transmembrane protease serine 2 (TMPRSS2) promoter with erythroblast transformation-specific transcription factor family members. The common fusion between TMPRESS2 and v-ets erythroblastosis virus E26 oncogene homolog (avian) (ERG) is associated with a more aggressive clinical phenotype, implying the existence of a distinct subclass of prostate cancer defined by this fusion. METHODS: We used complementary DNA-mediated annealing, selection, ligation, and extension to determine the expression profiles of 6144 transcriptionally informative genes in archived biopsy samples from 455 prostate cancer patients in the Swedish Watchful Waiting cohort (1987-1999) and the United States-based Physicians(&#39;) Health Study cohort (1983-2003). A gene expression signature for prostate cancers with the TMPRSS2-ERG fusion was determined using partitioning and classification models and used in computational functional analysis. Cell proliferation and TMPRSS2-ERG expression in androgen receptor-negative (NCI-H660) prostate cancer cells after treatment with vehicle or estrogenic compounds were assessed by viability assays and quantitative polymerase chain reaction, respectively. All statistical tests were two-sided. RESULTS: We identified an 87-gene expression signature that distinguishes TMPRSS2-ERG fusion prostate cancer as a discrete molecular entity (area under the curve = 0.80, 95% confidence interval [CI] = 0.792 to 0.81; P &lt; .001). Computational analysis suggested that this fusion signature was associated with estrogen receptor (ER) signaling. Viability of NCI-H660 cells decreased after treatment with estrogen (viability normalized to day 0, estrogen vs vehicle at day 8, mean = 2.04 vs 3.40, difference = 1.36, 95% CI = 1.12 to 1.62) or ERbeta agonist (ERbeta agonist vs vehicle at day 8, mean = 1.86 vs 3.40, difference = 1.54, 95% CI = 1.39 to 1.69) but increased after ERalpha agonist treatment (ERalpha agonist vs vehicle at day 8, mean = 4.36 vs 3.40, difference = 0.96, 95% CI = 0.68 to 1.23). Similarly, expression of TMPRSS2-ERG decreased after ERbeta agonist treatment (fold change over internal control, ERbeta agonist vs vehicle at 24 hours, NCI-H660, mean = 0.57- vs 1.0-fold, difference = 0.43-fold, 95% CI = 0.29- to 0.57-fold) and increased after ERalpha agonist treatment (ERalpha agonist vs vehicle at 24 hours, mean = 5.63- vs 1.0-fold, difference = 4.63-fold, 95% CI = 4.34- to 4.92-fold). CONCLUSIONS: TMPRSS2-ERG fusion prostate cancer is a distinct molecular subclass. TMPRSS2-ERG expression is regulated by a novel ER-dependent mechanism.</p>
            
            <dl>
                <dt>Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M3695">M3695</a></dd>
                
            </dl>


            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M3695&#39;, name=&#39;SETLUR_PROSTATE_CANCER_TMPRSS2_ERG_FUSION_UP&#39;), definition=&#34;BACKGROUND: The majority of prostate cancers harbor gene fusions of the 5&#39;-untranslated region of the androgen-regulated transmembrane protease serine 2 (TMPRSS2) promoter with erythroblast transformation-specific transcription factor family members. The common fusion between TMPRESS2 and v-ets erythroblastosis virus E26 oncogene homolog (avian) (ERG) is associated with a more aggressive clinical phenotype, implying the existence of a distinct subclass of prostate cancer defined by this fusion. METHODS: We used complementary DNA-mediated annealing, selection, ligation, and extension to determine the expression profiles of 6144 transcriptionally informative genes in archived biopsy samples from 455 prostate cancer patients in the Swedish Watchful Waiting cohort (1987-1999) and the United States-based Physicians(&#39;) Health Study cohort (1983-2003). A gene expression signature for prostate cancers with the TMPRSS2-ERG fusion was determined using partitioning and classification models and used in computational functional analysis. Cell proliferation and TMPRSS2-ERG expression in androgen receptor-negative (NCI-H660) prostate cancer cells after treatment with vehicle or estrogenic compounds were assessed by viability assays and quantitative polymerase chain reaction, respectively. All statistical tests were two-sided. RESULTS: We identified an 87-gene expression signature that distinguishes TMPRSS2-ERG fusion prostate cancer as a discrete molecular entity (area under the curve = 0.80, 95% confidence interval [CI] = 0.792 to 0.81; P &lt; .001). Computational analysis suggested that this fusion signature was associated with estrogen receptor (ER) signaling. Viability of NCI-H660 cells decreased after treatment with estrogen (viability normalized to day 0, estrogen vs vehicle at day 8, mean = 2.04 vs 3.40, difference = 1.36, 95% CI = 1.12 to 1.62) or ERbeta agonist (ERbeta agonist vs vehicle at day 8, mean = 1.86 vs 3.40, difference = 1.54, 95% CI = 1.39 to 1.69) but increased after ERalpha agonist treatment (ERalpha agonist vs vehicle at day 8, mean = 4.36 vs 3.40, difference = 0.96, 95% CI = 0.68 to 1.23). Similarly, expression of TMPRSS2-ERG decreased after ERbeta agonist treatment (fold change over internal control, ERbeta agonist vs vehicle at 24 hours, NCI-H660, mean = 0.57- vs 1.0-fold, difference = 0.43-fold, 95% CI = 0.29- to 0.57-fold) and increased after ERalpha agonist treatment (ERalpha agonist vs vehicle at 24 hours, mean = 5.63- vs 1.0-fold, difference = 4.63-fold, 95% CI = 4.34- to 4.92-fold). CONCLUSIONS: TMPRSS2-ERG fusion prostate cancer is a distinct molecular subclass. TMPRSS2-ERG expression is regulated by a novel ER-dependent mechanism.&#34;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;18505969&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;776&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7520&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4217&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6602&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8853&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10321&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6908&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2690&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9112&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7941&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;100506658&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;658&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7326&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55623&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10656&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6001&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5719&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5796&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5989&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;27314&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5226&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1983&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6294&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5832&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;272&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2078&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7088&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9053&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1298&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;22877&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4725&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3065&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4646&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;11052&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2812&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23613&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6833&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;26751&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7358&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;56099&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5192&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1889&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3831&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5575&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5152&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7551&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9766&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9411&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9529&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;397&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5218&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5891&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8648&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7174&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5607&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8618&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7027&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6629&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10801&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5754&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10557&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8672&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;26037&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3781&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4035&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1627&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7291&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Jessica Robertson&#39;], &#39;exact_source&#39;: [&#39;Table 3: Expression level=Increased&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div>
    </div>


<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>


    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
            integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
            crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"
            integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
            crossorigin="anonymous"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
            integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
            crossorigin="anonymous"></script>
    <script src="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.js"></script>

    <!-- DataTables, see: https://datatables.net/examples/styling/bootstrap4.html-->
    <script src="https://cdn.datatables.net/1.10.19/js/jquery.dataTables.min.js"></script>
    <script src="https://cdn.datatables.net/1.10.19/js/dataTables.bootstrap4.min.js"></script>

    <script src="https://d3js.org/d3.v5.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.js"></script>

    <script>
        $(function () {
            $('[data-toggle="tooltip"]').tooltip()
        })
    </script>
</body>
</html>