<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>WANG_BARRETTS_ESOPHAGUS_DN</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            WANG_BARRETTS_ESOPHAGUS_DN
        </div>
        <div class="card-body">
                <p>To investigate the relationship between Barrett&#39;s esophagus (BE) and esophageal adenocarcinoma (EAC), we determined gene expression profiles of discrete pathological stages of esophageal neoplasia using a sequence-verified human cDNA microarray. Fifty one RNAs, comprising 24 normal esophagi (NE), 18 BEs, and nine EACs were hybridized to cDNA microarrays. Five statistical analyses were used for the data analysis. Genes showing significantly different expression levels among the three sample groups were identified. Genes were grouped into functional categories based on the Gene Ontology Consortium. Surprisingly, the expression pattern of BE was significantly more similar to EAC than to NE, notwithstanding the known histopathologic differences between BE and EAC. The pattern of NE was clearly distinct from that of EAC. Thirty-six genes were the most differentially modulated, according to these microarray data, in BE-associated neoplastic progression. Twelve genes were significantly differentially expressed in cancer-associated BE&#39;s plus EAC (as a single combined tissue group) vs noncancer-associated BE&#39;s. These genes represent potential biomarkers to diagnose EAC at its early stages. Our results demonstrate that molecular events at the transcriptional level in BE are remarkably similar to BE&#39;s-associated adenocarcinoma of the esophagus. This finding alarmingly implies that BE is biologically closer to cancer than to normal esophagus, and that the cancer risk of BE is perhaps higher than we had imagined. These findings suggest that changes modulated at the molecular biologic level supervene earlier than histologic changes, and that BE is an early intermediate stage in the process of EAC.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M18213">M18213</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:2327">ncbigene:2327</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6707">ncbigene:6707</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:360">ncbigene:360</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8428">ncbigene:8428</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4780">ncbigene:4780</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5650">ncbigene:5650</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4048">ncbigene:4048</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:381">ncbigene:381</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1508">ncbigene:1508</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1992">ncbigene:1992</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:223">ncbigene:223</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2013">ncbigene:2013</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10096">ncbigene:10096</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5017">ncbigene:5017</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3728">ncbigene:3728</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5547">ncbigene:5547</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10457">ncbigene:10457</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6279">ncbigene:6279</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2261">ncbigene:2261</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1828">ncbigene:1828</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5355">ncbigene:5355</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:688">ncbigene:688</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5653">ncbigene:5653</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8452">ncbigene:8452</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2941">ncbigene:2941</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M18213&#39;, name=&#39;WANG_BARRETTS_ESOPHAGUS_DN&#39;), definition=&#34;To investigate the relationship between Barrett&#39;s esophagus (BE) and esophageal adenocarcinoma (EAC), we determined gene expression profiles of discrete pathological stages of esophageal neoplasia using a sequence-verified human cDNA microarray. Fifty one RNAs, comprising 24 normal esophagi (NE), 18 BEs, and nine EACs were hybridized to cDNA microarrays. Five statistical analyses were used for the data analysis. Genes showing significantly different expression levels among the three sample groups were identified. Genes were grouped into functional categories based on the Gene Ontology Consortium. Surprisingly, the expression pattern of BE was significantly more similar to EAC than to NE, notwithstanding the known histopathologic differences between BE and EAC. The pattern of NE was clearly distinct from that of EAC. Thirty-six genes were the most differentially modulated, according to these microarray data, in BE-associated neoplastic progression. Twelve genes were significantly differentially expressed in cancer-associated BE&#39;s plus EAC (as a single combined tissue group) vs noncancer-associated BE&#39;s. These genes represent potential biomarkers to diagnose EAC at its early stages. Our results demonstrate that molecular events at the transcriptional level in BE are remarkably similar to BE&#39;s-associated adenocarcinoma of the esophagus. This finding alarmingly implies that BE is biologically closer to cancer than to normal esophagus, and that the cancer risk of BE is perhaps higher than we had imagined. These findings suggest that changes modulated at the molecular biologic level supervene earlier than histologic changes, and that BE is an early intermediate stage in the process of EAC.&#34;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;16449976&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2327&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6707&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;360&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8428&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4780&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5650&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4048&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;381&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1508&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1992&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;223&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2013&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10096&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5017&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3728&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5547&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10457&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6279&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2261&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1828&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5355&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;688&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5653&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8452&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2941&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Arthur Liberzon&#39;], &#39;exact_source&#39;: [&#39;Table 1S: Fold change &lt; 1&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>