<!DOCTYPE html>
<html>
<head>
            <meta charset="utf-8">
            <meta name="viewport" content="width=device-width, initial-scale=1.0">
            <!-- Bootstrap -->
            <link
                    rel="stylesheet"
                    href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
                    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
                    crossorigin="anonymous"
            />
            <link
                    rel="stylesheet"
                    href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"
            />
            <link
                    rel="stylesheet"
                    href="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.css"
            />
            <!-- DataTables, see: https://datatables.net/examples/styling/bootstrap4.html-->
            <link
                    rel="stylesheet"
                    href="https://cdn.datatables.net/1.10.19/css/dataTables.bootstrap4.min.css"
            />
            <style type="text/css">
                h2 {
                    margin-top: 35px;
                }

                html, body {
                    height: 100%;
                }

                body {
                    display: flex;
                    flex-flow: column;
                }

                .footer {
                    margin-top: auto;
                    padding-top: 1em;
                    background-color: #f5f5f5;
                }

                nav[data-toggle="toc"] {
                    top: 42px;
                }
            </style>

        <script src="https://kit.fontawesome.com/4c86883252.js" crossorigin="anonymous"></script>
        <title>CHANGOLKAR_H2AFY_TARGETS_DN</title>
</head>

<body data-spy="scroll" data-target="#toc">
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>


    <div class="container" style="margin-top: 50px; margin-bottom: 50px">
        <div class="card">
        <div class="card-header">
            CHANGOLKAR_H2AFY_TARGETS_DN
        </div>
        <div class="card-body">
            
                <p>macroH2A histone variants have been implicated to function in gene silencing by several studies, including ones showing a preferential association of macroH2A on the inactive X chromosome. To examine macroH2A function in vivo, we knocked out macroH2A1. macroH2A1 knockout mice are viable and fertile. A broad screen of liver gene expression showed no evidence of defects in X inactivation but did identify genes that have increased expression levels in macroH2A1 knockouts. macroH2A1-containing nucleosomes are enriched on the coding and/or upstream regions of these genes, suggesting that their increased expression levels are a direct effect of the absence of macroH2A1. The concentrations of macroH2A1 nucleosomes on these genes are low in the livers of newborn mice, and the macroH2A1 knockout had little effect on the expression levels of these genes in newborn liver. Our results indicate that an increase in liver macroH2A1 during the transition from newborn to young-adult status contributes to a decrease in the expression levels of these genes. These genes cluster in the area of lipid metabolism, and we observed metabolic effects in macroH2A1 knockouts. Our results indicate that the function of macroH2A1 histones is not restricted to gene silencing but also involves fine tuning the expression of specific genes.</p>
            
            <dl>
                <dt>Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M2290">M2290</a></dd>
                
            </dl>


            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M2290&#39;, name=&#39;CHANGOLKAR_H2AFY_TARGETS_DN&#39;), definition=&#39;macroH2A histone variants have been implicated to function in gene silencing by several studies, including ones showing a preferential association of macroH2A on the inactive X chromosome. To examine macroH2A function in vivo, we knocked out macroH2A1. macroH2A1 knockout mice are viable and fertile. A broad screen of liver gene expression showed no evidence of defects in X inactivation but did identify genes that have increased expression levels in macroH2A1 knockouts. macroH2A1-containing nucleosomes are enriched on the coding and/or upstream regions of these genes, suggesting that their increased expression levels are a direct effect of the absence of macroH2A1. The concentrations of macroH2A1 nucleosomes on these genes are low in the livers of newborn mice, and the macroH2A1 knockout had little effect on the expression levels of these genes in newborn liver. Our results indicate that an increase in liver macroH2A1 during the transition from newborn to young-adult status contributes to a decrease in the expression levels of these genes. These genes cluster in the area of lipid metabolism, and we observed metabolic effects in macroH2A1 knockouts. Our results indicate that the function of macroH2A1 histones is not restricted to gene silencing but also involves fine tuning the expression of specific genes.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;17242180&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;10090&#39;, name=&#39;Mus musculus&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9555&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;26022&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5730&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5028&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2690&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;54737&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23560&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;733&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;56670&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;83873&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;286257&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55254&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1240&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8531&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7392&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;79048&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;80124&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;135228&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51631&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5629&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23545&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;84916&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2944&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;22936&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;64240&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;339287&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6787&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;79020&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9775&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;367&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55720&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5648&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;219899&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5824&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8470&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51084&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;140828&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10523&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55669&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5649&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4638&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Arthur Liberzon&#39;], &#39;exact_source&#39;: [&#39;Table 1S: ratio ko/cont &lt; 1&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div>
    </div>


<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>


    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
            integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
            crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"
            integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
            crossorigin="anonymous"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
            integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
            crossorigin="anonymous"></script>
    <script src="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.js"></script>

    <!-- DataTables, see: https://datatables.net/examples/styling/bootstrap4.html-->
    <script src="https://cdn.datatables.net/1.10.19/js/jquery.dataTables.min.js"></script>
    <script src="https://cdn.datatables.net/1.10.19/js/dataTables.bootstrap4.min.js"></script>

    <script src="https://d3js.org/d3.v5.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.js"></script>

    <script>
        $(function () {
            $('[data-toggle="tooltip"]').tooltip()
        })
    </script>
</body>
</html>