<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>EHRLICH_ICF_SYNDROM_UP</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            EHRLICH_ICF_SYNDROM_UP
        </div>
        <div class="card-body">
                <p>ICF (immunodeficiency, centromeric region instability and facial anomalies) is a recessive disease caused by mutations in the DNA methyltransferase 3B gene (DNMT3B). Patients have immunodeficiency, chromosome 1 (Chr1) and Chr16 pericentromeric anomalies in mitogen-stimulated lymphocytes, a small decrease in overall genomic 5-methylcytosine levels and much hypomethylation of Chr1 and Chr16 juxtacentromeric heterochromatin. Microarray expression analysis was done on B-cell lymphoblastoid cell lines (LCLs) from ICF patients with diverse DNMT3B mutations and on control LCLs using oligonucleotide arrays for approximately 5600 different genes, 510 of which showed a lymphoid lineage-restricted expression pattern among several different lineages tested. A set of 32 genes had consistent and significant ICF-specific changes in RNA levels. Half of these genes play a role in immune function. ICF-specific increases in immunoglobulin (Ig) heavy constant mu and delta RNA and cell surface IgM and IgD and decreases in Ig(gamma) and Ig(alpha) RNA and surface IgG and IgA indicate inhibition of the later steps of lymphocyte maturation. ICF-specific increases were seen in RNA for RGS1, a B-cell specific inhibitor of G-protein signaling implicated in negative regulation of B-cell migration, and in RNA for the pro-apoptotic protein kinase C eta gene. ICF-associated decreases were observed in RNAs encoding proteins involved in activation, migration or survival of lymphoid cells, namely, transcription factor negative regulator ID3, the enhancer-binding MEF2C, the iron regulatory transferrin receptor, integrin beta7, the stress protein heme oxygenase and the lymphocyte-specific tumor necrosis factor receptor family members 7 and 17. No differences in promoter methylation were seen between ICF and normal LCLs for three ICF upregulated genes and one downregulated gene by a quantitative methylation assay [combined bisulfite restriction analysis (COBRA)]. Our data suggest that DNMT3B mutations in the ICF syndrome cause lymphogenesis-associated gene dysregulation by indirect effects on gene expression that interfere with normal lymphocyte signaling, maturation and migration.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M5053">M5053</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:5914">ncbigene:5914</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2274">ncbigene:2274</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2162">ncbigene:2162</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2983">ncbigene:2983</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2982">ncbigene:2982</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1266">ncbigene:1266</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3507">ncbigene:3507</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5583">ncbigene:5583</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:27113">ncbigene:27113</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3495">ncbigene:3495</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10643">ncbigene:10643</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4140">ncbigene:4140</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5996">ncbigene:5996</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M5053&#39;, name=&#39;EHRLICH_ICF_SYNDROM_UP&#39;), definition=&#39;ICF (immunodeficiency, centromeric region instability and facial anomalies) is a recessive disease caused by mutations in the DNA methyltransferase 3B gene (DNMT3B). Patients have immunodeficiency, chromosome 1 (Chr1) and Chr16 pericentromeric anomalies in mitogen-stimulated lymphocytes, a small decrease in overall genomic 5-methylcytosine levels and much hypomethylation of Chr1 and Chr16 juxtacentromeric heterochromatin. Microarray expression analysis was done on B-cell lymphoblastoid cell lines (LCLs) from ICF patients with diverse DNMT3B mutations and on control LCLs using oligonucleotide arrays for approximately 5600 different genes, 510 of which showed a lymphoid lineage-restricted expression pattern among several different lineages tested. A set of 32 genes had consistent and significant ICF-specific changes in RNA levels. Half of these genes play a role in immune function. ICF-specific increases in immunoglobulin (Ig) heavy constant mu and delta RNA and cell surface IgM and IgD and decreases in Ig(gamma) and Ig(alpha) RNA and surface IgG and IgA indicate inhibition of the later steps of lymphocyte maturation. ICF-specific increases were seen in RNA for RGS1, a B-cell specific inhibitor of G-protein signaling implicated in negative regulation of B-cell migration, and in RNA for the pro-apoptotic protein kinase C eta gene. ICF-associated decreases were observed in RNAs encoding proteins involved in activation, migration or survival of lymphoid cells, namely, transcription factor negative regulator ID3, the enhancer-binding MEF2C, the iron regulatory transferrin receptor, integrin beta7, the stress protein heme oxygenase and the lymphocyte-specific tumor necrosis factor receptor family members 7 and 17. No differences in promoter methylation were seen between ICF and normal LCLs for three ICF upregulated genes and one downregulated gene by a quantitative methylation assay [combined bisulfite restriction analysis (COBRA)]. Our data suggest that DNMT3B mutations in the ICF syndrome cause lymphogenesis-associated gene dysregulation by indirect effects on gene expression that interfere with normal lymphocyte signaling, maturation and migration.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;11741835&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5914&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2274&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2162&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2983&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2982&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1266&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3507&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5583&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;27113&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3495&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10643&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4140&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5996&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;John Newman&#39;], &#39;exact_source&#39;: [&#39;Table 2-4: FC &gt; 0&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>