<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>MOOTHA_FFA_OXYDATION</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            MOOTHA_FFA_OXYDATION
        </div>
        <div class="card-body">
                <p>DNA microarrays can be used to identify gene expression changes characteristic of human disease. This is challenging, however, when relevant differences are subtle at the level of individual genes. We introduce an analytical strategy, Gene Set Enrichment Analysis, designed to detect modest but coordinate changes in the expression of groups of functionally related genes. Using this approach, we identify a set of genes involved in oxidative phosphorylation whose expression is coordinately decreased in human diabetic muscle. Expression of these genes is high at sites of insulin-mediated glucose disposal, activated by PGC-1alpha and correlated with total-body aerobic capacity. Our results associate this gene set with clinically important variation in human metabolism and illustrate the value of pathway relationships in the analysis of genomic profiling experiments.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M15531">M15531</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:3033">ncbigene:3033</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1384">ncbigene:1384</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2180">ncbigene:2180</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1374">ncbigene:1374</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3032">ncbigene:3032</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:37">ncbigene:37</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1666">ncbigene:1666</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:38">ncbigene:38</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:33">ncbigene:33</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:36">ncbigene:36</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1892">ncbigene:1892</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:23600">ncbigene:23600</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1632">ncbigene:1632</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1375">ncbigene:1375</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10449">ncbigene:10449</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1376">ncbigene:1376</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:34">ncbigene:34</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:35">ncbigene:35</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3030">ncbigene:3030</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4683">ncbigene:4683</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:91289">ncbigene:91289</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3028">ncbigene:3028</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:29781">ncbigene:29781</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M15531&#39;, name=&#39;MOOTHA_FFA_OXYDATION&#39;), definition=&#39;DNA microarrays can be used to identify gene expression changes characteristic of human disease. This is challenging, however, when relevant differences are subtle at the level of individual genes. We introduce an analytical strategy, Gene Set Enrichment Analysis, designed to detect modest but coordinate changes in the expression of groups of functionally related genes. Using this approach, we identify a set of genes involved in oxidative phosphorylation whose expression is coordinately decreased in human diabetic muscle. Expression of these genes is high at sites of insulin-mediated glucose disposal, activated by PGC-1alpha and correlated with total-body aerobic capacity. Our results associate this gene set with clinically important variation in human metabolism and illustrate the value of pathway relationships in the analysis of genomic profiling experiments.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;12808457&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3033&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1384&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2180&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1374&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3032&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;37&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1666&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;38&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;33&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;36&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1892&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23600&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1632&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1375&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10449&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1376&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;34&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;35&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3030&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4683&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;91289&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3028&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;29781&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Vamsi Mootha&#39;], &#39;exact_source&#39;: [&#39;Suppl. file: all_pathways/FA_HG-U133A_probes&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>