<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>STEINER_ERYTHROCYTE_MEMBRANE_GENES</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            STEINER_ERYTHROCYTE_MEMBRANE_GENES
        </div>
        <div class="card-body">
                <p>Erythrocyte membrane protein genes serve as excellent models of complex gene locus structure and function, but their study has been complicated by both their large size and their complexity. To begin to understand the intricate interplay of transcription, dynamic chromatin architecture, transcription factor binding, and genomic organization in regulation of erythrocyte membrane protein genes, we performed chromatin immunoprecipitation (ChIP) coupled with microarray analysis and ChIP coupled with massively parallel DNA sequencing in both erythroid and nonerythroid cells. Unexpectedly, most regions of GATA-1 and NF-E2 binding were remote from gene promoters and transcriptional start sites, located primarily in introns. Cooccupancy with FOG-1, SCL, and MTA-2 was found at all regions of GATA-1 binding, with cooccupancy of SCL and MTA-2 also found at regions of NF-E2 binding. Cooccupancy of GATA-1 and NF-E2 was found frequently. A common signature of histone H3 trimethylation at lysine 4, GATA-1, NF-E2, FOG-1, SCL, and MTA-2 binding and consensus GATA-1-E-box binding motifs located 34 to 90 bp away from NF-E2 binding motifs was found frequently in erythroid cell-expressed genes. These results provide insights into our understanding of membrane protein gene regulation in erythropoiesis and the regulation of complex genetic loci in erythroid and nonerythroid cells and identify numerous candidate regions for mutations associated with membrane-linked hemolytic anemia.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M2442">M2442</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:2038">ncbigene:2038</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2035">ncbigene:2035</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:286">ncbigene:286</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:120">ncbigene:120</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6708">ncbigene:6708</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:60">ncbigene:60</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6521">ncbigene:6521</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:118">ncbigene:118</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:119">ncbigene:119</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6710">ncbigene:6710</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7111">ncbigene:7111</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7170">ncbigene:7170</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2039">ncbigene:2039</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:114625">ncbigene:114625</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M2442&#39;, name=&#39;STEINER_ERYTHROCYTE_MEMBRANE_GENES&#39;), definition=&#39;Erythrocyte membrane protein genes serve as excellent models of complex gene locus structure and function, but their study has been complicated by both their large size and their complexity. To begin to understand the intricate interplay of transcription, dynamic chromatin architecture, transcription factor binding, and genomic organization in regulation of erythrocyte membrane protein genes, we performed chromatin immunoprecipitation (ChIP) coupled with microarray analysis and ChIP coupled with massively parallel DNA sequencing in both erythroid and nonerythroid cells. Unexpectedly, most regions of GATA-1 and NF-E2 binding were remote from gene promoters and transcriptional start sites, located primarily in introns. Cooccupancy with FOG-1, SCL, and MTA-2 was found at all regions of GATA-1 binding, with cooccupancy of SCL and MTA-2 also found at regions of NF-E2 binding. Cooccupancy of GATA-1 and NF-E2 was found frequently. A common signature of histone H3 trimethylation at lysine 4, GATA-1, NF-E2, FOG-1, SCL, and MTA-2 binding and consensus GATA-1-E-box binding motifs located 34 to 90 bp away from NF-E2 binding motifs was found frequently in erythroid cell-expressed genes. These results provide insights into our understanding of membrane protein gene regulation in erythropoiesis and the regulation of complex genetic loci in erythroid and nonerythroid cells and identify numerous candidate regions for mutations associated with membrane-linked hemolytic anemia.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;19687298&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2038&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2035&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;286&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;120&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6708&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;60&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6521&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;118&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;119&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6710&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7111&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7170&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2039&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;114625&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Arthur Liberzon&#39;], &#39;exact_source&#39;: [&#39;Table 1&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>