<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>EPPERT_CE_HSC_LSC</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            EPPERT_CE_HSC_LSC
        </div>
        <div class="card-body">
                <p>Xenograft studies indicate that some solid tumors and leukemias are organized as cellular hierarchies sustained by cancer stem cells (CSCs). Despite the promise of the CSC model, its relevance in humans remains uncertain. Here we show that acute myeloid leukemia (AML) follows a CSC model on the basis of sorting multiple populations from each of 16 primary human AML samples and identifying which contain leukemia stem cells (LSCs) using a sensitive xenograft assay. Analysis of gene expression from all functionally validated populations yielded an LSC-specific signature. Similarly, a hematopoietic stem cell (HSC) gene signature was established. Bioinformatic analysis identified a core transcriptional program shared by LSCs and HSCs, revealing the molecular machinery underlying stemness properties. Both stem cell programs were highly significant independent predictors of patient survival and were found in existing prognostic signatures. Thus, determinants of stemness influence the clinical outcome of AML, establishing that LSCs are clinically relevant and not artifacts of xenotransplantation.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M19233">M19233</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:11030">ncbigene:11030</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3131">ncbigene:3131</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2982">ncbigene:2982</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9834">ncbigene:9834</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2078">ncbigene:2078</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:783">ncbigene:783</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8835">ncbigene:8835</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6691">ncbigene:6691</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1393">ncbigene:1393</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:55930">ncbigene:55930</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7525">ncbigene:7525</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3355">ncbigene:3355</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4211">ncbigene:4211</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:26051">ncbigene:26051</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:23150">ncbigene:23150</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:54941">ncbigene:54941</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5243">ncbigene:5243</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:51186">ncbigene:51186</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2122">ncbigene:2122</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7718">ncbigene:7718</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3212">ncbigene:3212</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:57211">ncbigene:57211</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8821">ncbigene:8821</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3213">ncbigene:3213</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6711">ncbigene:6711</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7035">ncbigene:7035</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:53335">ncbigene:53335</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:55151">ncbigene:55151</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:214">ncbigene:214</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1612">ncbigene:1612</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6594">ncbigene:6594</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:79870">ncbigene:79870</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:57088">ncbigene:57088</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3202">ncbigene:3202</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2322">ncbigene:2322</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3126">ncbigene:3126</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:55332">ncbigene:55332</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5583">ncbigene:5583</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2004">ncbigene:2004</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:55186">ncbigene:55186</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7994">ncbigene:7994</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M19233&#39;, name=&#39;EPPERT_CE_HSC_LSC&#39;), definition=&#39;Xenograft studies indicate that some solid tumors and leukemias are organized as cellular hierarchies sustained by cancer stem cells (CSCs). Despite the promise of the CSC model, its relevance in humans remains uncertain. Here we show that acute myeloid leukemia (AML) follows a CSC model on the basis of sorting multiple populations from each of 16 primary human AML samples and identifying which contain leukemia stem cells (LSCs) using a sensitive xenograft assay. Analysis of gene expression from all functionally validated populations yielded an LSC-specific signature. Similarly, a hematopoietic stem cell (HSC) gene signature was established. Bioinformatic analysis identified a core transcriptional program shared by LSCs and HSCs, revealing the molecular machinery underlying stemness properties. Both stem cell programs were highly significant independent predictors of patient survival and were found in existing prognostic signatures. Thus, determinants of stemness influence the clinical outcome of AML, establishing that LSCs are clinically relevant and not artifacts of xenotransplantation.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;21873988&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;11030&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3131&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2982&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9834&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2078&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;783&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8835&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6691&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1393&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55930&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7525&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3355&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4211&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;26051&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23150&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;54941&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5243&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51186&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2122&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7718&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3212&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;57211&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8821&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3213&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6711&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7035&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;53335&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55151&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;214&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1612&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6594&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;79870&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;57088&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3202&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2322&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3126&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55332&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5583&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2004&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55186&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7994&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Kolja Eppert&#39;], &#39;exact_source&#39;: [&#39;Fig 2D&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>