<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>LASTOWSKA_COAMPLIFIED_WITH_MYCN</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            LASTOWSKA_COAMPLIFIED_WITH_MYCN
        </div>
        <div class="card-body">
                <p>Identifying genes, whose expression is consistently altered by chromosomal gains or losses, is an important step in defining genes of biological relevance in a wide variety of tumour types. However, additional criteria are needed to discriminate further among the large number of candidate genes identified. This is particularly true for neuroblastoma, where multiple genomic copy number changes of proven prognostic value exist. We have used Affymetrix microarrays and a combination of fluorescent in situ hybridization and single nucleotide polymorphism (SNP) microarrays to establish expression profiles and delineate copy number alterations in 30 primary neuroblastomas. Correlation of microarray data with patient survival and analysis of expression within rodent neuroblastoma cell lines were then used to define further genes likely to be involved in the disease process. Using this approach, we identify &gt;1000 genes within eight recurrent genomic alterations (loss of 1p, 3p, 4p, 10q and 11q, 2p gain, 17q gain, and the MYCN amplicon) whose expression is consistently altered by copy number change. Of these, 84 correlate with patient survival, with the minimal regions of 17q gain and 4p loss being enriched significantly for such genes. These include genes involved in RNA and DNA metabolism, and apoptosis. Orthologues of all but one of these genes on 17q are overexpressed in rodent neuroblastoma cell lines. A significant excess of SNPs whose copy number correlates with survival is also observed on proximal 4p in stage 4 tumours, and we find that deletion of 4p is associated with improved outcome in an extended cohort of tumours. These results define the major impact of genomic copy number alterations upon transcription within neuroblastoma, and highlight genes on distal 17q and proximal 4p for downstream analyses. They also suggest that integration of discriminators, such as survival and comparative gene expression, with microarray data may be useful in the identification of critical genes within regions of loss or gain in many human cancers.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M1375">M1375</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:2355">ncbigene:2355</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:64080">ncbigene:64080</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:51646">ncbigene:51646</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1653">ncbigene:1653</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:151354">ncbigene:151354</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:23175">ncbigene:23175</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10130">ncbigene:10130</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9913">ncbigene:9913</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:79745">ncbigene:79745</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8462">ncbigene:8462</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:54221">ncbigene:54221</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:23620">ncbigene:23620</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:101060391">ncbigene:101060391</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:22950">ncbigene:22950</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:100506405">ncbigene:100506405</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:151056">ncbigene:151056</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4613">ncbigene:4613</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9553">ncbigene:9553</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3241">ncbigene:3241</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:238">ncbigene:238</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:81553">ncbigene:81553</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:11321">ncbigene:11321</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:23160">ncbigene:23160</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7173">ncbigene:7173</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:26160">ncbigene:26160</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:79635">ncbigene:79635</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3754">ncbigene:3754</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:55006">ncbigene:55006</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:130814">ncbigene:130814</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:101929715">ncbigene:101929715</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9014">ncbigene:9014</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6241">ncbigene:6241</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9687">ncbigene:9687</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:112267877">ncbigene:112267877</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:84450">ncbigene:84450</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:28951">ncbigene:28951</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4953">ncbigene:4953</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10408">ncbigene:10408</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:29841">ncbigene:29841</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:51594">ncbigene:51594</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:79954">ncbigene:79954</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9475">ncbigene:9475</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:64838">ncbigene:64838</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M1375&#39;, name=&#39;LASTOWSKA_COAMPLIFIED_WITH_MYCN&#39;), definition=&#39;Identifying genes, whose expression is consistently altered by chromosomal gains or losses, is an important step in defining genes of biological relevance in a wide variety of tumour types. However, additional criteria are needed to discriminate further among the large number of candidate genes identified. This is particularly true for neuroblastoma, where multiple genomic copy number changes of proven prognostic value exist. We have used Affymetrix microarrays and a combination of fluorescent in situ hybridization and single nucleotide polymorphism (SNP) microarrays to establish expression profiles and delineate copy number alterations in 30 primary neuroblastomas. Correlation of microarray data with patient survival and analysis of expression within rodent neuroblastoma cell lines were then used to define further genes likely to be involved in the disease process. Using this approach, we identify &gt;1000 genes within eight recurrent genomic alterations (loss of 1p, 3p, 4p, 10q and 11q, 2p gain, 17q gain, and the MYCN amplicon) whose expression is consistently altered by copy number change. Of these, 84 correlate with patient survival, with the minimal regions of 17q gain and 4p loss being enriched significantly for such genes. These include genes involved in RNA and DNA metabolism, and apoptosis. Orthologues of all but one of these genes on 17q are overexpressed in rodent neuroblastoma cell lines. A significant excess of SNPs whose copy number correlates with survival is also observed on proximal 4p in stage 4 tumours, and we find that deletion of 4p is associated with improved outcome in an extended cohort of tumours. These results define the major impact of genomic copy number alterations upon transcription within neuroblastoma, and highlight genes on distal 17q and proximal 4p for downstream analyses. They also suggest that integration of discriminators, such as survival and comparative gene expression, with microarray data may be useful in the identification of critical genes within regions of loss or gain in many human cancers.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;17533364&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2355&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;64080&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51646&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1653&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;151354&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23175&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10130&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9913&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;79745&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8462&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;54221&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23620&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;101060391&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;22950&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;100506405&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;151056&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4613&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9553&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3241&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;238&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;81553&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;11321&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23160&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7173&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;26160&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;79635&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3754&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55006&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;130814&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;101929715&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9014&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6241&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9687&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;112267877&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;84450&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;28951&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4953&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10408&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;29841&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51594&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;79954&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9475&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;64838&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Leona Saunders&#39;], &#39;exact_source&#39;: [&#39;Table 2&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>