<!DOCTYPE html>
<html>
<head>
            <meta charset="utf-8">
            <meta name="viewport" content="width=device-width, initial-scale=1.0">
            <!-- Bootstrap -->
            <link
                    rel="stylesheet"
                    href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
                    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
                    crossorigin="anonymous"
            />
            <link
                    rel="stylesheet"
                    href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"
            />
            <link
                    rel="stylesheet"
                    href="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.css"
            />
            <!-- DataTables, see: https://datatables.net/examples/styling/bootstrap4.html-->
            <link
                    rel="stylesheet"
                    href="https://cdn.datatables.net/1.10.19/css/dataTables.bootstrap4.min.css"
            />
            <style type="text/css">
                h2 {
                    margin-top: 35px;
                }

                html, body {
                    height: 100%;
                }

                body {
                    display: flex;
                    flex-flow: column;
                }

                .footer {
                    margin-top: auto;
                    padding-top: 1em;
                    background-color: #f5f5f5;
                }

                nav[data-toggle="toc"] {
                    top: 42px;
                }
            </style>

        <script src="https://kit.fontawesome.com/4c86883252.js" crossorigin="anonymous"></script>
        <title>EHLERS_ANEUPLOIDY_DN</title>
</head>

<body data-spy="scroll" data-target="#toc">
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>


    <div class="container" style="margin-top: 50px; margin-bottom: 50px">
        <div class="card">
        <div class="card-header">
            EHLERS_ANEUPLOIDY_DN
        </div>
        <div class="card-body">
            
                <p>PURPOSE: Aneuploidy is a hallmark of cancer and is closely linked to metastasis and poor clinical outcome. Yet, the mechanisms leading to aneuploidy and its role in tumor progression remain poorly understood. The extensive and complex karyotypic abnormalities seen in many solid tumors could hinder the identification of pathogenetically relevant chromosomal alterations. Uveal melanoma is an attractive solid tumor for studying aneuploidy because it is a relatively homogeneous cancer that is highly metastatic and has low nonspecific chromosomal instability. EXPERIMENTAL DESIGN: Comparative genomic hybridization and gene expression profiling were used to analyze patterns of aneuploidy in 49 primary uveal melanomas. This analysis was supplemented by a review of cytogenetic findings in 336 published cases. RESULTS: Three prognostically significant tumor subgroups were identified based on the status of chromosomes 3 and 6p. Discrete patterns of chromosomal alterations accumulated in these three subgroups in a nonrandom temporal sequence. Poor clinical outcome was associated with early chromosomal alterations rather than overall aneuploidy. A gene expression signature associated with aneuploidy was enriched for genes involved in cell cycle regulation, centrosome function, and DNA damage repair. One of these genes was PTEN, a tumor suppressor and genomic integrity guardian, which was down-regulated in association with increasing aneuploidy (P = 0.003). CONCLUSIONS: The relationship between aneuploidy and poor prognosis may be determined by specific, pathogenetically relevant chromosomal alterations, rather than overall aneuploidy. Such alterations can be identified using integrative genomic methods and may provide insights for novel therapeutic approaches.</p>
            
            <dl>
                <dt>Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M2785">M2785</a></dd>
                
            </dl>


            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M2785&#39;, name=&#39;EHLERS_ANEUPLOIDY_DN&#39;), definition=&#39;PURPOSE: Aneuploidy is a hallmark of cancer and is closely linked to metastasis and poor clinical outcome. Yet, the mechanisms leading to aneuploidy and its role in tumor progression remain poorly understood. The extensive and complex karyotypic abnormalities seen in many solid tumors could hinder the identification of pathogenetically relevant chromosomal alterations. Uveal melanoma is an attractive solid tumor for studying aneuploidy because it is a relatively homogeneous cancer that is highly metastatic and has low nonspecific chromosomal instability. EXPERIMENTAL DESIGN: Comparative genomic hybridization and gene expression profiling were used to analyze patterns of aneuploidy in 49 primary uveal melanomas. This analysis was supplemented by a review of cytogenetic findings in 336 published cases. RESULTS: Three prognostically significant tumor subgroups were identified based on the status of chromosomes 3 and 6p. Discrete patterns of chromosomal alterations accumulated in these three subgroups in a nonrandom temporal sequence. Poor clinical outcome was associated with early chromosomal alterations rather than overall aneuploidy. A gene expression signature associated with aneuploidy was enriched for genes involved in cell cycle regulation, centrosome function, and DNA damage repair. One of these genes was PTEN, a tumor suppressor and genomic integrity guardian, which was down-regulated in association with increasing aneuploidy (P = 0.003). CONCLUSIONS: The relationship between aneuploidy and poor prognosis may be determined by specific, pathogenetically relevant chromosomal alterations, rather than overall aneuploidy. Such alterations can be identified using integrative genomic methods and may provide insights for novel therapeutic approaches.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;18172260&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8326&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5549&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2562&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9781&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;196527&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2766&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23413&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23632&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;137964&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4061&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51651&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;29097&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Jessica Robertson&#39;], &#39;exact_source&#39;: [&#39;Fig. 5A: DOWN in low aneuploidy, UP in high aneuploidy&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div>
    </div>


<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>


    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
            integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
            crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"
            integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
            crossorigin="anonymous"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
            integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
            crossorigin="anonymous"></script>
    <script src="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.js"></script>

    <!-- DataTables, see: https://datatables.net/examples/styling/bootstrap4.html-->
    <script src="https://cdn.datatables.net/1.10.19/js/jquery.dataTables.min.js"></script>
    <script src="https://cdn.datatables.net/1.10.19/js/dataTables.bootstrap4.min.js"></script>

    <script src="https://d3js.org/d3.v5.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.js"></script>

    <script>
        $(function () {
            $('[data-toggle="tooltip"]').tooltip()
        })
    </script>
</body>
</html>