<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>PYEON_CANCER_HEAD_AND_NECK_VS_CERVICAL_DN</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            PYEON_CANCER_HEAD_AND_NECK_VS_CERVICAL_DN
        </div>
        <div class="card-body">
                <p>Human papillomaviruses (HPV) are associated with nearly all cervical cancers, 20% to 30% of head and neck cancers (HNC), and other cancers. Because HNCs also arise in HPV-negative patients, this type of cancer provides unique opportunities to define similarities and differences of HPV-positive versus HPV-negative cancers arising in the same tissue. Here, we describe genome-wide expression profiling of 84 HNCs, cervical cancers, and site-matched normal epithelial samples in which we used laser capture microdissection to enrich samples for tumor-derived versus normal epithelial cells. This analysis revealed that HPV(+) HNCs and cervical cancers differed in their patterns of gene expression yet shared many changes compared with HPV(-) HNCs. Some of these shared changes were predicted, but many others were not. Notably, HPV(+) HNCs and cervical cancers were found to be up-regulated in their expression of a distinct and larger subset of cell cycle genes than that observed in HPV(-) HNC. Moreover, HPV(+) cancers overexpressed testis-specific genes that are normally expressed only in meiotic cells. Many, although not all, of the hallmark differences between HPV(+) HNC and HPV(-) HNC were a direct consequence of HPV and in particular the viral E6 and E7 oncogenes. This included a novel association of HPV oncogenes with testis-specific gene expression. These findings in primary human tumors provide novel biomarkers for early detection of HPV(+) and HPV(-) cancers, and emphasize the potential value of targeting E6 and E7 function, alone or combined with radiation and/or traditional chemotherapy, in the treatment of HPV(+) cancers.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M5126">M5126</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:246126">ncbigene:246126</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9638">ncbigene:9638</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9086">ncbigene:9086</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6192">ncbigene:6192</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8284">ncbigene:8284</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8287">ncbigene:8287</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:131578">ncbigene:131578</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5650">ncbigene:5650</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1041">ncbigene:1041</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:83690">ncbigene:83690</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7850">ncbigene:7850</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:151516">ncbigene:151516</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:11202">ncbigene:11202</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10630">ncbigene:10630</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:84466">ncbigene:84466</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7544">ncbigene:7544</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:55117">ncbigene:55117</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4111">ncbigene:4111</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:338324">ncbigene:338324</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1823">ncbigene:1823</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:283120">ncbigene:283120</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8653">ncbigene:8653</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4703">ncbigene:4703</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1673">ncbigene:1673</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1634">ncbigene:1634</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:84059">ncbigene:84059</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1805">ncbigene:1805</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:23643">ncbigene:23643</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:388533">ncbigene:388533</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M5126&#39;, name=&#39;PYEON_CANCER_HEAD_AND_NECK_VS_CERVICAL_DN&#39;), definition=&#39;Human papillomaviruses (HPV) are associated with nearly all cervical cancers, 20% to 30% of head and neck cancers (HNC), and other cancers. Because HNCs also arise in HPV-negative patients, this type of cancer provides unique opportunities to define similarities and differences of HPV-positive versus HPV-negative cancers arising in the same tissue. Here, we describe genome-wide expression profiling of 84 HNCs, cervical cancers, and site-matched normal epithelial samples in which we used laser capture microdissection to enrich samples for tumor-derived versus normal epithelial cells. This analysis revealed that HPV(+) HNCs and cervical cancers differed in their patterns of gene expression yet shared many changes compared with HPV(-) HNCs. Some of these shared changes were predicted, but many others were not. Notably, HPV(+) HNCs and cervical cancers were found to be up-regulated in their expression of a distinct and larger subset of cell cycle genes than that observed in HPV(-) HNC. Moreover, HPV(+) cancers overexpressed testis-specific genes that are normally expressed only in meiotic cells. Many, although not all, of the hallmark differences between HPV(+) HNC and HPV(-) HNC were a direct consequence of HPV and in particular the viral E6 and E7 oncogenes. This included a novel association of HPV oncogenes with testis-specific gene expression. These findings in primary human tumors provide novel biomarkers for early detection of HPV(+) and HPV(-) cancers, and emphasize the potential value of targeting E6 and E7 function, alone or combined with radiation and/or traditional chemotherapy, in the treatment of HPV(+) cancers.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;17510386&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;246126&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9638&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9086&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6192&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8284&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8287&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;131578&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5650&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1041&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;83690&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7850&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;151516&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;11202&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10630&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;84466&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7544&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55117&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4111&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;338324&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1823&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;283120&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8653&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4703&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1673&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1634&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;84059&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1805&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23643&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;388533&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Arthur Liberzon&#39;], &#39;exact_source&#39;: [&#39;Table 6S: t-statistic &lt; -4&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>