<!DOCTYPE html>
<html>
<head>
            <meta charset="utf-8">
            <meta name="viewport" content="width=device-width, initial-scale=1.0">
            <!-- Bootstrap -->
            <link
                    rel="stylesheet"
                    href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
                    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
                    crossorigin="anonymous"
            />
            <link
                    rel="stylesheet"
                    href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"
            />
            <link
                    rel="stylesheet"
                    href="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.css"
            />
            <!-- DataTables, see: https://datatables.net/examples/styling/bootstrap4.html-->
            <link
                    rel="stylesheet"
                    href="https://cdn.datatables.net/1.10.19/css/dataTables.bootstrap4.min.css"
            />
            <style type="text/css">
                h2 {
                    margin-top: 35px;
                }

                html, body {
                    height: 100%;
                }

                body {
                    display: flex;
                    flex-flow: column;
                }

                .footer {
                    margin-top: auto;
                    padding-top: 1em;
                    background-color: #f5f5f5;
                }

                nav[data-toggle="toc"] {
                    top: 42px;
                }
            </style>

        <script src="https://kit.fontawesome.com/4c86883252.js" crossorigin="anonymous"></script>
        <title>ROYLANCE_BREAST_CANCER_16Q_COPY_NUMBER_DN</title>
</head>

<body data-spy="scroll" data-target="#toc">
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>


    <div class="container" style="margin-top: 50px; margin-bottom: 50px">
        <div class="card">
        <div class="card-header">
            ROYLANCE_BREAST_CANCER_16Q_COPY_NUMBER_DN
        </div>
        <div class="card-body">
            
                <p>We analysed chromosome 16q in 106 breast cancers using tiling-path array-comparative genomic hybridization (aCGH). About 80% of ductal cancers (IDCs) and all lobular cancers (ILCs) lost at least part of 16q. Grade I (GI) IDCs and ILCs often lost the whole chromosome arm. Grade II (GII) and grade III (GIII) IDCs showed less frequent whole-arm loss, but often had complex changes, typically small regions of gain together with larger regions of loss. The boundaries of gains/losses tended to cluster, common sites being 54.5-55.5 Mb and 57.4-58.8 Mb. Overall, the peak frequency of loss (83% cancers) occurred at 61.9-62.9 Mb. We also found several &#39;minimal&#39; regions of loss/gain. However, no mutations in candidate genes (TRADD, CDH5, CDH8 and CDH11) were detected. Cluster analysis based on copy number changes identified a large group of cancers that had lost most of 16q, and two smaller groups (one with few changes, one with a tendency to show copy number gain). Although all morphological types occurred in each cluster group, IDCs (especially GII/GIII) were relatively overrepresented in the smaller groups. Cluster groups were not independently associated with survival. Use of tiling-path aCGH prompted re-evaluation of the hypothetical pathways of breast carcinogenesis. ILCs have the simplest changes on 16q and probably diverge from the IDC lineage close to the stage of 16q loss. Higher-grade IDCs probably develop from low-grade lesions in most cases, but there remains evidence that some GII/GIII IDCs arise without a GI precursor.</p>
            
            <dl>
                <dt>Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M8224">M8224</a></dd>
                
            </dl>


            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M8224&#39;, name=&#39;ROYLANCE_BREAST_CANCER_16Q_COPY_NUMBER_DN&#39;), definition=&#34;We analysed chromosome 16q in 106 breast cancers using tiling-path array-comparative genomic hybridization (aCGH). About 80% of ductal cancers (IDCs) and all lobular cancers (ILCs) lost at least part of 16q. Grade I (GI) IDCs and ILCs often lost the whole chromosome arm. Grade II (GII) and grade III (GIII) IDCs showed less frequent whole-arm loss, but often had complex changes, typically small regions of gain together with larger regions of loss. The boundaries of gains/losses tended to cluster, common sites being 54.5-55.5 Mb and 57.4-58.8 Mb. Overall, the peak frequency of loss (83% cancers) occurred at 61.9-62.9 Mb. We also found several &#39;minimal&#39; regions of loss/gain. However, no mutations in candidate genes (TRADD, CDH5, CDH8 and CDH11) were detected. Cluster analysis based on copy number changes identified a large group of cancers that had lost most of 16q, and two smaller groups (one with few changes, one with a tendency to show copy number gain). Although all morphological types occurred in each cluster group, IDCs (especially GII/GIII) were relatively overrepresented in the smaller groups. Cluster groups were not independently associated with survival. Use of tiling-path aCGH prompted re-evaluation of the hypothetical pathways of breast carcinogenesis. ILCs have the simplest changes on 16q and probably diverge from the IDC lineage close to the stage of 16q loss. Higher-grade IDCs probably develop from low-grade lesions in most cases, but there remains evidence that some GII/GIII IDCs arise without a GI precursor.&#34;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;16702952&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23594&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;84290&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;54947&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4313&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;81831&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;81533&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5257&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4094&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;79801&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55737&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;85445&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;388272&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2775&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10294&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;84706&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1006&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51741&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;91807&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;124359&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1066&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;221223&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6530&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;83657&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;124149&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;79190&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51716&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Arthur Liberzon&#39;], &#39;exact_source&#39;: [&#39;Table 2&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div>
    </div>


<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>


    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
            integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
            crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"
            integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
            crossorigin="anonymous"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
            integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
            crossorigin="anonymous"></script>
    <script src="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.js"></script>

    <!-- DataTables, see: https://datatables.net/examples/styling/bootstrap4.html-->
    <script src="https://cdn.datatables.net/1.10.19/js/jquery.dataTables.min.js"></script>
    <script src="https://cdn.datatables.net/1.10.19/js/dataTables.bootstrap4.min.js"></script>

    <script src="https://d3js.org/d3.v5.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.js"></script>

    <script>
        $(function () {
            $('[data-toggle="tooltip"]').tooltip()
        })
    </script>
</body>
</html>