<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>TURASHVILI_BREAST_NORMAL_DUCTAL_VS_LOBULAR_DN</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            TURASHVILI_BREAST_NORMAL_DUCTAL_VS_LOBULAR_DN
        </div>
        <div class="card-body">
                <p>BACKGROUND: Invasive ductal and lobular carcinomas (IDC and ILC) are the most common histological types of breast cancer. Clinical follow-up data and metastatic patterns suggest that the development and progression of these tumors are different. The aim of our study was to identify gene expression profiles of IDC and ILC in relation to normal breast epithelial cells. METHODS: We examined 30 samples (normal ductal and lobular cells from 10 patients, IDC cells from 5 patients, ILC cells from 5 patients) microdissected from cryosections of ten mastectomy specimens from postmenopausal patients. Fifty nanograms of total RNA were amplified and labeled by PCR and in vitro transcription. Samples were analysed upon Affymetrix U133 Plus 2.0 Arrays. The expression of seven differentially expressed genes (CDH1, EMP1, DDR1, DVL1, KRT5, KRT6, KRT17) was verified by immunohistochemistry on tissue microarrays. Expression of ASPN mRNA was validated by in situ hybridization on frozen sections, and CTHRC1, ASPN and COL3A1 were tested by PCR. RESULTS: Using GCOS pairwise comparison algorithm and rank products we have identified 84 named genes common to ILC versus normal cell types, 74 named genes common to IDC versus normal cell types, 78 named genes differentially expressed between normal ductal and lobular cells, and 28 named genes between IDC and ILC. Genes distinguishing between IDC and ILC are involved in epithelial-mesenchymal transition, TGF-beta and Wnt signaling. These changes were present in both tumor types but appeared to be more prominent in ILC. Immunohistochemistry for several novel markers (EMP1, DVL1, DDR1) distinguished large sets of IDC from ILC. CONCLUSION: IDC and ILC can be differentiated both at the gene and protein levels. In this study we report two candidate genes, asporin (ASPN) and collagen triple helix repeat containing 1 (CTHRC1) which might be significant in breast carcinogenesis. Besides E-cadherin, the proteins validated on tissue microarrays (EMP1, DVL1, DDR1) may represent novel immunohistochemical markers helpful in distinguishing between IDC and ILC. Further studies with larger sets of patients are needed to verify the gene expression profiles of various histological types of breast cancer in order to determine molecular subclassifications, prognosis and the optimum treatment strategies.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M13172">M13172</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:10098">ncbigene:10098</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:51360">ncbigene:51360</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:22836">ncbigene:22836</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:23301">ncbigene:23301</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:51351">ncbigene:51351</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:23229">ncbigene:23229</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1634">ncbigene:1634</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M13172&#39;, name=&#39;TURASHVILI_BREAST_NORMAL_DUCTAL_VS_LOBULAR_DN&#39;), definition=&#39;BACKGROUND: Invasive ductal and lobular carcinomas (IDC and ILC) are the most common histological types of breast cancer. Clinical follow-up data and metastatic patterns suggest that the development and progression of these tumors are different. The aim of our study was to identify gene expression profiles of IDC and ILC in relation to normal breast epithelial cells. METHODS: We examined 30 samples (normal ductal and lobular cells from 10 patients, IDC cells from 5 patients, ILC cells from 5 patients) microdissected from cryosections of ten mastectomy specimens from postmenopausal patients. Fifty nanograms of total RNA were amplified and labeled by PCR and in vitro transcription. Samples were analysed upon Affymetrix U133 Plus 2.0 Arrays. The expression of seven differentially expressed genes (CDH1, EMP1, DDR1, DVL1, KRT5, KRT6, KRT17) was verified by immunohistochemistry on tissue microarrays. Expression of ASPN mRNA was validated by in situ hybridization on frozen sections, and CTHRC1, ASPN and COL3A1 were tested by PCR. RESULTS: Using GCOS pairwise comparison algorithm and rank products we have identified 84 named genes common to ILC versus normal cell types, 74 named genes common to IDC versus normal cell types, 78 named genes differentially expressed between normal ductal and lobular cells, and 28 named genes between IDC and ILC. Genes distinguishing between IDC and ILC are involved in epithelial-mesenchymal transition, TGF-beta and Wnt signaling. These changes were present in both tumor types but appeared to be more prominent in ILC. Immunohistochemistry for several novel markers (EMP1, DVL1, DDR1) distinguished large sets of IDC from ILC. CONCLUSION: IDC and ILC can be differentiated both at the gene and protein levels. In this study we report two candidate genes, asporin (ASPN) and collagen triple helix repeat containing 1 (CTHRC1) which might be significant in breast carcinogenesis. Besides E-cadherin, the proteins validated on tissue microarrays (EMP1, DVL1, DDR1) may represent novel immunohistochemical markers helpful in distinguishing between IDC and ILC. Further studies with larger sets of patients are needed to verify the gene expression profiles of various histological types of breast cancer in order to determine molecular subclassifications, prognosis and the optimum treatment strategies.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;17389037&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10098&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51360&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;22836&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23301&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51351&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23229&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1634&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Arthur Liberzon&#39;], &#39;exact_source&#39;: [&#39;Table 1S&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>