<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>BOGNI_TREATMENT_RELATED_MYELOID_LEUKEMIA_UP</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            BOGNI_TREATMENT_RELATED_MYELOID_LEUKEMIA_UP
        </div>
        <div class="card-body">
                <p>Using a target gene approach, only a few host genetic risk factors for treatment-related myeloid leukemia (t-ML) have been defined. Gene expression microarrays allow for a more genome-wide approach to assess possible genetic risk factors for t-ML. We assessed gene expression profiles (n=12 625 probe sets) in diagnostic acute lymphoblastic leukemic cells from 228 children treated on protocols that included leukemogenic agents such as etoposide, 13 of whom developed t-ML. Expression of 68 probes, corresponding to 63 genes, was significantly related to risk of t-ML. Hierarchical clustering of these probe sets clustered patients into three groups with 94, 122 and 12 patients, respectively; 12 of the 13 patients who went on to develop t-ML were overrepresented in the latter group (P&lt;0.0001). A permutation test indicated a low likelihood that these probe sets and clusters were obtained by chance (P&lt;0.001). Distinguishing genes included transcription-related oncogenes (v-Myb, Pax-5), cyclins (CCNG1, CCNG2 and CCND1) and histone HIST1H4C. Common transcription factor recognition elements among similarly up- or downregulated genes included several involved in hematopoietic differentiation or leukemogenesis (Maz, PU.1, ARNT). This approach has identified several genes whose expression distinguishes patients at risk of t-ML, and suggests targets for assessing germline predisposition to leukemogenesis.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M14736">M14736</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:3577">ncbigene:3577</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5430">ncbigene:5430</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5199">ncbigene:5199</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10500">ncbigene:10500</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6233">ncbigene:6233</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2560">ncbigene:2560</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8192">ncbigene:8192</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5604">ncbigene:5604</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6273">ncbigene:6273</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1388">ncbigene:1388</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6139">ncbigene:6139</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5230">ncbigene:5230</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2202">ncbigene:2202</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:23660">ncbigene:23660</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8879">ncbigene:8879</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5079">ncbigene:5079</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8364">ncbigene:8364</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10294">ncbigene:10294</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3821">ncbigene:3821</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6554">ncbigene:6554</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4882">ncbigene:4882</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3240">ncbigene:3240</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2316">ncbigene:2316</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:293">ncbigene:293</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4357">ncbigene:4357</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:113177">ncbigene:113177</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6135">ncbigene:6135</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:388714">ncbigene:388714</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5986">ncbigene:5986</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1943">ncbigene:1943</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2035">ncbigene:2035</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M14736&#39;, name=&#39;BOGNI_TREATMENT_RELATED_MYELOID_LEUKEMIA_UP&#39;), definition=&#39;Using a target gene approach, only a few host genetic risk factors for treatment-related myeloid leukemia (t-ML) have been defined. Gene expression microarrays allow for a more genome-wide approach to assess possible genetic risk factors for t-ML. We assessed gene expression profiles (n=12 625 probe sets) in diagnostic acute lymphoblastic leukemic cells from 228 children treated on protocols that included leukemogenic agents such as etoposide, 13 of whom developed t-ML. Expression of 68 probes, corresponding to 63 genes, was significantly related to risk of t-ML. Hierarchical clustering of these probe sets clustered patients into three groups with 94, 122 and 12 patients, respectively; 12 of the 13 patients who went on to develop t-ML were overrepresented in the latter group (P&lt;0.0001). A permutation test indicated a low likelihood that these probe sets and clusters were obtained by chance (P&lt;0.001). Distinguishing genes included transcription-related oncogenes (v-Myb, Pax-5), cyclins (CCNG1, CCNG2 and CCND1) and histone HIST1H4C. Common transcription factor recognition elements among similarly up- or downregulated genes included several involved in hematopoietic differentiation or leukemogenesis (Maz, PU.1, ARNT). This approach has identified several genes whose expression distinguishes patients at risk of t-ML, and suggests targets for assessing germline predisposition to leukemogenesis.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;16341039&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3577&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5430&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5199&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10500&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6233&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2560&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8192&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5604&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6273&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1388&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6139&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5230&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2202&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23660&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8879&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5079&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8364&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10294&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3821&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6554&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4882&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3240&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2316&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;293&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4357&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;113177&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6135&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;388714&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5986&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1943&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2035&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Arthur Liberzon&#39;], &#39;exact_source&#39;: [&#39;Table 1: Relative ratio &gt; 0&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>