<!DOCTYPE html>
<html>
<head>
            <meta charset="utf-8">
            <meta name="viewport" content="width=device-width, initial-scale=1.0">
            <!-- Bootstrap -->
            <link
                    rel="stylesheet"
                    href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
                    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
                    crossorigin="anonymous"
            />
            <link
                    rel="stylesheet"
                    href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"
            />
            <link
                    rel="stylesheet"
                    href="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.css"
            />
            <!-- DataTables, see: https://datatables.net/examples/styling/bootstrap4.html-->
            <link
                    rel="stylesheet"
                    href="https://cdn.datatables.net/1.10.19/css/dataTables.bootstrap4.min.css"
            />
            <style type="text/css">
                h2 {
                    margin-top: 35px;
                }

                html, body {
                    height: 100%;
                }

                body {
                    display: flex;
                    flex-flow: column;
                }

                .footer {
                    margin-top: auto;
                    padding-top: 1em;
                    background-color: #f5f5f5;
                }

                nav[data-toggle="toc"] {
                    top: 42px;
                }
            </style>

        <script src="https://kit.fontawesome.com/4c86883252.js" crossorigin="anonymous"></script>
        <title>BIOCARTA_GLEEVEC_PATHWAY</title>
</head>

<body data-spy="scroll" data-target="#toc">
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>


    <div class="container" style="margin-top: 50px; margin-bottom: 50px">
        <div class="card">
        <div class="card-header">
            BIOCARTA_GLEEVEC_PATHWAY
        </div>
        <div class="card-body">
            
                <p>The drug Gleevec (also known as imatinib mesylate or STI-571) was approved by the FDA in 2001 for the treatment of CML, chronic myeloid leukemia. While traditional cytotoxic cancer treatments such as chemotherapy or radiation therapy kill all dividing cells, Gleevec acts on a molecular target by a mechanism that is more specific to cancer cells. Traditional cytotoxic cancer agents have serious side effects such as nausea, weight loss, hair loss and severe fatigue that result from their lack of specificity in killing cells. Gleevec was designed as an inhibitor of a specific receptor associated with CML, and so produces less severe side effects than other cancer agents. CML is associated in most cases with a specific chromosomal defect, a translocation between chromosomes 9 and 22 that creates the Philadelphia chromosome. This translocation occurs at the site in the genome of a protein tyrosine kinase called abl, creating the abnormal bcr-abl protein, a fusion of the abl gene with another gene called bcr. The kinase activity of abl in the bcr-abl fusion is activated and unregulated, driving the uncontrolled cell growth observed in CML. White blood cells containing the bcr-abl mutation become able to proliferate in the absence of growth factors they normally require. Gleevec inhibits abl kinase activity, helping to reverse uncontrolled cell growth. Gleevec also inhibits the PDGF tyrosine kinase and the c-kit tyrosine kinase. There are a variety of cellular substrates of the bcr-abl kinase that may be involved in cellular transformation. Bcr-abl is associated with the cytoplasm as part of a large signaling complex. Some of the downstream factors in bcr-abl signaling include PI3 kinase/AKT and STAT transcription factors. The activation of bcr-abl also represses apoptosis through induction of anti-apoptosis factors such as Bad, allowing transformed cells to divide. JAK2 kinase activity appears to be one target of bcr-abl. Grb-2 phosphorylation by bcr-abl may play a role in down-regulation of tyrosine kinase signaling. STAT5 may be involved in the failure of apoptosis in bcr-abl cells. In addition to supporting the idea that cancer therapies targeting specific molecular targets should be efficacious with fewer side effects, Gleevec has also demonstrated that drugs inhibiting protein kinases can be developed successfully. Tyrosine kinases are important in a range of cellular processes, including other cancers, and will provide additional drug targets. Gleevec itself has already demonstrated potential in other cancers such as gastrointestinal stromal tumor that do not respond to existing treatments. Although Gleevec has produced very strong clinical responses in patients with early stage CML, patients with late stage disease have had an initial response followed by a relapse of drug resistant CML cells. Cancer cells in patients with Gleevec resistant cancer either had amplification of the bcr-abl gene, or mutation of a key amino acid involved in binding drug from threonine to isoleucine.</p>
            
            <dl>
                <dt>Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M7897">M7897</a></dd>
                
            </dl>


            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M7897&#39;, name=&#39;BIOCARTA_GLEEVEC_PATHWAY&#39;), definition=&#39;The drug Gleevec (also known as imatinib mesylate or STI-571) was approved by the FDA in 2001 for the treatment of CML, chronic myeloid leukemia. While traditional cytotoxic cancer treatments such as chemotherapy or radiation therapy kill all dividing cells, Gleevec acts on a molecular target by a mechanism that is more specific to cancer cells. Traditional cytotoxic cancer agents have serious side effects such as nausea, weight loss, hair loss and severe fatigue that result from their lack of specificity in killing cells. Gleevec was designed as an inhibitor of a specific receptor associated with CML, and so produces less severe side effects than other cancer agents. CML is associated in most cases with a specific chromosomal defect, a translocation between chromosomes 9 and 22 that creates the Philadelphia chromosome. This translocation occurs at the site in the genome of a protein tyrosine kinase called abl, creating the abnormal bcr-abl protein, a fusion of the abl gene with another gene called bcr. The kinase activity of abl in the bcr-abl fusion is activated and unregulated, driving the uncontrolled cell growth observed in CML. White blood cells containing the bcr-abl mutation become able to proliferate in the absence of growth factors they normally require. Gleevec inhibits abl kinase activity, helping to reverse uncontrolled cell growth. Gleevec also inhibits the PDGF tyrosine kinase and the c-kit tyrosine kinase. There are a variety of cellular substrates of the bcr-abl kinase that may be involved in cellular transformation. Bcr-abl is associated with the cytoplasm as part of a large signaling complex. Some of the downstream factors in bcr-abl signaling include PI3 kinase/AKT and STAT transcription factors. The activation of bcr-abl also represses apoptosis through induction of anti-apoptosis factors such as Bad, allowing transformed cells to divide. JAK2 kinase activity appears to be one target of bcr-abl. Grb-2 phosphorylation by bcr-abl may play a role in down-regulation of tyrosine kinase signaling. STAT5 may be involved in the failure of apoptosis in bcr-abl cells. In addition to supporting the idea that cancer therapies targeting specific molecular targets should be efficacious with fewer side effects, Gleevec has also demonstrated that drugs inhibiting protein kinases can be developed successfully. Tyrosine kinases are important in a range of cellular processes, including other cancers, and will provide additional drug targets. Gleevec itself has already demonstrated potential in other cancers such as gastrointestinal stromal tumor that do not respond to existing treatments. Although Gleevec has produced very strong clinical responses in patients with early stage CML, patients with late stage disease have had an initial response followed by a relapse of drug resistant CML cells. Cancer cells in patients with Gleevec resistant cancer either had amplification of the bcr-abl gene, or mutation of a key amino acid involved in binding drug from threonine to isoleucine.&#39;, provenance=[], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5595&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1399&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5294&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3265&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5295&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5599&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5604&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5894&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4609&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;572&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6776&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2885&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6416&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;613&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;207&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5290&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6777&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4214&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3717&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3725&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6654&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2353&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6772&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CP:BIOCARTA&#39;], &#39;contributor&#39;: [&#39;BioCarta&#39;], &#39;external_details_url&#39;: [&#39;https://data.broadinstitute.org/gsea-msigdb/msigdb/biocarta/human/h_gleevecPathway.gif&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div>
    </div>


<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>


    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
            integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
            crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"
            integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
            crossorigin="anonymous"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
            integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
            crossorigin="anonymous"></script>
    <script src="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.js"></script>

    <!-- DataTables, see: https://datatables.net/examples/styling/bootstrap4.html-->
    <script src="https://cdn.datatables.net/1.10.19/js/jquery.dataTables.min.js"></script>
    <script src="https://cdn.datatables.net/1.10.19/js/dataTables.bootstrap4.min.js"></script>

    <script src="https://d3js.org/d3.v5.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.js"></script>

    <script>
        $(function () {
            $('[data-toggle="tooltip"]').tooltip()
        })
    </script>
</body>
</html>