<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>COLLER_MYC_TARGETS_UP</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            COLLER_MYC_TARGETS_UP
        </div>
        <div class="card-body">
                <p>MYC affects normal and neoplastic cell proliferation by altering gene expression, but the precise pathways remain unclear. We used oligonucleotide microarray analysis of 6,416 genes and expressed sequence tags to determine changes in gene expression caused by activation of c-MYC in primary human fibroblasts. In these experiments, 27 genes were consistently induced, and 9 genes were repressed. The identity of the genes revealed that MYC may affect many aspects of cell physiology altered in transformed cells: cell growth, cell cycle, adhesion, and cytoskeletal organization. Identified targets possibly linked to MYC&#39;s effects on cell growth include the nucleolar proteins nucleolin and fibrillarin, as well as the eukaryotic initiation factor 5A. Among the cell cycle genes identified as targets, the G1 cyclin D2 and the cyclin-dependent kinase binding protein CksHs2 were induced whereas the cyclin-dependent kinase inhibitor p21(Cip1) was repressed. A role for MYC in regulating cell adhesion and structure is suggested by repression of genes encoding the extracellular matrix proteins fibronectin and collagen, and the cytoskeletal protein tropomyosin. A possible mechanism for MYC-mediated apoptosis was revealed by identification of the tumor necrosis factor receptor associated protein TRAP1 as a MYC target. Finally, two immunophilins, peptidyl-prolyl cis-trans isomerase F and FKBP52, the latter of which plays a role in cell division in Arabidopsis, were up-regulated by MYC. We also explored pattern-matching methods as an alternative approach for identifying MYC target genes. The genes that displayed an expression profile most similar to endogenous Myc in microarray-based expression profiling of myeloid differentiation models were highly enriched for MYC target genes.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M5955">M5955</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:7037">ncbigene:7037</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:80273">ncbigene:80273</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2091">ncbigene:2091</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2821">ncbigene:2821</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3068">ncbigene:3068</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10135">ncbigene:10135</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1164">ncbigene:1164</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:191">ncbigene:191</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6566">ncbigene:6566</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4953">ncbigene:4953</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10131">ncbigene:10131</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2171">ncbigene:2171</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5437">ncbigene:5437</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:50486">ncbigene:50486</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3329">ncbigene:3329</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:445">ncbigene:445</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10105">ncbigene:10105</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10153">ncbigene:10153</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:708">ncbigene:708</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1984">ncbigene:1984</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:205">ncbigene:205</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3376">ncbigene:3376</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:894">ncbigene:894</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4691">ncbigene:4691</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M5955&#39;, name=&#39;COLLER_MYC_TARGETS_UP&#39;), definition=&#34;MYC affects normal and neoplastic cell proliferation by altering gene expression, but the precise pathways remain unclear. We used oligonucleotide microarray analysis of 6,416 genes and expressed sequence tags to determine changes in gene expression caused by activation of c-MYC in primary human fibroblasts. In these experiments, 27 genes were consistently induced, and 9 genes were repressed. The identity of the genes revealed that MYC may affect many aspects of cell physiology altered in transformed cells: cell growth, cell cycle, adhesion, and cytoskeletal organization. Identified targets possibly linked to MYC&#39;s effects on cell growth include the nucleolar proteins nucleolin and fibrillarin, as well as the eukaryotic initiation factor 5A. Among the cell cycle genes identified as targets, the G1 cyclin D2 and the cyclin-dependent kinase binding protein CksHs2 were induced whereas the cyclin-dependent kinase inhibitor p21(Cip1) was repressed. A role for MYC in regulating cell adhesion and structure is suggested by repression of genes encoding the extracellular matrix proteins fibronectin and collagen, and the cytoskeletal protein tropomyosin. A possible mechanism for MYC-mediated apoptosis was revealed by identification of the tumor necrosis factor receptor associated protein TRAP1 as a MYC target. Finally, two immunophilins, peptidyl-prolyl cis-trans isomerase F and FKBP52, the latter of which plays a role in cell division in Arabidopsis, were up-regulated by MYC. We also explored pattern-matching methods as an alternative approach for identifying MYC target genes. The genes that displayed an expression profile most similar to endogenous Myc in microarray-based expression profiling of myeloid differentiation models were highly enriched for MYC target genes.&#34;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;10737792&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7037&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;80273&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2091&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2821&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3068&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10135&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1164&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;191&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6566&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4953&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10131&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2171&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5437&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;50486&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3329&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;445&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10105&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10153&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;708&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1984&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;205&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3376&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;894&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4691&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Yujin Hoshida&#39;], &#39;exact_source&#39;: [&#39;Table 1: induced genes&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>