<!DOCTYPE html>
<html>
<head>
            <meta charset="utf-8">
            <meta name="viewport" content="width=device-width, initial-scale=1.0">
            <!-- Bootstrap -->
            <link
                    rel="stylesheet"
                    href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
                    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
                    crossorigin="anonymous"
            />
            <link
                    rel="stylesheet"
                    href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"
            />
            <link
                    rel="stylesheet"
                    href="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.css"
            />
            <!-- DataTables, see: https://datatables.net/examples/styling/bootstrap4.html-->
            <link
                    rel="stylesheet"
                    href="https://cdn.datatables.net/1.10.19/css/dataTables.bootstrap4.min.css"
            />
            <style type="text/css">
                h2 {
                    margin-top: 35px;
                }

                html, body {
                    height: 100%;
                }

                body {
                    display: flex;
                    flex-flow: column;
                }

                .footer {
                    margin-top: auto;
                    padding-top: 1em;
                    background-color: #f5f5f5;
                }

                nav[data-toggle="toc"] {
                    top: 42px;
                }
            </style>

        <script src="https://kit.fontawesome.com/4c86883252.js" crossorigin="anonymous"></script>
        <title>KOBAYASHI_EGFR_SIGNALING_6HR_UP</title>
</head>

<body data-spy="scroll" data-target="#toc">
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>


    <div class="container" style="margin-top: 50px; margin-bottom: 50px">
        <div class="card">
        <div class="card-header">
            KOBAYASHI_EGFR_SIGNALING_6HR_UP
        </div>
        <div class="card-body">
            
                <p>Activating mutations in the epidermal growth factor receptor (EGFR) tyrosine kinase domain determine responsiveness to EGFR tyrosine kinase inhibitors in patients with advanced non-small cell lung cancer (NSCLC). The modulation of transcriptional pathways by mutant EGFR signaling is not fully understood. Previously, we and others identified a single base pair change leading to a threonine to methionine (T790M) amino acid alteration in the ATP-binding pocket of the EGFR as a common mechanism of acquired resistance. The gefitinib-resistant, T790M-mutant H1975 NSCLC cell line undergoes prominent growth arrest and apoptosis when treated with the irreversible EGFR inhibitor, CL-387,785. We did a transcriptional profiling study of mutant EGFR target genes that are differentially expressed in the resistant gefitinib-treated and the sensitive CL387,785-treated H1975 cells to identify the pivotal transcriptional changes in NSCLC with EGFR-activating mutations. We identified a small subset of early gene changes, including significant reduction of cyclin D1 as a result of EGFR inhibition by CL-387,785 but not by gefitinib. The reduction in cyclin D1 transcription was associated with subsequent suppression of E2F-responsive genes, consistent with proliferation arrest. Furthermore, cyclin D1 expression was higher in EGFR-mutant lung cancer cells compared with cells with wild-type EGFR. EGFR-mutant cells were routinely sensitive to the cyclin-dependent kinase inhibitor flavopiridol, confirming the functional relevance of the cyclin D axis. These studies suggest that cyclin D1 may contribute to the emergence of EGFR-driven tumorigenesis and can be an alternative target of therapy.</p>
            
            <dl>
                <dt>Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M7331">M7331</a></dd>
                
            </dl>


            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M7331&#39;, name=&#39;KOBAYASHI_EGFR_SIGNALING_6HR_UP&#39;), definition=&#39;Activating mutations in the epidermal growth factor receptor (EGFR) tyrosine kinase domain determine responsiveness to EGFR tyrosine kinase inhibitors in patients with advanced non-small cell lung cancer (NSCLC). The modulation of transcriptional pathways by mutant EGFR signaling is not fully understood. Previously, we and others identified a single base pair change leading to a threonine to methionine (T790M) amino acid alteration in the ATP-binding pocket of the EGFR as a common mechanism of acquired resistance. The gefitinib-resistant, T790M-mutant H1975 NSCLC cell line undergoes prominent growth arrest and apoptosis when treated with the irreversible EGFR inhibitor, CL-387,785. We did a transcriptional profiling study of mutant EGFR target genes that are differentially expressed in the resistant gefitinib-treated and the sensitive CL387,785-treated H1975 cells to identify the pivotal transcriptional changes in NSCLC with EGFR-activating mutations. We identified a small subset of early gene changes, including significant reduction of cyclin D1 as a result of EGFR inhibition by CL-387,785 but not by gefitinib. The reduction in cyclin D1 transcription was associated with subsequent suppression of E2F-responsive genes, consistent with proliferation arrest. Furthermore, cyclin D1 expression was higher in EGFR-mutant lung cancer cells compared with cells with wild-type EGFR. EGFR-mutant cells were routinely sensitive to the cyclin-dependent kinase inhibitor flavopiridol, confirming the functional relevance of the cyclin D axis. These studies suggest that cyclin D1 may contribute to the emergence of EGFR-driven tumorigenesis and can be an alternative target of therapy.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;17145885&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;360&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;57162&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;27250&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23177&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;901&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;54800&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;25840&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Leona Saunders&#39;], &#39;exact_source&#39;: [&#39;Table 1: Up-regulated genes&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div>
    </div>


<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>


    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
            integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
            crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"
            integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
            crossorigin="anonymous"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
            integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
            crossorigin="anonymous"></script>
    <script src="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.js"></script>

    <!-- DataTables, see: https://datatables.net/examples/styling/bootstrap4.html-->
    <script src="https://cdn.datatables.net/1.10.19/js/jquery.dataTables.min.js"></script>
    <script src="https://cdn.datatables.net/1.10.19/js/dataTables.bootstrap4.min.js"></script>

    <script src="https://d3js.org/d3.v5.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.js"></script>

    <script>
        $(function () {
            $('[data-toggle="tooltip"]').tooltip()
        })
    </script>
</body>
</html>