<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>KIM_MYCL1_AMPLIFICATION_TARGETS_UP</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            KIM_MYCL1_AMPLIFICATION_TARGETS_UP
        </div>
        <div class="card-body">
                <p>DNA amplifications and deletions frequently contribute to the development and progression of lung cancer. To identify such novel alterations in small cell lung cancer (SCLC), we performed comparative genomic hybridization on a set of 24 SCLC cell lines, using cDNA microarrays representing approximately 22,000 human genes (providing an average mapping resolution of &lt;70 kb). We identified localized DNA amplifications corresponding to oncogenes known to be amplified in SCLC, including MYC (8q24), MYCN (2p24) and MYCL1 (1p34). Additional highly localized DNA amplifications suggested candidate oncogenes not previously identified as amplified in SCLC, including the antiapoptotic genes TNFRSF4 (1p36), DAD1 (14q11), BCL2L1 (20q11) and BCL2L2 (14q11). Likewise, newly discovered PCR-validated homozygous deletions suggested candidate tumor-suppressor genes, including the proapoptotic genes MAPK10 (4q21) and TNFRSF6 (10q23). To characterize the effect of DNA amplification on gene expression patterns, we performed expression profiling using the same microarray platform. Among our findings, we identified sets of genes whose expression correlated with MYC, MYCN or MYCL1 amplification, with surprisingly little overlap among gene sets. While both MYC and MYCN amplification were associated with increased and decreased expression of known MYC upregulated and downregulated targets, respectively, MYCL1 amplification was associated only with the latter. Our findings support a role of altered apoptotic balance in the pathogenesis of SCLC, and suggest that MYC family genes might affect oncogenesis through distinct sets of targets, in particular implicating the importance of transcriptional repression.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M5329">M5329</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:930">ncbigene:930</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4753">ncbigene:4753</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:25927">ncbigene:25927</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:220965">ncbigene:220965</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8492">ncbigene:8492</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4747">ncbigene:4747</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:65009">ncbigene:65009</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2850">ncbigene:2850</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3751">ncbigene:3751</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4610">ncbigene:4610</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:55924">ncbigene:55924</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8851">ncbigene:8851</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:22836">ncbigene:22836</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:51440">ncbigene:51440</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M5329&#39;, name=&#39;KIM_MYCL1_AMPLIFICATION_TARGETS_UP&#39;), definition=&#39;DNA amplifications and deletions frequently contribute to the development and progression of lung cancer. To identify such novel alterations in small cell lung cancer (SCLC), we performed comparative genomic hybridization on a set of 24 SCLC cell lines, using cDNA microarrays representing approximately 22,000 human genes (providing an average mapping resolution of &lt;70 kb). We identified localized DNA amplifications corresponding to oncogenes known to be amplified in SCLC, including MYC (8q24), MYCN (2p24) and MYCL1 (1p34). Additional highly localized DNA amplifications suggested candidate oncogenes not previously identified as amplified in SCLC, including the antiapoptotic genes TNFRSF4 (1p36), DAD1 (14q11), BCL2L1 (20q11) and BCL2L2 (14q11). Likewise, newly discovered PCR-validated homozygous deletions suggested candidate tumor-suppressor genes, including the proapoptotic genes MAPK10 (4q21) and TNFRSF6 (10q23). To characterize the effect of DNA amplification on gene expression patterns, we performed expression profiling using the same microarray platform. Among our findings, we identified sets of genes whose expression correlated with MYC, MYCN or MYCL1 amplification, with surprisingly little overlap among gene sets. While both MYC and MYCN amplification were associated with increased and decreased expression of known MYC upregulated and downregulated targets, respectively, MYCL1 amplification was associated only with the latter. Our findings support a role of altered apoptotic balance in the pathogenesis of SCLC, and suggest that MYC family genes might affect oncogenesis through distinct sets of targets, in particular implicating the importance of transcriptional repression.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;16116477&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;930&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4753&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;25927&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;220965&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8492&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4747&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;65009&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2850&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3751&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4610&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55924&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8851&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;22836&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51440&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Arthur Liberzon&#39;], &#39;exact_source&#39;: [&#39;Table 3S: positively correlated with MYCL1&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>