<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>WANG_METASTASIS_OF_BREAST_CANCER_ESR1_UP</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            WANG_METASTASIS_OF_BREAST_CANCER_ESR1_UP
        </div>
        <div class="card-body">
                <p>BACKGROUND: Genome-wide measures of gene expression can identify patterns of gene activity that subclassify tumours and might provide a better means than is currently available for individual risk assessment in patients with lymph-node-negative breast cancer. METHODS: We analysed, with Affymetrix Human U133a GeneChips, the expression of 22000 transcripts from total RNA of frozen tumour samples from 286 lymph-node-negative patients who had not received adjuvant systemic treatment. FINDINGS: In a training set of 115 tumours, we identified a 76-gene signature consisting of 60 genes for patients positive for oestrogen receptors (ER) and 16 genes for ER-negative patients. This signature showed 93% sensitivity and 48% specificity in a subsequent independent testing set of 171 lymph-node-negative patients. The gene profile was highly informative in identifying patients who developed distant metastases within 5 years (hazard ratio 5.67 [95% CI 2.59-12.4]), even when corrected for traditional prognostic factors in multivariate analysis (5.55 [2.46-12.5]). The 76-gene profile also represented a strong prognostic factor for the development of metastasis in the subgroups of 84 premenopausal patients (9.60 [2.28-40.5]), 87 postmenopausal patients (4.04 [1.57-10.4]), and 79 patients with tumours of 10-20 mm (14.1 [3.34-59.2]), a group of patients for whom prediction of prognosis is especially difficult. INTERPRETATION: The identified signature provides a powerful tool for identification of patients at high risk of distant recurrence. The ability to identify patients who have a favourable prognosis could, after independent confirmation, allow clinicians to avoid adjuvant systemic therapy or to choose less aggressive therapeutic options.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd>
                    <a href="https://bioregistry.io/msigdb:M2082">M2082</a>
                </dd>

                
                
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<a href="https://bioregistry.io/RO:0002162"><code>RO:0002162</code></a>
)
                    </dt>
                    <dt>
                        <ul>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606"><code>NCBITaxon:9606</code></a>
</li>
                        </ul>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<a href="https://bioregistry.io/BFO:0000051"><code>BFO:0000051</code></a>
)
                    </dt>
                    <dt>
                        <ul>
                                <li><a href="https://bioregistry.io/ncbigene:3838"><code>ncbigene:3838</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:50810"><code>ncbigene:50810</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:10051"><code>ncbigene:10051</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:11335"><code>ncbigene:11335</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:745"><code>ncbigene:745</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:54892"><code>ncbigene:54892</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:10897"><code>ncbigene:10897</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:79682"><code>ncbigene:79682</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:9134"><code>ncbigene:9134</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:5501"><code>ncbigene:5501</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:54963"><code>ncbigene:54963</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:8365"><code>ncbigene:8365</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:1917"><code>ncbigene:1917</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:58986"><code>ncbigene:58986</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:29028"><code>ncbigene:29028</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:10721"><code>ncbigene:10721</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:55596"><code>ncbigene:55596</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:5347"><code>ncbigene:5347</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:51512"><code>ncbigene:51512</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:2237"><code>ncbigene:2237</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:11198"><code>ncbigene:11198</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:5701"><code>ncbigene:5701</code></a>
</li>
                        </ul>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Property</span>
                        category_code
                    </dt>
                    <dt>
                            C2
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Property</span>
                        sub_category_code
                    </dt>
                    <dt>
                            CGP
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Property</span>
                        contributor
                    </dt>
                    <dt>
                            Arthur Liberzon
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Property</span>
                        exact_source
                    </dt>
                    <dt>
                            Table 2: ER-positive group, Cox &gt; 0
                    </dt>
            </dl>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>