<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>WONG_PROTEASOME_GENE_MODULE</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            WONG_PROTEASOME_GENE_MODULE
        </div>
        <div class="card-body">
                <p>A major goal of cancer research is to match specific therapies to molecular targets in cancer. Genome-scale expression profiling has identified new subtypes of cancer based on consistent patterns of variation in gene expression, leading to improved prognostic predictions. However, how these new genetic subtypes of cancers should be treated is unknown. Here, we show that a gene module map can guide the prospective identification of targeted therapies for genetic subtypes of cancer. By visualizing genome-scale gene expression in cancer as combinations of activated and deactivated functional modules, gene module maps can reveal specific functional pathways associated with each subtype that might be susceptible to targeted therapies. We show that in human breast cancers, activation of a poor-prognosis wound signature is strongly associated with induction of both a mitochondria gene module and a proteasome gene module. We found that 3-bromopyruvic acid, which inhibits glycolysis, selectively killed breast cells expressing the mitochondria and wound signatures. In addition, inhibition of proteasome activity by bortezomib, a drug approved for human use in multiple myeloma, abrogated wound signature expression and selectively killed breast cells expressing the wound signature. Thus, gene module maps may enable rapid translation of complex genomic signatures in human disease to targeted therapeutic strategies.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M11458">M11458</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:8975">ncbigene:8975</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10713">ncbigene:10713</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5688">ncbigene:5688</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5714">ncbigene:5714</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5720">ncbigene:5720</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:23014">ncbigene:23014</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5716">ncbigene:5716</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5699">ncbigene:5699</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5702">ncbigene:5702</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:11065">ncbigene:11065</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:25820">ncbigene:25820</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5694">ncbigene:5694</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10197">ncbigene:10197</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:55148">ncbigene:55148</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6477">ncbigene:6477</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5705">ncbigene:5705</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8878">ncbigene:8878</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:11274">ncbigene:11274</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5696">ncbigene:5696</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5692">ncbigene:5692</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7326">ncbigene:7326</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:64854">ncbigene:64854</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5691">ncbigene:5691</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10213">ncbigene:10213</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:991">ncbigene:991</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5707">ncbigene:5707</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:220594">ncbigene:220594</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5682">ncbigene:5682</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5709">ncbigene:5709</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:54926">ncbigene:54926</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7347">ncbigene:7347</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9100">ncbigene:9100</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7398">ncbigene:7398</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8237">ncbigene:8237</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6478">ncbigene:6478</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5686">ncbigene:5686</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:29089">ncbigene:29089</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:29761">ncbigene:29761</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7375">ncbigene:7375</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5690">ncbigene:5690</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:55236">ncbigene:55236</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9101">ncbigene:9101</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7322">ncbigene:7322</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:92912">ncbigene:92912</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5685">ncbigene:5685</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9039">ncbigene:9039</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9097">ncbigene:9097</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:997">ncbigene:997</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7337">ncbigene:7337</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5718">ncbigene:5718</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M11458&#39;, name=&#39;WONG_PROTEASOME_GENE_MODULE&#39;), definition=&#39;A major goal of cancer research is to match specific therapies to molecular targets in cancer. Genome-scale expression profiling has identified new subtypes of cancer based on consistent patterns of variation in gene expression, leading to improved prognostic predictions. However, how these new genetic subtypes of cancers should be treated is unknown. Here, we show that a gene module map can guide the prospective identification of targeted therapies for genetic subtypes of cancer. By visualizing genome-scale gene expression in cancer as combinations of activated and deactivated functional modules, gene module maps can reveal specific functional pathways associated with each subtype that might be susceptible to targeted therapies. We show that in human breast cancers, activation of a poor-prognosis wound signature is strongly associated with induction of both a mitochondria gene module and a proteasome gene module. We found that 3-bromopyruvic acid, which inhibits glycolysis, selectively killed breast cells expressing the mitochondria and wound signatures. In addition, inhibition of proteasome activity by bortezomib, a drug approved for human use in multiple myeloma, abrogated wound signature expression and selectively killed breast cells expressing the wound signature. Thus, gene module maps may enable rapid translation of complex genomic signatures in human disease to targeted therapeutic strategies.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;18199530&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8975&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10713&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5688&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5714&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5720&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23014&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5716&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5699&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5702&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;11065&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;25820&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5694&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10197&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55148&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6477&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5705&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8878&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;11274&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5696&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5692&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7326&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;64854&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5691&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10213&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;991&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5707&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;220594&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5682&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5709&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;54926&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7347&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9100&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7398&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8237&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6478&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5686&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;29089&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;29761&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7375&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5690&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55236&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9101&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7322&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;92912&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5685&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9039&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9097&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;997&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7337&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5718&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Jessica Robertson&#39;], &#39;exact_source&#39;: [&#39;Table 3S&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>