<!DOCTYPE html>
<html>
<head>
            <meta charset="utf-8">
            <meta name="viewport" content="width=device-width, initial-scale=1.0">
            <!-- Bootstrap -->
            <link
                    rel="stylesheet"
                    href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
                    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
                    crossorigin="anonymous"
            />
            <link
                    rel="stylesheet"
                    href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"
            />
            <link
                    rel="stylesheet"
                    href="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.css"
            />
            <!-- DataTables, see: https://datatables.net/examples/styling/bootstrap4.html-->
            <link
                    rel="stylesheet"
                    href="https://cdn.datatables.net/1.10.19/css/dataTables.bootstrap4.min.css"
            />
            <style type="text/css">
                h2 {
                    margin-top: 35px;
                }

                html, body {
                    height: 100%;
                }

                body {
                    display: flex;
                    flex-flow: column;
                }

                .footer {
                    margin-top: auto;
                    padding-top: 1em;
                    background-color: #f5f5f5;
                }

                nav[data-toggle="toc"] {
                    top: 42px;
                }
            </style>

        <script src="https://kit.fontawesome.com/4c86883252.js" crossorigin="anonymous"></script>
        <title>ROVERSI_GLIOMA_LOH_REGIONS</title>
</head>

<body data-spy="scroll" data-target="#toc">
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>


    <div class="container" style="margin-top: 50px; margin-bottom: 50px">
        <div class="card">
        <div class="card-header">
            ROVERSI_GLIOMA_LOH_REGIONS
        </div>
        <div class="card-body">
            
                <p>Identification of genetic copy number changes in glial tumors is of importance in the context of improved/refined diagnostic, prognostic procedures and therapeutic decision-making. In order to detect recurrent genomic copy number changes that might play a role in glioma pathogenesis and/or progression, we characterized 25 primary glioma cell lines including 15 non glioblastoma (non GBM) (I-III WHO grade) and 10 GBM (IV WHO grade), by array comparative genomic hybridization, using a DNA microarray comprising approx. 3500 BACs covering the entire genome with a 1 Mb resolution and additional 800 BACs covering chromosome 19 at tiling path resolution. Combined evaluation by single clone and whole chromosome analysis plus &#39;moving average (MA) approach&#39; enabled us to confirm most of the genetic abnormalities previously identified to be associated with glioma progression, including +1q32, +7, -10, -22q, PTEN and p16 loss, and to disclose new small genomic regions, some correlating with grade malignancy. Grade I-III gliomas exclusively showed losses at 3p26 (53%), 4q13-21 (33%) and 7p15-p21 (26%), whereas only GBMs exhibited 4p16.1 losses (40%). Other recurrent imbalances, such as losses at 4p15, 5q22-q23, 6p23-25, 12p13 and gains at 11p11-q13, were shared by different glioma grades. Three intervals with peak of loss could be further refined for chromosome 10 by our MA approach. Data analysis of full-coverage chromosome 19 highlighted two main regions of copy number gain, never described before in gliomas, at 19p13.11 and 19q13.13-13.2. The well-known 19q13.3 loss of heterozygosity area in gliomas was not frequently affected in our cell lines. Genomic hotspot detection facilitated the identification of small intervals resulting in positional candidate genes such as PRDM2 (1p36.21), LRP1B (2q22.3), ADARB2 (10p15.3), BCCIP (10q26.2) and ING1 (13q34) for losses and ECT2 (3q26.3), MDK, DDB2, IG20 (11p11.2) for gains. These data increase our current knowledge about cryptic genetic changes in gliomas and may facilitate the further identification of novel genetic elements, which may provide us with molecular tools for the improved diagnostics and therapeutic decision-making in these tumors.</p>
            
            <dl>
                <dt>Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M6712">M6712</a></dd>
                
            </dl>


            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M6712&#39;, name=&#39;ROVERSI_GLIOMA_LOH_REGIONS&#39;), definition=&#34;Identification of genetic copy number changes in glial tumors is of importance in the context of improved/refined diagnostic, prognostic procedures and therapeutic decision-making. In order to detect recurrent genomic copy number changes that might play a role in glioma pathogenesis and/or progression, we characterized 25 primary glioma cell lines including 15 non glioblastoma (non GBM) (I-III WHO grade) and 10 GBM (IV WHO grade), by array comparative genomic hybridization, using a DNA microarray comprising approx. 3500 BACs covering the entire genome with a 1 Mb resolution and additional 800 BACs covering chromosome 19 at tiling path resolution. Combined evaluation by single clone and whole chromosome analysis plus &#39;moving average (MA) approach&#39; enabled us to confirm most of the genetic abnormalities previously identified to be associated with glioma progression, including +1q32, +7, -10, -22q, PTEN and p16 loss, and to disclose new small genomic regions, some correlating with grade malignancy. Grade I-III gliomas exclusively showed losses at 3p26 (53%), 4q13-21 (33%) and 7p15-p21 (26%), whereas only GBMs exhibited 4p16.1 losses (40%). Other recurrent imbalances, such as losses at 4p15, 5q22-q23, 6p23-25, 12p13 and gains at 11p11-q13, were shared by different glioma grades. Three intervals with peak of loss could be further refined for chromosome 10 by our MA approach. Data analysis of full-coverage chromosome 19 highlighted two main regions of copy number gain, never described before in gliomas, at 19p13.11 and 19q13.13-13.2. The well-known 19q13.3 loss of heterozygosity area in gliomas was not frequently affected in our cell lines. Genomic hotspot detection facilitated the identification of small intervals resulting in positional candidate genes such as PRDM2 (1p36.21), LRP1B (2q22.3), ADARB2 (10p15.3), BCCIP (10q26.2) and ING1 (13q34) for losses and ECT2 (3q26.3), MDK, DDB2, IG20 (11p11.2) for gains. These data increase our current knowledge about cryptic genetic changes in gliomas and may facilitate the further identification of novel genetic elements, which may provide us with molecular tools for the improved diagnostics and therapeutic decision-making in these tumors.&#34;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;16247447&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55729&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3568&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;83445&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10630&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;29116&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;116984&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;152330&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3764&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2984&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10886&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6310&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10744&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10060&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7799&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2188&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9052&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23284&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6815&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;64841&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;143662&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;50865&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4745&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5817&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;57084&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;203859&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2044&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2904&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8496&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2012&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55507&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23216&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10979&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10553&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2620&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9152&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;80821&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9508&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3945&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;57633&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2638&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2766&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5706&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10196&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1060&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51095&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Arthur Liberzon&#39;], &#39;exact_source&#39;: [&#39;Table 3&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div>
    </div>


<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>


    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
            integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
            crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"
            integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
            crossorigin="anonymous"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
            integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
            crossorigin="anonymous"></script>
    <script src="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.js"></script>

    <!-- DataTables, see: https://datatables.net/examples/styling/bootstrap4.html-->
    <script src="https://cdn.datatables.net/1.10.19/js/jquery.dataTables.min.js"></script>
    <script src="https://cdn.datatables.net/1.10.19/js/dataTables.bootstrap4.min.js"></script>

    <script src="https://d3js.org/d3.v5.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.js"></script>

    <script>
        $(function () {
            $('[data-toggle="tooltip"]').tooltip()
        })
    </script>
</body>
</html>