<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>KOBAYASHI_EGFR_SIGNALING_6HR_DN</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            KOBAYASHI_EGFR_SIGNALING_6HR_DN
        </div>
        <div class="card-body">
                <p>Activating mutations in the epidermal growth factor receptor (EGFR) tyrosine kinase domain determine responsiveness to EGFR tyrosine kinase inhibitors in patients with advanced non-small cell lung cancer (NSCLC). The modulation of transcriptional pathways by mutant EGFR signaling is not fully understood. Previously, we and others identified a single base pair change leading to a threonine to methionine (T790M) amino acid alteration in the ATP-binding pocket of the EGFR as a common mechanism of acquired resistance. The gefitinib-resistant, T790M-mutant H1975 NSCLC cell line undergoes prominent growth arrest and apoptosis when treated with the irreversible EGFR inhibitor, CL-387,785. We did a transcriptional profiling study of mutant EGFR target genes that are differentially expressed in the resistant gefitinib-treated and the sensitive CL387,785-treated H1975 cells to identify the pivotal transcriptional changes in NSCLC with EGFR-activating mutations. We identified a small subset of early gene changes, including significant reduction of cyclin D1 as a result of EGFR inhibition by CL-387,785 but not by gefitinib. The reduction in cyclin D1 transcription was associated with subsequent suppression of E2F-responsive genes, consistent with proliferation arrest. Furthermore, cyclin D1 expression was higher in EGFR-mutant lung cancer cells compared with cells with wild-type EGFR. EGFR-mutant cells were routinely sensitive to the cyclin-dependent kinase inhibitor flavopiridol, confirming the functional relevance of the cyclin D axis. These studies suggest that cyclin D1 may contribute to the emergence of EGFR-driven tumorigenesis and can be an alternative target of therapy.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd>
                    <a href="https://bioregistry.io/msigdb:M659">M659</a>
                </dd>

                
                
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<a href="https://bioregistry.io/RO:0002162"><code>RO:0002162</code></a>
)
                    </dt>
                    <dt>
                        <ul>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606"><code>NCBITaxon:9606</code></a>
</li>
                        </ul>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<a href="https://bioregistry.io/BFO:0000051"><code>BFO:0000051</code></a>
)
                    </dt>
                    <dt>
                        <ul>
                                <li><a href="https://bioregistry.io/ncbigene:8870"><code>ncbigene:8870</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:8061"><code>ncbigene:8061</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:1848"><code>ncbigene:1848</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:7039"><code>ncbigene:7039</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:6376"><code>ncbigene:6376</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:1847"><code>ncbigene:1847</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:1846"><code>ncbigene:1846</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:595"><code>ncbigene:595</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:7422"><code>ncbigene:7422</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:23764"><code>ncbigene:23764</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:50486"><code>ncbigene:50486</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:3725"><code>ncbigene:3725</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:3491"><code>ncbigene:3491</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:8091"><code>ncbigene:8091</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:2152"><code>ncbigene:2152</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:3589"><code>ncbigene:3589</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:7262"><code>ncbigene:7262</code></a>
</li>
                                <li><a href="https://bioregistry.io/ncbigene:1969"><code>ncbigene:1969</code></a>
</li>
                        </ul>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Property</span>
                        category_code
                    </dt>
                    <dt>
                            C2
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Property</span>
                        sub_category_code
                    </dt>
                    <dt>
                            CGP
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Property</span>
                        contributor
                    </dt>
                    <dt>
                            Leona Saunders
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Property</span>
                        exact_source
                    </dt>
                    <dt>
                            Table 1: Down-regulated genes
                    </dt>
            </dl>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>