<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>HATADA_METHYLATED_IN_LUNG_CANCER_DN</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            HATADA_METHYLATED_IN_LUNG_CANCER_DN
        </div>
        <div class="card-body">
                <p>DNA methylation in the promoter region of a gene is associated with a loss of that gene&#39;s expression and plays an important role in gene silencing. The inactivation of tumor-suppressor genes by aberrant methylation in the promoter region is well recognized in carcinogenesis. However, there has been little study in this area when it comes to genome-wide profiling of the promoter methylation. Here, we developed a genome-wide profiling method called Microarray-based Integrated Analysis of Methylation by Isoschizomers to analyse the DNA methylation of promoter regions of 8091 human genes. With this method, resistance to both the methylation-sensitive restriction enzyme HpaII and the methylation-insensitive isoschizomer MspI was compared between samples by using a microarray with promoter regions of the 8091 genes. The reliability of the difference in HpaII resistance was judged using the difference in MspI resistance. We demonstrated the utility of this method by finding epigenetic mutations in cancer. Aberrant hypermethylation is known to inactivate tumour suppressor genes. Using this method, we found that frequency of the aberrant promoter hypermethylation in cancer is higher than previously hypothesized. Aberrant hypomethylation is known to induce activation of oncogenes in cancer. Genome-wide analysis of hypomethylated promoter sequences in cancer demonstrated low CG/GC ratio of these sequences, suggesting that CpG-poor genes are sensitive to demethylation activity in cancer.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M11680">M11680</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:51059">ncbigene:51059</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:84643">ncbigene:84643</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9833">ncbigene:9833</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:22839">ncbigene:22839</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:196477">ncbigene:196477</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9623">ncbigene:9623</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:80274">ncbigene:80274</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1999">ncbigene:1999</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:255101">ncbigene:255101</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3909">ncbigene:3909</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:54937">ncbigene:54937</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:338322">ncbigene:338322</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3718">ncbigene:3718</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:55691">ncbigene:55691</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10151">ncbigene:10151</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:729956">ncbigene:729956</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:79713">ncbigene:79713</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:80032">ncbigene:80032</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:204801">ncbigene:204801</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:126205">ncbigene:126205</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:25791">ncbigene:25791</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:54831">ncbigene:54831</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:140893">ncbigene:140893</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:59284">ncbigene:59284</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:23532">ncbigene:23532</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:57597">ncbigene:57597</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:137209">ncbigene:137209</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:11172">ncbigene:11172</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:158809">ncbigene:158809</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:55731">ncbigene:55731</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4210">ncbigene:4210</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4908">ncbigene:4908</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:285533">ncbigene:285533</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7681">ncbigene:7681</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:23650">ncbigene:23650</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:27004">ncbigene:27004</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M11680&#39;, name=&#39;HATADA_METHYLATED_IN_LUNG_CANCER_DN&#39;), definition=&#34;DNA methylation in the promoter region of a gene is associated with a loss of that gene&#39;s expression and plays an important role in gene silencing. The inactivation of tumor-suppressor genes by aberrant methylation in the promoter region is well recognized in carcinogenesis. However, there has been little study in this area when it comes to genome-wide profiling of the promoter methylation. Here, we developed a genome-wide profiling method called Microarray-based Integrated Analysis of Methylation by Isoschizomers to analyse the DNA methylation of promoter regions of 8091 human genes. With this method, resistance to both the methylation-sensitive restriction enzyme HpaII and the methylation-insensitive isoschizomer MspI was compared between samples by using a microarray with promoter regions of the 8091 genes. The reliability of the difference in HpaII resistance was judged using the difference in MspI resistance. We demonstrated the utility of this method by finding epigenetic mutations in cancer. Aberrant hypermethylation is known to inactivate tumour suppressor genes. Using this method, we found that frequency of the aberrant promoter hypermethylation in cancer is higher than previously hypothesized. Aberrant hypomethylation is known to induce activation of oncogenes in cancer. Genome-wide analysis of hypomethylated promoter sequences in cancer demonstrated low CG/GC ratio of these sequences, suggesting that CpG-poor genes are sensitive to demethylation activity in cancer.&#34;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;16407832&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51059&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;84643&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9833&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;22839&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;196477&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9623&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;80274&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1999&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;255101&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3909&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;54937&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;338322&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3718&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55691&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10151&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;729956&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;79713&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;80032&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;204801&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;126205&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;25791&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;54831&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;140893&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;59284&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23532&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;57597&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;137209&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;11172&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;158809&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55731&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4210&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4908&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;285533&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7681&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23650&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;27004&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Arthur Liberzon&#39;], &#39;exact_source&#39;: [&#39;Table 2S: hypomethylated&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>