<!DOCTYPE html>
<html>
<head>
            <meta charset="utf-8">
            <meta name="viewport" content="width=device-width, initial-scale=1.0">
            <!-- Bootstrap -->
            <link
                    rel="stylesheet"
                    href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
                    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
                    crossorigin="anonymous"
            />
            <link
                    rel="stylesheet"
                    href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"
            />
            <link
                    rel="stylesheet"
                    href="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.css"
            />
            <!-- DataTables, see: https://datatables.net/examples/styling/bootstrap4.html-->
            <link
                    rel="stylesheet"
                    href="https://cdn.datatables.net/1.10.19/css/dataTables.bootstrap4.min.css"
            />
            <style type="text/css">
                h2 {
                    margin-top: 35px;
                }

                html, body {
                    height: 100%;
                }

                body {
                    display: flex;
                    flex-flow: column;
                }

                .footer {
                    margin-top: auto;
                    padding-top: 1em;
                    background-color: #f5f5f5;
                }

                nav[data-toggle="toc"] {
                    top: 42px;
                }
            </style>

        <script src="https://kit.fontawesome.com/4c86883252.js" crossorigin="anonymous"></script>
        <title>RAGHAVACHARI_PLATELET_SPECIFIC_GENES</title>
</head>

<body data-spy="scroll" data-target="#toc">
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>


    <div class="container" style="margin-top: 50px; margin-bottom: 50px">
        <div class="card">
        <div class="card-header">
            RAGHAVACHARI_PLATELET_SPECIFIC_GENES
        </div>
        <div class="card-body">
            
                <p>BACKGROUND: In sickle cell disease, ischemia-reperfusion injury and intravascular hemolysis produce endothelial dysfunction and vasculopathy characterized by reduced nitric oxide and arginine bioavailability. Recent functional studies of platelets in patients with sickle cell disease reveal a basally activated state, which suggests that pathological platelet activation may contribute to sickle cell disease vasculopathy. METHODS AND RESULTS: Studies were therefore undertaken to examine transcriptional signaling pathways in platelets that may be dysregulated in sickle cell disease. We demonstrate and validate in the present study the feasibility of comparative platelet transcriptome studies on clinical samples from single donors by the application of RNA amplification followed by microarray-based analysis of 54,000 probe sets. Data mining an existing microarray database, we identified 220 highly abundant genes in platelets and a subset of 72 relatively platelet-specific genes, defined by &gt;10-fold increased expression compared with the median of other cell types in the database with amplified transcripts. The highly abundant platelet transcripts found in the present study included 82% or 70% of platelet-abundant genes identified in 2 previous gene expression studies on nonamplified mRNA from pooled or apheresis samples, respectively. On comparing the platelet gene expression profiles in 18 patients with sickle cell disease in steady state to those of 12 black control subjects, at a 3-fold cutoff and 5% false-discovery rate, we identified approximately 100 differentially expressed genes, including multiple genes involved in arginine metabolism and redox homeostasis. Further characterization of these pathways with real-time polymerase chain reaction and biochemical assays revealed increased arginase II expression and activity and decreased platelet polyamine levels. CONCLUSIONS: The present studies suggest a potential pathogenic role for platelet arginase and altered arginine and polyamine metabolism in sickle cell disease and provide a novel framework for the study of disease-specific platelet biology.</p>
            
            <dl>
                <dt>Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M7732">M7732</a></dd>
                
            </dl>


            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M7732&#39;, name=&#39;RAGHAVACHARI_PLATELET_SPECIFIC_GENES&#39;), definition=&#39;BACKGROUND: In sickle cell disease, ischemia-reperfusion injury and intravascular hemolysis produce endothelial dysfunction and vasculopathy characterized by reduced nitric oxide and arginine bioavailability. Recent functional studies of platelets in patients with sickle cell disease reveal a basally activated state, which suggests that pathological platelet activation may contribute to sickle cell disease vasculopathy. METHODS AND RESULTS: Studies were therefore undertaken to examine transcriptional signaling pathways in platelets that may be dysregulated in sickle cell disease. We demonstrate and validate in the present study the feasibility of comparative platelet transcriptome studies on clinical samples from single donors by the application of RNA amplification followed by microarray-based analysis of 54,000 probe sets. Data mining an existing microarray database, we identified 220 highly abundant genes in platelets and a subset of 72 relatively platelet-specific genes, defined by &gt;10-fold increased expression compared with the median of other cell types in the database with amplified transcripts. The highly abundant platelet transcripts found in the present study included 82% or 70% of platelet-abundant genes identified in 2 previous gene expression studies on nonamplified mRNA from pooled or apheresis samples, respectively. On comparing the platelet gene expression profiles in 18 patients with sickle cell disease in steady state to those of 12 black control subjects, at a 3-fold cutoff and 5% false-discovery rate, we identified approximately 100 differentially expressed genes, including multiple genes involved in arginine metabolism and redox homeostasis. Further characterization of these pathways with real-time polymerase chain reaction and biochemical assays revealed increased arginase II expression and activity and decreased platelet polyamine levels. CONCLUSIONS: The present studies suggest a potential pathogenic role for platelet arginase and altered arginine and polyamine metabolism in sickle cell disease and provide a novel framework for the study of disease-specific platelet biology.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;17353439&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55281&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7292&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4673&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5742&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8334&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;27071&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3039&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7414&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3043&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2778&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4953&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9848&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5577&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9124&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;26994&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5782&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5197&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;351&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;11031&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;56911&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2717&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3796&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;80230&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;427&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4900&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10857&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;27018&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5494&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51646&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23429&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6622&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4149&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3693&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;54741&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4638&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9588&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8848&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6001&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6352&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;58497&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9910&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7168&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8349&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5196&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3047&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6678&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23531&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2273&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5476&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3690&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;27102&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5305&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3674&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7277&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2017&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4354&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8303&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7057&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6374&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5473&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6251&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7456&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;27030&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2791&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2162&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23052&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7280&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;30851&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5579&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7739&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8357&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Arthur Liberzon&#39;], &#39;exact_source&#39;: [&#39;Table 2S: Relative Expression Index &gt; 1&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div>
    </div>


<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>


    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
            integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
            crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"
            integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
            crossorigin="anonymous"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
            integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
            crossorigin="anonymous"></script>
    <script src="https://cdn.rawgit.com/afeld/bootstrap-toc/v1.0.1/dist/bootstrap-toc.min.js"></script>

    <!-- DataTables, see: https://datatables.net/examples/styling/bootstrap4.html-->
    <script src="https://cdn.datatables.net/1.10.19/js/jquery.dataTables.min.js"></script>
    <script src="https://cdn.datatables.net/1.10.19/js/dataTables.bootstrap4.min.js"></script>

    <script src="https://d3js.org/d3.v5.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.js"></script>

    <script>
        $(function () {
            $('[data-toggle="tooltip"]').tooltip()
        })
    </script>
</body>
</html>