<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>GHO_ATF5_TARGETS_UP</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            GHO_ATF5_TARGETS_UP
        </div>
        <div class="card-body">
                <p>Transcription factors represent an important class of genes that play key roles in controlling cellular proliferation, cell cycle modulation, and attractive targets for cancer therapy. Here, we report on the novel finding of common ATF5 down-regulations in hepatocellular carcinoma (HCC), a highly malignant tumor with a dismal clinical course. Array-based mapping in HCC highlighted a high and consistent incidence of transcription factor ATF5 repressions on regional chr.19q13. By quantitative reverse transcription-PCR, profound down-regulations of ATF5 were further suggested in 78% of HCC tumors (60 of 77 cases) compared to their adjacent nontumoral liver (P = 0.0004). Restoration of ATF5 expression in 3 nonexpressing HCC cell lines demonstrated a consistent growth inhibitory effect (P &lt; 0.029) but minimal induction on cellular apoptosis. Subsequent flow cytometric investigations revealed a G(2)-M cell cycle arrest in HCC cells that were ectopically transfected with ATF5 (P &lt; 0.002). The differential expressed genes from the functional effects of ATF5 were examined by array profiling. Over a hundred genes were identified, among which ID1 contains the ATF/CREB target binding sequences within its promoter region. An inverse relationship between ATF5 expressions with ID1 transcriptions was verified in HCC (P = 0.019), and a direct interaction of ATF5 on the promoter of ID1 was further demonstrated from electromobility shift assay. Examination of causal events underlying the silencing of ATF5 in HCC suggested copy number losses, promoter hypermethylation, histone deacetylation, and DNA mutations to be the likely inactivating mechanisms. In conclusion, our finding supports a tumor suppressive role for ATF5 in HCC, and highlighted ID1 as a potential downstream target.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M12008">M12008</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:8100">ncbigene:8100</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4703">ncbigene:4703</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:51602">ncbigene:51602</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1982">ncbigene:1982</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:26149">ncbigene:26149</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:23200">ncbigene:23200</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:55183">ncbigene:55183</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3913">ncbigene:3913</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9886">ncbigene:9886</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:84706">ncbigene:84706</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:129831">ncbigene:129831</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3699">ncbigene:3699</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:23118">ncbigene:23118</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M12008&#39;, name=&#39;GHO_ATF5_TARGETS_UP&#39;), definition=&#39;Transcription factors represent an important class of genes that play key roles in controlling cellular proliferation, cell cycle modulation, and attractive targets for cancer therapy. Here, we report on the novel finding of common ATF5 down-regulations in hepatocellular carcinoma (HCC), a highly malignant tumor with a dismal clinical course. Array-based mapping in HCC highlighted a high and consistent incidence of transcription factor ATF5 repressions on regional chr.19q13. By quantitative reverse transcription-PCR, profound down-regulations of ATF5 were further suggested in 78% of HCC tumors (60 of 77 cases) compared to their adjacent nontumoral liver (P = 0.0004). Restoration of ATF5 expression in 3 nonexpressing HCC cell lines demonstrated a consistent growth inhibitory effect (P &lt; 0.029) but minimal induction on cellular apoptosis. Subsequent flow cytometric investigations revealed a G(2)-M cell cycle arrest in HCC cells that were ectopically transfected with ATF5 (P &lt; 0.002). The differential expressed genes from the functional effects of ATF5 were examined by array profiling. Over a hundred genes were identified, among which ID1 contains the ATF/CREB target binding sequences within its promoter region. An inverse relationship between ATF5 expressions with ID1 transcriptions was verified in HCC (P = 0.019), and a direct interaction of ATF5 on the promoter of ID1 was further demonstrated from electromobility shift assay. Examination of causal events underlying the silencing of ATF5 in HCC suggested copy number losses, promoter hypermethylation, histone deacetylation, and DNA mutations to be the likely inactivating mechanisms. In conclusion, our finding supports a tumor suppressive role for ATF5 in HCC, and highlighted ID1 as a potential downstream target.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;18701499&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8100&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4703&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51602&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1982&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;26149&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23200&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55183&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3913&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9886&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;84706&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;129831&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3699&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23118&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Jessica Robertson&#39;], &#39;exact_source&#39;: [&#39;Fig 4A: red in ATF5&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>