<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>SCHLESINGER_H3K27ME3_IN_NORMAL_AND_METHYLATED_IN_CANCER</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            SCHLESINGER_H3K27ME3_IN_NORMAL_AND_METHYLATED_IN_CANCER
        </div>
        <div class="card-body">
                <p>Many genes associated with CpG islands undergo de novo methylation in cancer. Studies have suggested that the pattern of this modification may be partially determined by an instructive mechanism that recognizes specifically marked regions of the genome. Using chromatin immunoprecipitation analysis, here we show that genes methylated in cancer cells are specifically packaged with nucleosomes containing histone H3 trimethylated on Lys27. This chromatin mark is established on these unmethylated CpG island genes early in development and then maintained in differentiated cell types by the presence of an EZH2-containing Polycomb complex. In cancer cells, as opposed to normal cells, the presence of this complex brings about the recruitment of DNA methyl transferases, leading to de novo methylation. These results suggest that tumor-specific targeting of de novo methylation is pre-programmed by an established epigenetic system that normally has a role in marking embryonic genes for repression.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M16297">M16297</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:362">ncbigene:362</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:50674">ncbigene:50674</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:57369">ncbigene:57369</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2917">ncbigene:2917</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5459">ncbigene:5459</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2572">ncbigene:2572</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3350">ncbigene:3350</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3238">ncbigene:3238</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6424">ncbigene:6424</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1674">ncbigene:1674</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3205">ncbigene:3205</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:56101">ncbigene:56101</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9956">ncbigene:9956</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:83541">ncbigene:83541</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:29953">ncbigene:29953</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8651">ncbigene:8651</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2916">ncbigene:2916</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:56135">ncbigene:56135</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2662">ncbigene:2662</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1395">ncbigene:1395</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1277">ncbigene:1277</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4661">ncbigene:4661</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2893">ncbigene:2893</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2774">ncbigene:2774</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:273">ncbigene:273</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3040">ncbigene:3040</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2830">ncbigene:2830</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3736">ncbigene:3736</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M16297&#39;, name=&#39;SCHLESINGER_H3K27ME3_IN_NORMAL_AND_METHYLATED_IN_CANCER&#39;), definition=&#39;Many genes associated with CpG islands undergo de novo methylation in cancer. Studies have suggested that the pattern of this modification may be partially determined by an instructive mechanism that recognizes specifically marked regions of the genome. Using chromatin immunoprecipitation analysis, here we show that genes methylated in cancer cells are specifically packaged with nucleosomes containing histone H3 trimethylated on Lys27. This chromatin mark is established on these unmethylated CpG island genes early in development and then maintained in differentiated cell types by the presence of an EZH2-containing Polycomb complex. In cancer cells, as opposed to normal cells, the presence of this complex brings about the recruitment of DNA methyl transferases, leading to de novo methylation. These results suggest that tumor-specific targeting of de novo methylation is pre-programmed by an established epigenetic system that normally has a role in marking embryonic genes for repression.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;17200670&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;362&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;50674&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;57369&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2917&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5459&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2572&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3350&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3238&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6424&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1674&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3205&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;56101&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9956&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;83541&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;29953&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8651&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2916&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;56135&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2662&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1395&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1277&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4661&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2893&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2774&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;273&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3040&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2830&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3736&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Leona Saunders&#39;], &#39;exact_source&#39;: [&#39;Table 1S&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>