<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>POS_RESPONSE_TO_HISTAMINE_UP</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            POS_RESPONSE_TO_HISTAMINE_UP
        </div>
        <div class="card-body">
                <p>We previously showed that transgenic enhancement of histamine production in B16-F10 melanomas strongly supports tumor growth in C57BL/6 mice. In the present study, gene expression profiles of transgenic mouse melanomas, secreting different amounts of histamine, were compared by whole genome microarrays. Array results were validated by real-time PCR, and genes showing histamine-affected behavior were further analyzed by immunohistochemistry. Regulation of histamine-coupled genes was investigated by checking the presence and functional integrity of all four known histamine receptors in experimental melanomas and by administering histamine H1 receptor (H1R) and H2 receptor (H2R) antagonists to tumor-bearing mice. Finally, an attempt was made to integrate histamine-affected genes in known gene regulatory circuits by in silico pathway analysis. Our results show that histamine enhances melanoma growth via H1R rather than through H2R. We show that H1R activation suppresses RNA-level expression of the tumor suppressor insulin-like growth factor II receptor (IGF-IIR) and the antiangiogenic matrix protein fibulin-5 (FBLN5), decreases their intracellular protein levels, and also reduces their availability in the plasma membrane and extracellular matrix, respectively. Pathway analysis suggests that because plasma membrane-bound IGF-IIR is required to activate matrix-bound, latent transforming growth factor-beta1, a factor suggested to sustain FBLN5 expression, the data can be integrated in a known antineoplastic regulatory pathway that is suppressed by H1R. On the other hand, we show that engagement of H2R also reduces intracellular protein pools of IGF-IIR and FBLN5, but being a downstream acting posttranslational effect with minimal consequences on exported IGF-IIR and FBLN5 protein levels, H2R is rather irrelevant compared with H1R in melanoma.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M1782">M1782</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:10090">NCBITaxon:10090</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:51244">ncbigene:51244</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4956">ncbigene:4956</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:79622">ncbigene:79622</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9895">ncbigene:9895</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10516">ncbigene:10516</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:285527">ncbigene:285527</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9555">ncbigene:9555</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:377677">ncbigene:377677</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3482">ncbigene:3482</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5217">ncbigene:5217</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10362">ncbigene:10362</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:81609">ncbigene:81609</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M1782&#39;, name=&#39;POS_RESPONSE_TO_HISTAMINE_UP&#39;), definition=&#39;We previously showed that transgenic enhancement of histamine production in B16-F10 melanomas strongly supports tumor growth in C57BL/6 mice. In the present study, gene expression profiles of transgenic mouse melanomas, secreting different amounts of histamine, were compared by whole genome microarrays. Array results were validated by real-time PCR, and genes showing histamine-affected behavior were further analyzed by immunohistochemistry. Regulation of histamine-coupled genes was investigated by checking the presence and functional integrity of all four known histamine receptors in experimental melanomas and by administering histamine H1 receptor (H1R) and H2 receptor (H2R) antagonists to tumor-bearing mice. Finally, an attempt was made to integrate histamine-affected genes in known gene regulatory circuits by in silico pathway analysis. Our results show that histamine enhances melanoma growth via H1R rather than through H2R. We show that H1R activation suppresses RNA-level expression of the tumor suppressor insulin-like growth factor II receptor (IGF-IIR) and the antiangiogenic matrix protein fibulin-5 (FBLN5), decreases their intracellular protein levels, and also reduces their availability in the plasma membrane and extracellular matrix, respectively. Pathway analysis suggests that because plasma membrane-bound IGF-IIR is required to activate matrix-bound, latent transforming growth factor-beta1, a factor suggested to sustain FBLN5 expression, the data can be integrated in a known antineoplastic regulatory pathway that is suppressed by H1R. On the other hand, we show that engagement of H2R also reduces intracellular protein pools of IGF-IIR and FBLN5, but being a downstream acting posttranslational effect with minimal consequences on exported IGF-IIR and FBLN5 protein levels, H2R is rather irrelevant compared with H1R in melanoma.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;18339882&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;10090&#39;, name=&#39;Mus musculus&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51244&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4956&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;79622&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9895&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10516&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;285527&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9555&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;377677&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3482&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5217&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10362&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;81609&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Jessica Robertson&#39;], &#39;exact_source&#39;: [&#39;Fig 1A: black&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>