<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>NIKOLSKY_BREAST_CANCER_6P24_P22_AMPLICON</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            NIKOLSKY_BREAST_CANCER_6P24_P22_AMPLICON
        </div>
        <div class="card-body">
                <p>A single cancer cell contains large numbers of genetic alterations that in combination create the malignant phenotype. However, whether amplified and mutated genes form functional and physical interaction networks that could explain the selection for cells with combined alterations is unknown. To investigate this issue, we characterized copy number alterations in 191 breast tumors using dense single nucleotide polymorphism arrays and identified 1,747 genes with copy number gain organized into 30 amplicons. Amplicons were distributed unequally throughout the genome. Each amplicon had distinct enrichment pattern in pathways, networks, and molecular functions, but genes within individual amplicons did not form coherent functional units. Genes in amplicons included all major tumorigenic pathways and were highly enriched in breast cancer-causative genes. In contrast, 1,188 genes with somatic mutations in breast cancer were distributed randomly over the genome, did not represent a functionally cohesive gene set, and were relatively less enriched in breast cancer marker genes. Mutated and gained genes did not show statistically significant overlap but were highly synergistic in populating key tumorigenic pathways including transforming growth factor beta, WNT, fibroblast growth factor, and PIP3 signaling. In general, mutated genes were more frequently upstream of gained genes in transcription regulation signaling than vice versa, suggesting that mutated genes are mainly regulators, whereas gained genes are mostly regulated. ESR1 was the major transcription factor regulating amplified but not mutated genes. Our results support the hypothesis that multiple genetic events, including copy number gains and somatic mutations, are necessary for establishing the malignant cell phenotype.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M4276">M4276</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:9972">ncbigene:9972</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:51256">ncbigene:51256</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:221692">ncbigene:221692</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:9308">ncbigene:9308</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3400">ncbigene:3400</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:221656">ncbigene:221656</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:154141">ncbigene:154141</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:255488">ncbigene:255488</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:54438">ncbigene:54438</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7172">ncbigene:7172</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7913">ncbigene:7913</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:63933">ncbigene:63933</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:221662">ncbigene:221662</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:23408">ncbigene:23408</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10486">ncbigene:10486</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:221687">ncbigene:221687</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:63971">ncbigene:63971</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:378884">ncbigene:378884</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:51406">ncbigene:51406</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10048">ncbigene:10048</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:51439">ncbigene:51439</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M4276&#39;, name=&#39;NIKOLSKY_BREAST_CANCER_6P24_P22_AMPLICON&#39;), definition=&#39;A single cancer cell contains large numbers of genetic alterations that in combination create the malignant phenotype. However, whether amplified and mutated genes form functional and physical interaction networks that could explain the selection for cells with combined alterations is unknown. To investigate this issue, we characterized copy number alterations in 191 breast tumors using dense single nucleotide polymorphism arrays and identified 1,747 genes with copy number gain organized into 30 amplicons. Amplicons were distributed unequally throughout the genome. Each amplicon had distinct enrichment pattern in pathways, networks, and molecular functions, but genes within individual amplicons did not form coherent functional units. Genes in amplicons included all major tumorigenic pathways and were highly enriched in breast cancer-causative genes. In contrast, 1,188 genes with somatic mutations in breast cancer were distributed randomly over the genome, did not represent a functionally cohesive gene set, and were relatively less enriched in breast cancer marker genes. Mutated and gained genes did not show statistically significant overlap but were highly synergistic in populating key tumorigenic pathways including transforming growth factor beta, WNT, fibroblast growth factor, and PIP3 signaling. In general, mutated genes were more frequently upstream of gained genes in transcription regulation signaling than vice versa, suggesting that mutated genes are mainly regulators, whereas gained genes are mostly regulated. ESR1 was the major transcription factor regulating amplified but not mutated genes. Our results support the hypothesis that multiple genetic events, including copy number gains and somatic mutations, are necessary for establishing the malignant cell phenotype.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;19010930&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9972&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51256&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;221692&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;9308&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3400&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;221656&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;154141&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;255488&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;54438&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7172&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7913&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;63933&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;221662&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23408&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10486&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;221687&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;63971&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;378884&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51406&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10048&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51439&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Jessica Robertson&#39;], &#39;exact_source&#39;: [&#39;Suppl. Excel File 1S: 6p24-p22&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>