<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>GRADE_COLON_VS_RECTAL_CANCER_UP</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            GRADE_COLON_VS_RECTAL_CANCER_UP
        </div>
        <div class="card-body">
                <p>To characterize patterns of global transcriptional deregulation in primary colon carcinomas, we did gene expression profiling of 73 tumors [Unio Internationale Contra Cancrum stage II (n = 33) and stage III (n = 40)] using oligonucleotide microarrays. For 30 of the tumors, expression profiles were compared with those from matched normal mucosa samples. We identified a set of 1,950 genes with highly significant deregulation between tumors and mucosa samples (P &lt; 1e-7). A significant proportion of these genes mapped to chromosome 20 (P = 0.01). Seventeen genes had a &gt;5-fold average expression difference between normal colon mucosa and carcinomas, including up-regulation of MYC and of HMGA1, a putative oncogene. Furthermore, we identified 68 genes that were significantly differentially expressed between lymph node-negative and lymph node-positive tumors (P &lt; 0.001), the functional annotation of which revealed a preponderance of genes that play a role in cellular immune response and surveillance. The microarray-derived gene expression levels of 20 deregulated genes were validated using quantitative real-time reverse transcription-PCR in &gt;40 tumor and normal mucosa samples with good concordance between the techniques. Finally, we established a relationship between specific genomic imbalances, which were mapped for 32 of the analyzed colon tumors by comparative genomic hybridization, and alterations of global transcriptional activity. Previously, we had conducted a similar analysis of primary rectal carcinomas. The systematic comparison of colon and rectal carcinomas revealed a significant overlap of genomic imbalances and transcriptional deregulation, including activation of the Wnt/beta-catenin signaling cascade, suggesting similar pathogenic pathways.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M9538">M9538</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:57214">ncbigene:57214</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2054">ncbigene:2054</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:29880">ncbigene:29880</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8270">ncbigene:8270</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:64772">ncbigene:64772</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:821">ncbigene:821</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:56852">ncbigene:56852</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:51705">ncbigene:51705</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:79009">ncbigene:79009</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:221395">ncbigene:221395</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10495">ncbigene:10495</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3552">ncbigene:3552</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:54914">ncbigene:54914</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:64919">ncbigene:64919</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:22927">ncbigene:22927</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:94031">ncbigene:94031</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:11167">ncbigene:11167</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4616">ncbigene:4616</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:79621">ncbigene:79621</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:947">ncbigene:947</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:54881">ncbigene:54881</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:54873">ncbigene:54873</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7372">ncbigene:7372</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6259">ncbigene:6259</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:55055">ncbigene:55055</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:55197">ncbigene:55197</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5375">ncbigene:5375</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:64881">ncbigene:64881</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:51141">ncbigene:51141</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1503">ncbigene:1503</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:23433">ncbigene:23433</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3183">ncbigene:3183</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:63939">ncbigene:63939</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3726">ncbigene:3726</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:55723">ncbigene:55723</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:51776">ncbigene:51776</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:30001">ncbigene:30001</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:51029">ncbigene:51029</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M9538&#39;, name=&#39;GRADE_COLON_VS_RECTAL_CANCER_UP&#39;), definition=&#39;To characterize patterns of global transcriptional deregulation in primary colon carcinomas, we did gene expression profiling of 73 tumors [Unio Internationale Contra Cancrum stage II (n = 33) and stage III (n = 40)] using oligonucleotide microarrays. For 30 of the tumors, expression profiles were compared with those from matched normal mucosa samples. We identified a set of 1,950 genes with highly significant deregulation between tumors and mucosa samples (P &lt; 1e-7). A significant proportion of these genes mapped to chromosome 20 (P = 0.01). Seventeen genes had a &gt;5-fold average expression difference between normal colon mucosa and carcinomas, including up-regulation of MYC and of HMGA1, a putative oncogene. Furthermore, we identified 68 genes that were significantly differentially expressed between lymph node-negative and lymph node-positive tumors (P &lt; 0.001), the functional annotation of which revealed a preponderance of genes that play a role in cellular immune response and surveillance. The microarray-derived gene expression levels of 20 deregulated genes were validated using quantitative real-time reverse transcription-PCR in &gt;40 tumor and normal mucosa samples with good concordance between the techniques. Finally, we established a relationship between specific genomic imbalances, which were mapped for 32 of the analyzed colon tumors by comparative genomic hybridization, and alterations of global transcriptional activity. Previously, we had conducted a similar analysis of primary rectal carcinomas. The systematic comparison of colon and rectal carcinomas revealed a significant overlap of genomic imbalances and transcriptional deregulation, including activation of the Wnt/beta-catenin signaling cascade, suggesting similar pathogenic pathways.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;17210682&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;57214&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2054&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;29880&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8270&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;64772&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;821&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;56852&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51705&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;79009&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;221395&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10495&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3552&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;54914&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;64919&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;22927&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;94031&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;11167&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4616&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;79621&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;947&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;54881&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;54873&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7372&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6259&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55055&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55197&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5375&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;64881&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51141&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1503&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;23433&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3183&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;63939&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3726&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55723&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51776&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;30001&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51029&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Jessica Robertson&#39;], &#39;exact_source&#39;: [&#39;Table 5S: Colon &lt; 1 &amp; Rectum &gt; 1&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>