<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>BOYAULT_LIVER_CANCER_SUBCLASS_G56_UP</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            BOYAULT_LIVER_CANCER_SUBCLASS_G56_UP
        </div>
        <div class="card-body">
                <p>Hepatocellular carcinomas (HCCs) are a heterogeneous group of tumors that differ in risk factors and genetic alterations. We further investigated transcriptome-genotype-phenotype correlations in HCC. Global transcriptome analyses were performed on 57 HCCs and 3 hepatocellular adenomas and validated by quantitative RT-PCR using 63 additional HCCs. We determined loss of heterozygosity, gene mutations, promoter methylation of CDH1 and CDKN2A, and HBV DNA copy number for each tumor. Unsupervised transcriptome analysis identified 6 robust subgroups of HCC (G1-G6) associated with clinical and genetic characteristics. G1 tumors were associated with low copy number of HBV and overexpression of genes expressed in fetal liver and controlled by parental imprinting. G2 included HCCs infected with a high copy number of HBV and mutations in PIK3CA and TP53. In these first groups, we detected specific activation of the AKT pathway. G3 tumors were typified by mutation of TP53 and overexpression of genes controlling the cell cycle. G4 was a heterogeneous subgroup of tumors including TCF1-mutated hepatocellular adenomas and carcinomas. G5 and G6 were strongly related to beta-catenin mutations that lead to Wnt pathway activation; in particular, G6 tumors were characterized by satellite nodules, higher activation of the Wnt pathway, and E-cadherin underexpression. CONCLUSION: These results have furthered our understanding of the genetic diversity of human HCC and have provided specific identifiers for classifying tumors. In addition, our classification has potential therapeutic implications because 50% of the tumors were related to WNT or AKT pathway activation, which potentially could be targeted by specific inhibiting therapies.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M1925">M1925</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:9606">NCBITaxon:9606</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:2752">ncbigene:2752</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:57127">ncbigene:57127</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1803">ncbigene:1803</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1854">ncbigene:1854</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3909">ncbigene:3909</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5068">ncbigene:5068</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8404">ncbigene:8404</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6926">ncbigene:6926</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4734">ncbigene:4734</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:56950">ncbigene:56950</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:55313">ncbigene:55313</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M1925&#39;, name=&#39;BOYAULT_LIVER_CANCER_SUBCLASS_G56_UP&#39;), definition=&#39;Hepatocellular carcinomas (HCCs) are a heterogeneous group of tumors that differ in risk factors and genetic alterations. We further investigated transcriptome-genotype-phenotype correlations in HCC. Global transcriptome analyses were performed on 57 HCCs and 3 hepatocellular adenomas and validated by quantitative RT-PCR using 63 additional HCCs. We determined loss of heterozygosity, gene mutations, promoter methylation of CDH1 and CDKN2A, and HBV DNA copy number for each tumor. Unsupervised transcriptome analysis identified 6 robust subgroups of HCC (G1-G6) associated with clinical and genetic characteristics. G1 tumors were associated with low copy number of HBV and overexpression of genes expressed in fetal liver and controlled by parental imprinting. G2 included HCCs infected with a high copy number of HBV and mutations in PIK3CA and TP53. In these first groups, we detected specific activation of the AKT pathway. G3 tumors were typified by mutation of TP53 and overexpression of genes controlling the cell cycle. G4 was a heterogeneous subgroup of tumors including TCF1-mutated hepatocellular adenomas and carcinomas. G5 and G6 were strongly related to beta-catenin mutations that lead to Wnt pathway activation; in particular, G6 tumors were characterized by satellite nodules, higher activation of the Wnt pathway, and E-cadherin underexpression. CONCLUSION: These results have furthered our understanding of the genetic diversity of human HCC and have provided specific identifiers for classifying tumors. In addition, our classification has potential therapeutic implications because 50% of the tumors were related to WNT or AKT pathway activation, which potentially could be targeted by specific inhibiting therapies.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;17187432&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;9606&#39;, name=&#39;Homo sapiens&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2752&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;57127&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1803&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1854&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3909&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5068&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8404&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6926&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4734&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;56950&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55313&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Yujin Hoshida&#39;], &#39;exact_source&#39;: [&#39;Table 4S: G56_UP&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>