<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>JACKSON_DNMT1_TARGETS_DN</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            JACKSON_DNMT1_TARGETS_DN
        </div>
        <div class="card-body">
                <p>Cytosine methylation of mammalian DNA is essential for the proper epigenetic regulation of gene expression and maintenance of genomic integrity. To define the mechanism through which demethylated cells die, and to establish a paradigm for identifying genes regulated by DNA methylation, we have generated mice with a conditional allele for the maintenance DNA methyltransferase gene Dnmt1. Cre-mediated deletion of Dnmt1 causes demethylation of cultured fibroblasts and a uniform p53-dependent cell death. Mutational inactivation of Trp53 partially rescues the demethylated fibroblasts for up to five population doublings in culture. Oligonucleotide microarray analysis showed that up to 10% of genes are aberrantly expressed in demethylated fibroblasts. Our results demonstrate that loss of Dnmt1 causes cell-type-specific changes in gene expression that impinge on several pathways, including expression of imprinted genes, cell-cycle control, growth factor/receptor signal transduction and mobilization of retroelements.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M1627">M1627</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:10090">NCBITaxon:10090</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:2706">ncbigene:2706</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3949">ncbigene:3949</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1381">ncbigene:1381</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1297">ncbigene:1297</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3481">ncbigene:3481</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3728">ncbigene:3728</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5424">ncbigene:5424</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1292">ncbigene:1292</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3146">ncbigene:3146</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2620">ncbigene:2620</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:11338">ncbigene:11338</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2887">ncbigene:2887</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1301">ncbigene:1301</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:8325">ncbigene:8325</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1786">ncbigene:1786</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1058">ncbigene:1058</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1291">ncbigene:1291</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:5178">ncbigene:5178</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1947">ncbigene:1947</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4191">ncbigene:4191</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6319">ncbigene:6319</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M1627&#39;, name=&#39;JACKSON_DNMT1_TARGETS_DN&#39;), definition=&#39;Cytosine methylation of mammalian DNA is essential for the proper epigenetic regulation of gene expression and maintenance of genomic integrity. To define the mechanism through which demethylated cells die, and to establish a paradigm for identifying genes regulated by DNA methylation, we have generated mice with a conditional allele for the maintenance DNA methyltransferase gene Dnmt1. Cre-mediated deletion of Dnmt1 causes demethylation of cultured fibroblasts and a uniform p53-dependent cell death. Mutational inactivation of Trp53 partially rescues the demethylated fibroblasts for up to five population doublings in culture. Oligonucleotide microarray analysis showed that up to 10% of genes are aberrantly expressed in demethylated fibroblasts. Our results demonstrate that loss of Dnmt1 causes cell-type-specific changes in gene expression that impinge on several pathways, including expression of imprinted genes, cell-cycle control, growth factor/receptor signal transduction and mobilization of retroelements.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;11137995&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;10090&#39;, name=&#39;Mus musculus&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2706&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3949&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1381&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1297&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3481&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3728&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5424&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1292&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3146&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2620&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;11338&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2887&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1301&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;8325&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1786&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1058&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1291&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;5178&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1947&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4191&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6319&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;John Newman&#39;], &#39;exact_source&#39;: [&#39;Table 2: Fold &lt;= -2&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>