<!DOCTYPE html>
<html lang="en">
<head>        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link
        rel="stylesheet"
        href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm"
        crossorigin="anonymous"
    />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/c3/0.6.12/c3.min.css"/>
    <style>
        h2 {
            margin-top: 35px;
        }

        html, body {
            height: 100%;
        }

        body {
            display: flex;
            flex-flow: column;
        }

        .footer {
            margin-top: auto;
            padding-top: 1em;
            background-color: #f5f5f5;
        }
    </style>    <title>HORTON_SREBF_TARGETS</title></head>

<body>
<nav class="navbar navbar-expand-lg navbar-light bg-light">
    <span class="navbar-brand">
        Molecular Signatures Database
    </span>
</nav>

<div class="container" style="margin-top: 50px; margin-bottom: 50px">
<div class="card">
        <div class="card-header">
            HORTON_SREBF_TARGETS
        </div>
        <div class="card-body">
                <p>The synthesis of fatty acids and cholesterol, the building blocks of membranes, is regulated by three membrane-bound transcription factors: sterol regulatory element-binding proteins (SREBP)-1a, -1c, and -2. Their function in liver has been characterized in transgenic mice that overexpress each SREBP isoform and in mice that lack all three nuclear SREBPs as a result of gene knockout of SREBP cleavage-activating protein (SCAP), a protein required for nuclear localization of SREBPs. Here, we use oligonucleotide arrays hybridized with RNA from livers of three lines of mice (transgenic for SREBP-1a, transgenic for SREBP-2, and knockout for SCAP) to identify genes that are likely to be direct targets of SREBPs in liver. A total of 1,003 genes showed statistically significant increased expression in livers of transgenic SREBP-1a mice, 505 increased in livers of transgenic SREBP-2 mice, and 343 showed decreased expression in Scap-/- livers. A subset of 33 genes met the stringent combinatorial criteria of induction in both SREBP transgenics and decreased expression in SCAP-deficient mice. Of these 33 genes, 13 were previously identified as direct targets of SREBP action. Of the remaining 20 genes, 13 encode enzymes or carrier proteins involved in cholesterol metabolism, 3 participate in fatty acid metabolism, and 4 have no known connection to lipid metabolism. Through application of stringent combinatorial criteria, the transgenic/knockout approach allows identification of genes whose activities are likely to be controlled directly by one family of transcription factors, in this case the SREBPs.</p>
            <dl>
                <dt>Local Unique Identifier</dt>
                <dd><a href="https://bioregistry.io/msigdb:M3009">M3009</a>
                </dd>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            in taxon (<code>RO:0002162</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/NCBITaxon:10090">NCBITaxon:10090</a></li>
                        </ol>
                    </dt>
                    <dt>
                        <span class="badge badge-primary">Relation</span>
                            has part (<code>BFO:0000051</code>)
                    </dt>
                    <dt>
                        <ol>
                                <li><a href="https://bioregistry.io/ncbigene:65985">ncbigene:65985</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:7108">ncbigene:7108</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2224">ncbigene:2224</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6713">ncbigene:6713</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:79071">ncbigene:79071</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:55902">ncbigene:55902</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:51109">ncbigene:51109</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2194">ncbigene:2194</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:10654">ncbigene:10654</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:134429">ncbigene:134429</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:230">ncbigene:230</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4199">ncbigene:4199</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3422">ncbigene:3422</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:1595">ncbigene:1595</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:2222">ncbigene:2222</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:51478">ncbigene:51478</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:31">ncbigene:31</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:6309">ncbigene:6309</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3949">ncbigene:3949</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3638">ncbigene:3638</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:3157">ncbigene:3157</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:255738">ncbigene:255738</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:47">ncbigene:47</a></li>
                                <li><a href="https://bioregistry.io/ncbigene:4047">ncbigene:4047</a></li>
                        </ol>
                    </dt>
            </dl>
            <span class="badge badge-warning">Term generation via PyOBO is currently under construction</span>
            <blockquote>
                Term(reference=Reference(prefix=&#39;msigdb&#39;, identifier=&#39;M3009&#39;, name=&#39;HORTON_SREBF_TARGETS&#39;), definition=&#39;The synthesis of fatty acids and cholesterol, the building blocks of membranes, is regulated by three membrane-bound transcription factors: sterol regulatory element-binding proteins (SREBP)-1a, -1c, and -2. Their function in liver has been characterized in transgenic mice that overexpress each SREBP isoform and in mice that lack all three nuclear SREBPs as a result of gene knockout of SREBP cleavage-activating protein (SCAP), a protein required for nuclear localization of SREBPs. Here, we use oligonucleotide arrays hybridized with RNA from livers of three lines of mice (transgenic for SREBP-1a, transgenic for SREBP-2, and knockout for SCAP) to identify genes that are likely to be direct targets of SREBPs in liver. A total of 1,003 genes showed statistically significant increased expression in livers of transgenic SREBP-1a mice, 505 increased in livers of transgenic SREBP-2 mice, and 343 showed decreased expression in Scap-/- livers. A subset of 33 genes met the stringent combinatorial criteria of induction in both SREBP transgenics and decreased expression in SCAP-deficient mice. Of these 33 genes, 13 were previously identified as direct targets of SREBP action. Of the remaining 20 genes, 13 encode enzymes or carrier proteins involved in cholesterol metabolism, 3 participate in fatty acid metabolism, and 4 have no known connection to lipid metabolism. Through application of stringent combinatorial criteria, the transgenic/knockout approach allows identification of genes whose activities are likely to be controlled directly by one family of transcription factors, in this case the SREBPs.&#39;, provenance=[Reference(prefix=&#39;pubmed&#39;, identifier=&#39;14512514&#39;, name=None)], relationships=defaultdict(&lt;class &#39;list&#39;&gt;, {TypeDef(reference=Reference(prefix=&#39;RO&#39;, identifier=&#39;0002162&#39;, name=&#39;in taxon&#39;), comment=None, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=None, holds_over_chain=None): [Reference(prefix=&#39;NCBITaxon&#39;, identifier=&#39;10090&#39;, name=&#39;Mus musculus&#39;)], TypeDef(reference=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000051&#39;, name=&#39;has part&#39;), comment=&#39;Inverse of part_of&#39;, namespace=None, definition=None, is_transitive=None, is_symmetric=None, domain=None, range=None, parents=[], xrefs=[], inverse=Reference(prefix=&#39;BFO&#39;, identifier=&#39;0000050&#39;, name=&#39;part of&#39;), holds_over_chain=None): [Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;65985&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;7108&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2224&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6713&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;79071&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;55902&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51109&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2194&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;10654&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;134429&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;230&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4199&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3422&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;1595&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;2222&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;51478&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;31&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;6309&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3949&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3638&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;3157&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;255738&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;47&#39;, name=None), Reference(prefix=&#39;ncbigene&#39;, identifier=&#39;4047&#39;, name=None)]}), properties=defaultdict(&lt;class &#39;list&#39;&gt;, {&#39;category_code&#39;: [&#39;C2&#39;], &#39;sub_category_code&#39;: [&#39;CGP&#39;], &#39;contributor&#39;: [&#39;Arthur Liberzon&#39;], &#39;exact_source&#39;: [&#39;Table 1&#39;]}), parents=[], synonyms=[], xrefs=[], alt_ids=[], namespace=None, is_obsolete=False)
            </blockquote>
        </div>
    </div></div>

<footer class="footer">
    <p class="small text-center text-muted">
        Generated with PyOBO
    </p>
</footer>

<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
        crossorigin="anonymous"></script>
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"
        integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
        crossorigin="anonymous"></script></body>
</html>